<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>1&nbsp; Data Exploration - Kidney Cancer – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./transcriptomics-analysis.html" rel="next">
<link href="./gather-datasets-and-QC.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-fixedcolumns-1.13.6/css/fixedColumns.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedcolumns-1.13.6/js/dataTables.fixedColumns.min.js"></script>
<link href="site_libs/dt-ext-fixedheader-1.13.6/css/fixedHeader.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedheader-1.13.6/js/dataTables.fixedHeader.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./dataset-exploration.html"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./survival-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./kidney-carcinomas-lessons-learnt.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">kidney-carcinomas-lessons-learnt.html</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell Cancer molecular subtypes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cancer-signatures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">cancer-signatures.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./autoencoder.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">autoencoder.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cellular-deconvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Cellular Deconvolution from Bulk RNA-seq</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">1.1</span> On this page</a></li>
  <li><a href="#selection-of-kidney-samples" id="toc-selection-of-kidney-samples" class="nav-link" data-scroll-target="#selection-of-kidney-samples"><span class="header-section-number">1.2</span> Selection of Kidney samples</a>
  <ul>
  <li><a href="#sec-UMAP-unfiltered" id="toc-sec-UMAP-unfiltered" class="nav-link" data-scroll-target="#sec-UMAP-unfiltered"><span class="header-section-number">1.2.1</span> General Overview of the Dataset</a></li>
  <li><a href="#cohort-filtering" id="toc-cohort-filtering" class="nav-link" data-scroll-target="#cohort-filtering"><span class="header-section-number">1.2.2</span> Cohort filtering</a></li>
  </ul></li>
  <li><a href="#contingency-tables" id="toc-contingency-tables" class="nav-link" data-scroll-target="#contingency-tables"><span class="header-section-number">1.3</span> Contingency tables</a>
  <ul>
  <li><a href="#sex" id="toc-sex" class="nav-link" data-scroll-target="#sex"><span class="header-section-number">1.3.1</span> Sex</a></li>
  <li><a href="#disease-status" id="toc-disease-status" class="nav-link" data-scroll-target="#disease-status"><span class="header-section-number">1.3.2</span> Disease status</a></li>
  <li><a href="#tumor-stages" id="toc-tumor-stages" class="nav-link" data-scroll-target="#tumor-stages"><span class="header-section-number">1.3.3</span> Tumor Stages</a></li>
  <li><a href="#tumor-histology" id="toc-tumor-histology" class="nav-link" data-scroll-target="#tumor-histology"><span class="header-section-number">1.3.4</span> Tumor Histology</a></li>
  <li><a href="#tmn-t-main-tumor-pathology" id="toc-tmn-t-main-tumor-pathology" class="nav-link" data-scroll-target="#tmn-t-main-tumor-pathology"><span class="header-section-number">1.3.5</span> TMN, T: Main Tumor Pathology</a></li>
  <li><a href="#tmn-n-lymph-nodes-metastatis" id="toc-tmn-n-lymph-nodes-metastatis" class="nav-link" data-scroll-target="#tmn-n-lymph-nodes-metastatis"><span class="header-section-number">1.3.6</span> TMN, N: Lymph Nodes metastatis</a></li>
  <li><a href="#sec-dataset-exploration" id="toc-sec-dataset-exploration" class="nav-link" data-scroll-target="#sec-dataset-exploration"><span class="header-section-number">1.3.7</span> Prognosis Outcome</a></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-dataset-exploration" id="toc-sec-lesson-learnt-dataset-exploration" class="nav-link" data-scroll-target="#sec-lesson-learnt-dataset-exploration"><span class="header-section-number">1.4</span> Lessons Learnt</a></li>
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information"><span class="header-section-number">1.5</span> Session Information</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./dataset-exploration.html"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="1.1">
<h2 data-number="1.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">1.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at Lesson Learnt: <a href="#sec-lesson-learnt-dataset-exploration" class="quarto-xref"><span>Section 1.4</span></a>.</p>
<ul>
<li>Here we focus on samples from Kidney Cancers patients and their overall relationships.</li>
<li>We then explore the association between each Kidney Cancer subtype and the major clinical covariates (e.g.: sex, disease status, cancer stage, clinical outcome).</li>
</ul>
</section>
<section id="selection-of-kidney-samples" class="level2" data-number="1.2">
<h2 data-number="1.2" class="anchored" data-anchor-id="selection-of-kidney-samples"><span class="header-section-number">1.2</span> Selection of Kidney samples</h2>
<p>For the next sections, we will focus on kidney cancer. The choice is simply dictated by the abundance of samples which provide us a large playground to test different strategies for biomarker discovery, higher statistical power, and the existance of some <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-022-01105-y">Molecular Subtypes gene signatures</a> that we can use as a benchmark and we can try to improve upon.</p>
<p>Let’s select and explore the samples available for Kidney tissue.</p>
<section id="sec-UMAP-unfiltered" class="level3" data-number="1.2.1">
<h3 data-number="1.2.1" class="anchored" data-anchor-id="sec-UMAP-unfiltered"><span class="header-section-number">1.2.1</span> General Overview of the Dataset</h3>
<p>To have a first overview of all the Kidney samples, we will plot a UMAP based on the gene expression of all the biopsys. A <a href="https://umap-learn.readthedocs.io/en/latest/how_umap_works.html">UMAP</a> is an algorithm for dimension reduction based on manifold learning techniques and ideas from topological data analysis. The <a href="https://arxiv.org/abs/1802.03426">UMAP algorithm</a> first constructs a weighted graph of the data in input (with N-dimensions), where the weight in the graph edge corresponds to how similar is a point (i.e.: sample) to the next one, and then projects this similarity graph into a bi-dimensional space. The 2-D projection is optimized to faithfully represents the relationships between the points in the N-dimensional space.</p>
<p>To build the UMAP, we focus on the 1,000 top most variable genes between the transcriptomics samples. This allows us to maximize similarities and differences between sample groups (if any).</p>
<p>The UMAP of all the samples divides the biopsys into 4 major clusters. Three clusters are localized on the upper part of the UMAP, and they loosely correspond to the three cancer subtypes in this cohort: kidney chromophobe (<a href="https://www.cancer.gov/ccg/research/genome-sequencing/tcga/studied-cancers/chromophobe-renal-cell-carcinoma-study">KICH</a>), kidney renal papillary cell carcinoma (<a href="https://www.cancer.gov/pediatric-adult-rare-tumor/rare-tumors/rare-kidney-tumors/papillary-renal-cell-carcinoma">KIRP</a>), and kidney renal clear cell carcinoma (<a href="https://www.cancer.gov/pediatric-adult-rare-tumor/rare-tumors/rare-kidney-tumors/clear-cell-renal-cell-carcinoma">KIRC</a>). A further separation of the samples is visible on the y axis: biopsys from primary tumor are localized on top of the UMAP, while a second cluster of normal solid tissue are localized in the bottom of the UMAP, containing samples from non-diseased tissues of patients of all of the three cancer types.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 1: Kidney UMAP - unfiltered data.</figcaption>
</figure>
</div>
</div>
</div>
<p>Since we want to focus on discovering new biomarkers for the three kidney cancer types, we will remove the biopsys from normal solid tissue. Then, we test if there is a strong imbalance between the sex and age of the subjects in the three kidney cancer cohorts, and wheter we should balance the cohorts or not.</p>
</section>
<section id="cohort-filtering" class="level3" data-number="1.2.2">
<h3 data-number="1.2.2" class="anchored" data-anchor-id="cohort-filtering"><span class="header-section-number">1.2.2</span> Cohort filtering</h3>
<p>First, we retain only biopsys from “Primary Tumor”, removing “Solid Tissue Normal” and “Additional - New Primary” neoplasis. Then we look at the distribution of age, sex and ethnicity in the three different cancer cohorts.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 2: Kidney Cancer: demographic overview.</figcaption>
</figure>
</div>
</div>
</div>
<p>We can immediatly see that the sampling biase towards patients with white background observed in all the cancer cohorts in <a href="gather-datasets-and-QC.html#sec-All-cancers-cohort-demographic" class="quarto-xref"><span>Section 3.1</span></a> is reflected as well in kidney cancer cohorts. In terms of geneder demographics, males represents from 60% (KICH) to 74% (KIRP) of the cohorts. The histograms split by cancer types, however, show that the age distributions between the sexs int he different kidney cancer types are similar.</p>
<p>Since we may loose too many samples if we would generate balanced cohorts in terms of sex and race, we can keep the samples as is, and take Age, Sex and Race in account during the modeling phases downstream.</p>
</section>
</section>
<section id="contingency-tables" class="level2" data-number="1.3">
<h2 data-number="1.3" class="anchored" data-anchor-id="contingency-tables"><span class="header-section-number">1.3</span> Contingency tables</h2>
<p>As we saw previously TCGA provides rich clinical metadata for each patient. We can run some simple demographic statistics (chi-squared and Fisher exact tests) to see if any clinical covariate is significantly overrepresented in any of the three cancer types.</p>
<p>We can test if there is a significant difference of each observed clinical covariate across the three kidney cancer types. The null hypothesis here is that the distribution of each covariate in the the three cancer subtypes is the same.</p>
<section id="sex" class="level3" data-number="1.3.1">
<h3 data-number="1.3.1" class="anchored" data-anchor-id="sex"><span class="header-section-number">1.3.1</span> Sex</h3>
<p>There are 596 males and 291 females in the three kidney cancer cohorts after filtering. Despite the imbalance, the difference across cancer types is not significant.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-fa34d8e27d962b85c692" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-fa34d8e27d962b85c692">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 1: Contigency table, Sex.<\/caption>","data":[["KICH","KIRC","KIRP"],[27,188,76],[39,343,214]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>female<\/th>\n      <th>male<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"female","targets":1},{"name":"male","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 3: Contigency table, Sex.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="disease-status" class="level3" data-number="1.3.2">
<h3 data-number="1.3.2" class="anchored" data-anchor-id="disease-status"><span class="header-section-number">1.3.2</span> Disease status</h3>
<p>After filtering, the three kidney cancer types contains 563 subject that are now tumor free, and 190 subject that still have tumor. It seems that KIRP have significantly more patients that are now tumor free, while there are still significantly more patients with tumor for KIRC. The small sample size of KICH patients may skew these results, so this outcome should be interpreted with a pinch of salt.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-f70bee21871d1a39a1c3" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f70bee21871d1a39a1c3">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 2: Contigency table, Disease Status.<\/caption>","data":[["KICH","KIRC","KIRP"],[37,327,199],[8,148,34]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>TUMOR FREE<\/th>\n      <th>WITH TUMOR<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"TUMOR FREE","targets":1},{"name":"WITH TUMOR","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 4: Contigency table, Disease Status.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="tumor-stages" class="level3" data-number="1.3.3">
<h3 data-number="1.3.3" class="anchored" data-anchor-id="tumor-stages"><span class="header-section-number">1.3.3</span> Tumor Stages</h3>
<p>The tumors across the patients were scored as: 458 Stage I, 105 Stage II, 189 Stage III and 105 Stage IV tumors.</p>
<p>This staging system is less granular than the <a href="https://www.cancer.gov/about-cancer/diagnosis-staging/staging">TNM Staging System</a> (that we look at into below), and it describe the tumro as follow:</p>
<ul>
<li>Stage 0 = Abnormal cells are present but have not spread to nearby tissue. Also called carcinoma in situ, or CIS. CIS is not cancer, but it may become cancer.</li>
<li>Stage I, Stage II, and Stage III = Cancer is present. The higher the number, the larger the cancer tumor and the more it has spread into nearby tissues.</li>
<li>Stage IV = The cancer has spread to distant parts of the body.</li>
</ul>
<p>The higher the number, the more advanced the cancer is.</p>
<p>KIRC patients seem to have a higher number of Stage IV cancer, while KIRP patients have more STage I cancers. This can either be bias in the sampling of the cohort, or it could really indicate that KIRC cancer is more aggressive and fast progressing than KIRP.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-089e0c4e3f692cc64c3e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-089e0c4e3f692cc64c3e">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 3: Contigency table, Tumor Stage.<\/caption>","data":[["KICH","KIRC","KIRP"],[21,266,171],[25,57,23],[14,123,52],[6,84,15]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Stage I<\/th>\n      <th>Stage II<\/th>\n      <th>Stage III<\/th>\n      <th>Stage IV<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Stage I","targets":1},{"name":"Stage II","targets":2},{"name":"Stage III","targets":3},{"name":"Stage IV","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 5: Contigency table, Tumor Stage.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="tumor-histology" class="level3" data-number="1.3.4">
<h3 data-number="1.3.4" class="anchored" data-anchor-id="tumor-histology"><span class="header-section-number">1.3.4</span> Tumor Histology</h3>
<p>Histological evaluation of the tumor cells essential to allow for accurate prognostication and tailored therapy, both surgical and adjuvant.</p>
<p>Histological grade have five categories:</p>
<ul>
<li>G1 = Well differentiated</li>
<li>G2 = Moderately differentiated</li>
<li>G3 = Poorly differentiated, Undifferentiated, Anaplastic</li>
<li>G4 = Undifferentiated (high grade)</li>
<li>GX = Grade cannot be assessed</li>
</ul>
<p>Unfortunately, within the TCGA kidney cohorts, only KIRC biopsys where graded histologically. No histological grading was available for KICH and KIRP tumors.</p>
<p>For the KIRC tumors, 14 biopsys had G1 grade, 229 G2 grade, 204 had G3 grade, 76 G4 and 5 biopsys could not be graded (GX).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-db3d197e94d58e4340f3" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-db3d197e94d58e4340f3">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 4: Contigency table, Tumor Histology.<\/caption>","data":[["","G1","G2","G3","G4","GX"],["66","0","0","0","0","0"],["3","14","229","204","76","5"],["290","0","0","0","0","0"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>KICH<\/th>\n      <th>KIRC<\/th>\n      <th>KIRP<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"KICH","targets":1},{"name":"KIRC","targets":2},{"name":"KIRP","targets":3}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 6: Contigency table, Tumor Histology.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="tmn-t-main-tumor-pathology" class="level3" data-number="1.3.5">
<h3 data-number="1.3.5" class="anchored" data-anchor-id="tmn-t-main-tumor-pathology"><span class="header-section-number">1.3.5</span> TMN, T: Main Tumor Pathology</h3>
<p>Let’s look now into the TMN staging system. We have multiple staging and substaging that describes the tumor presence, dimension and invasion of adjacent tissues:</p>
<ul>
<li>TX = Primary tumor cannot be assessed.</li>
<li>T0 = No evidence of primary tumor.</li>
<li>T1 = Tumor ≤2 cm in greatest dimension.</li>
<li>T2 = Tumor &gt;2 cm but ≤4 cm.</li>
<li>T3 = Tumor &gt;4 cm or with subserosal invasion or involvement of the mesoappendix.</li>
<li>T4 = Tumor perforates the peritoneum or directly invades other adjacent organs or structures (excluding direct mural extension to adjacent subserosa of adjacent bowel), e.g., abdominal wall and skeletal muscle.</li>
</ul>
<p>The presence of 12 distinct substaging for TMN, and the relative small sample size across the cohort makes the outcome of this comparison difficult to judge. We could claim that KIRP seems to be enriched in lower tumor stages (T1a) and depleted in higher tumor stages (T2, T3a, T3b) than KIRC.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-f5c16b13a9d1ad8ff34a" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f5c16b13a9d1ad8ff34a">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 5: Contigency table, TMN Main Tumor.<\/caption>","data":[["KICH","KIRC","KIRP"],[5,21,23],[5,139,107],[11,111,62],[19,55,18],[0,10,9],[6,4,5],[1,5,9],[14,120,39],[3,52,11],[0,2,1],[2,11,2],[0,0,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>T1<\/th>\n      <th>T1a<\/th>\n      <th>T1b<\/th>\n      <th>T2<\/th>\n      <th>T2a<\/th>\n      <th>T2b<\/th>\n      <th>T3<\/th>\n      <th>T3a<\/th>\n      <th>T3b<\/th>\n      <th>T3c<\/th>\n      <th>T4<\/th>\n      <th>TX<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6,7,8,9,10,11,12]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"T1","targets":1},{"name":"T1a","targets":2},{"name":"T1b","targets":3},{"name":"T2","targets":4},{"name":"T2a","targets":5},{"name":"T2b","targets":6},{"name":"T3","targets":7},{"name":"T3a","targets":8},{"name":"T3b","targets":9},{"name":"T3c","targets":10},{"name":"T4","targets":11},{"name":"TX","targets":12}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-17-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 7: Contigency table, TMN Main Tumor.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="tmn-n-lymph-nodes-metastatis" class="level3" data-number="1.3.6">
<h3 data-number="1.3.6" class="anchored" data-anchor-id="tmn-n-lymph-nodes-metastatis"><span class="header-section-number">1.3.6</span> TMN, N: Lymph Nodes metastatis</h3>
<p>The TMN tumor staging system provides as well a score for the lymph nodes:</p>
<ul>
<li>NX = Regional lymph nodes cannot be assessed.</li>
<li>N0 = No regional lymph node metastasis.</li>
<li>N1 = Regional lymph node metastasis &lt;12 nodes.</li>
<li>N2 = Large mesenteric masses (&gt;2 cm) and/or extensive nodal deposits (≥12), especially those that encase the superior mesenteric vessels.</li>
</ul>
<p>In the kidney cohort, lymph nodes of 506 patiens could not be scored (NX), 328 patients had a N0 score, 43 N1 and 7 had a N2 score. Lymph nodes of most patients with KIRP tumors could not be scored, they they were depleted in No and enriched in N1 if compared with patients with KIRC tumor.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-a8cfd547327390c958c5" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-a8cfd547327390c958c5">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 6: Contigency table, TMN Lymph nodes.<\/caption>","data":[["KICH","KIRC","KIRP"],[40,239,49],[3,16,24],[2,0,5],[21,275,210]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>N0<\/th>\n      <th>N1<\/th>\n      <th>N2<\/th>\n      <th>NX<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"N0","targets":1},{"name":"N1","targets":2},{"name":"N2","targets":3},{"name":"NX","targets":4}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-19-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 8: Contigency table, TMN Lymph nodes.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="sec-dataset-exploration" class="level3" data-number="1.3.7">
<h3 data-number="1.3.7" class="anchored" data-anchor-id="sec-dataset-exploration"><span class="header-section-number">1.3.7</span> Prognosis Outcome</h3>
<p>Of the 887 petients with kidney tumor, 659 were alive at the time of the last check up, while 228 died. It appears that KIRC patients (48.7% dead) have worse prognosis than KIRP patients (17.9% dead), as observed in the previous comparisons. However, before drawing a formal conclusion, we should validate this observation at least in another independent cohort.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-c874d1357a0dd3061044" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c874d1357a0dd3061044">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 7: Contigency table, Prognosis Outcome.<\/caption>","data":[["KICH","KIRC","KIRP"],[56,357,246],[10,174,44]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Alive<\/th>\n      <th>Dead<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Alive","targets":1},{"name":"Dead","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="dataset-exploration_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 9: Contigency table, Vital Status.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="sec-lesson-learnt-dataset-exploration" class="level2" data-number="1.4">
<h2 data-number="1.4" class="anchored" data-anchor-id="sec-lesson-learnt-dataset-exploration"><span class="header-section-number">1.4</span> Lessons Learnt</h2>
<p>So far, we have learnt:</p>
<ul>
<li>A UMAP is a handy way to summarize relationship between data:
<ul>
<li>Projecting the transcriptomics data from kidney cancer biopsies with a UMAP showed that samples are grouped in three clusters, loosely corresponding to the three kidney cancer types (KICH, KIRC, KIRP).</li>
<li>Some samples seem to be misclassified and clustered in the wrong cancer type group, or there are additional nuances that were not classified with bulk RNA-seq.</li>
</ul></li>
<li>Contingency tables reports on large clinical trends between the three cancer types:
<ul>
<li>KIRC patients seems to have a higher chance to be tumor free than KIRP patients.</li>
<li>Despite that, KIRC patients seems to be enriched for Stage IV tumors and depleted in Stage I tumors when compared to KIRP patients.</li>
<li>Likewise, KIRP patients have more T1a tumor stages and less T2, T3a and T3b tumours when compared to KIRC.</li>
<li>Unfortunately, KIRC patients seems to had a worse outcome than KIRP patients.</li>
</ul></li>
<li>The clinical outcome for KIRC patients seems to be more polarized: or the patient were cancer free at the end of the study, or the patient had more advanced tumour or a worse outcome when compared to KIRP patients. This overview suggest the possibility that if we could develop biomarkers for early detection of KIRC cancers we could improve the clinical outcome for these patients.</li>
</ul>
</section>
<section id="session-information" class="level2" data-number="1.5">
<h2 data-number="1.5" class="anchored" data-anchor-id="session-information"><span class="header-section-number">1.5</span> Session Information</h2>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: openSUSE Tumbleweed

Matrix products: default
BLAS/LAPACK: /home/andrea/miniforge3/envs/moai/lib/libmkl_rt.so.2;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=it_IT.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Brussels
tzcode source: system (glibc)

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] vcd_1.4-13         umap_0.2.10.0      stringr_1.5.1      scales_1.3.0      
 [5] RColorBrewer_1.1-3 matrixStats_1.5.0  gridExtra_2.3      ggplot2_3.5.1     
 [9] forcats_1.0.0      edgeR_4.0.16       limma_3.58.1       DT_0.33           
[13] dplyr_1.1.4        data.table_1.16.4  cowplot_1.1.3     

loaded via a namespace (and not attached):
 [1] sass_0.4.9          tidyr_1.3.1         generics_0.1.3     
 [4] stringi_1.8.4       lattice_0.22-6      digest_0.6.37      
 [7] magrittr_2.0.3      evaluate_1.0.3      fastmap_1.2.0      
[10] jsonlite_1.8.9      Matrix_1.6-5        RSpectra_0.16-2    
[13] purrr_1.0.2         crosstalk_1.2.1     jquerylib_0.1.4    
[16] cli_3.6.3           rlang_1.1.5         munsell_0.5.1      
[19] cachem_1.1.0        withr_3.0.2         yaml_2.3.10        
[22] tools_4.3.2         colorspace_2.1-1    locfit_1.5-9.10    
[25] BiocGenerics_0.48.1 reticulate_1.40.0   vctrs_0.6.5        
[28] R6_2.5.1            png_0.1-8           zoo_1.8-12         
[31] lifecycle_1.0.4     htmlwidgets_1.6.4   MASS_7.3-60.0.1    
[34] pkgconfig_2.0.3     bslib_0.8.0         pillar_1.10.1      
[37] gtable_0.3.6        glue_1.8.0          Rcpp_1.0.14        
[40] statmod_1.5.0       xfun_0.50           tibble_3.2.1       
[43] lmtest_0.9-40       tidyselect_1.2.1    knitr_1.49         
[46] farver_2.1.2        htmltools_0.5.8.1   labeling_0.4.3     
[49] rmarkdown_2.29      compiler_4.3.2      askpass_1.2.1      
[52] openssl_2.3.1      </code></pre>
</div>
</div>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./gather-datasets-and-QC.html" class="pagination-link" aria-label="Gather Datasets and QC">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">Gather Datasets and QC</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./transcriptomics-analysis.html" class="pagination-link" aria-label="Transcriptomics Analysis">
        <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>