<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>8&nbsp; Characterization of Factor 3 (associated with worse outcome) – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./kidney-carcinomas-lessons-learnt.html" rel="next">
<link href="./survival-analysis.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./factor-3-worse-outcome.html"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Characterization of Factor 3 (associated with worse outcome)</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./survival-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-3-worse-outcome.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Characterization of Factor 3 (associated with worse outcome)</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./kidney-carcinomas-lessons-learnt.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Characterization of Kidney Carcinomas: Take Home Message</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <span class="sidebar-item-text sidebar-link text-start">
 <span class="menu-text">Discovery of Prognostic Biomarkers in Kidney Carcinomas</span></span>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <span class="sidebar-item-text sidebar-link text-start">
 <span class="menu-text">Therapeutic Target Discovery in Kidney Clear Cell Cancer</span></span>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">8.1</span> On this page</a></li>
  <li><a href="#factor-3-higher-hazard-ratio" id="toc-factor-3-higher-hazard-ratio" class="nav-link" data-scroll-target="#factor-3-higher-hazard-ratio"><span class="header-section-number">8.2</span> Factor 3: Higher Hazard Ratio</a>
  <ul>
  <li><a href="#multi-omics-signature" id="toc-multi-omics-signature" class="nav-link" data-scroll-target="#multi-omics-signature"><span class="header-section-number">8.2.1</span> Multi-omics signature</a>
  <ul>
  <li><a href="#multi-omics-interaction-network" id="toc-multi-omics-interaction-network" class="nav-link" data-scroll-target="#multi-omics-interaction-network"><span class="header-section-number">8.2.1.1</span> Multi-omics interaction network</a></li>
  <li><a href="#pathway-analysis" id="toc-pathway-analysis" class="nav-link" data-scroll-target="#pathway-analysis"><span class="header-section-number">8.2.1.2</span> Pathway analysis</a></li>
  </ul></li>
  <li><a href="#cnvs-on-factor3-genes" id="toc-cnvs-on-factor3-genes" class="nav-link" data-scroll-target="#cnvs-on-factor3-genes"><span class="header-section-number">8.2.2</span> CNVs on Factor3 genes</a>
  <ul>
  <li><a href="#cnvs-signatures-associated-to-factor3" id="toc-cnvs-signatures-associated-to-factor3" class="nav-link" data-scroll-target="#cnvs-signatures-associated-to-factor3"><span class="header-section-number">8.2.2.1</span> CNVs signatures associated to Factor3</a></li>
  </ul></li>
  <li><a href="#somatic-mutations-on-factor3-genes" id="toc-somatic-mutations-on-factor3-genes" class="nav-link" data-scroll-target="#somatic-mutations-on-factor3-genes"><span class="header-section-number">8.2.3</span> Somatic Mutations on Factor3 genes</a>
  <ul>
  <li><a href="#mutational-signatures-associated-to-factor3" id="toc-mutational-signatures-associated-to-factor3" class="nav-link" data-scroll-target="#mutational-signatures-associated-to-factor3"><span class="header-section-number">8.2.3.1</span> Mutational signatures associated to Factor3</a></li>
  </ul></li>
  <li><a href="#mutations-in-a-manhattan-plots" id="toc-mutations-in-a-manhattan-plots" class="nav-link" data-scroll-target="#mutations-in-a-manhattan-plots"><span class="header-section-number">8.2.4</span> MUTATIONS IN A MANHATTAN PLOTS</a></li>
  <li><a href="#survival" id="toc-survival" class="nav-link" data-scroll-target="#survival"><span class="header-section-number">8.2.5</span> survival</a></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-worse-outcome" id="toc-sec-lesson-learnt-worse-outcome" class="nav-link" data-scroll-target="#sec-lesson-learnt-worse-outcome"><span class="header-section-number">8.3</span> Lessons Learnt</a></li>
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information"><span class="header-section-number">8.4</span> Session Information</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./factor-3-worse-outcome.html"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Characterization of Factor 3 (associated with worse outcome)</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Characterization of Factor 3 (associated with worse outcome)</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="8.1">
<h2 data-number="8.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">8.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at section Lesson Learnt: <a href="#sec-lesson-learnt-worse-outcome" class="quarto-xref"><span>Section 8.3</span></a>.</p>
<ul>
<li>He</li>
</ul>
</section>
<section id="factor-3-higher-hazard-ratio" class="level2" data-number="8.2">
<h2 data-number="8.2" class="anchored" data-anchor-id="factor-3-higher-hazard-ratio"><span class="header-section-number">8.2</span> Factor 3: Higher Hazard Ratio</h2>
<section id="multi-omics-signature" class="level3" data-number="8.2.1">
<h3 data-number="8.2.1" class="anchored" data-anchor-id="multi-omics-signature"><span class="header-section-number">8.2.1</span> Multi-omics signature</h3>
<p>As we mentioned before, each factor captures a different dimension of heterogeneity in the data. Mathematically, each factor ordinates the samples along a one-dimensional axis centered at zero. Samples with different signs manifest opposite phenotypes along the inferred axis of variation, with higher absolute value indicating a stronger effect. Interpretation of the factor analysis and the relative position of the samples across each factor is similar to the interpretation of the principal components in a PCA.</p>
<p>Indeed, we can confirm that KICH, KIRC and KIRP are separated along the axis of Factor 1 and Factor 2, as observed in the correlation between the hidden factors and the clinical covariates. More in detail, Factor 1 seems to be a gradient that differentiate first KIRC, then KICH and finally KIRP along its axis. Factor 2, instead, strongly separates KICH from KIRC and KIRP. This result is very encouraging for characterizing molecularly the three kidney carcinoma types, and for inferring multi-omics signatures that discriminate them.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-3-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Let’s explore the contribution of each omics layer to Factor 3.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">Transcriptomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">micro-RNA</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">Poteomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false">Epigenomics</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-5-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>daadadad</p>
<section id="multi-omics-interaction-network" class="level4" data-number="8.2.1.1">
<h4 data-number="8.2.1.1" class="anchored" data-anchor-id="multi-omics-interaction-network"><span class="header-section-number">8.2.1.1</span> Multi-omics interaction network</h4>
<p>GET top molecules</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>many molecules stands out: mir-21 PAI-1 (known marker of worse prognosis) https://jcp.bmj.com/content/64/10/893</p>
<p>get Omnipath interactions</p>
<p>Let’s build the network now</p>
<p>We can also remove the PPI interactions retrieved from STRING, and use only a gene interaction networks reconstructed from the OmniPath interaction networks. This may help to generate a simplified, more readable gene network.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>ss</p>
</section>
<section id="pathway-analysis" class="level4" data-number="8.2.1.2">
<h4 data-number="8.2.1.2" class="anchored" data-anchor-id="pathway-analysis"><span class="header-section-number">8.2.1.2</span> Pathway analysis</h4>
<p>prep gene list</p>
<p>Some relevant biological processes</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">GO BP ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">GO BP genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">KEGG ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-4" role="tab" aria-controls="tabset-2-4" aria-selected="false">KEGG genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-5" role="tab" aria-controls="tabset-2-5" aria-selected="false">Reactome ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-6" role="tab" aria-controls="tabset-2-6" aria-selected="false">Reactome genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-7-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-7" role="tab" aria-controls="tabset-2-7" aria-selected="false">WikiPathways ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-8-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-8" role="tab" aria-controls="tabset-2-8" aria-selected="false">WikiPathways heatplot</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-19-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 23: Heatplot of enriched GO Biological Process terms in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>KEGG metabolic is underrepresented with PKLR LDHD FMO1 DDC GBA3 MGAM SAT1 ALDOB OGDHL HSD11B2 NAT8 HAO2 MIOX ALDH1L1 PCK1</p>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 16: Dotplot of enriched KEGG pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-4-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-22-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 17: Heatplot of enriched KEGG pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-5-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-23-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 18: Dotplot of enriched Reactome pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-6-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-24-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 19: Heatplot of enriched Reactome pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-7" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-7-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-25-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 20: Dotplot of enriched WikiPathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-8" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-8-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-26-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 21: Heatplot of enriched WikiPathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</section>
<section id="cnvs-on-factor3-genes" class="level3" data-number="8.2.2">
<h3 data-number="8.2.2" class="anchored" data-anchor-id="cnvs-on-factor3-genes"><span class="header-section-number">8.2.2</span> CNVs on Factor3 genes</h3>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-28-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<section id="cnvs-signatures-associated-to-factor3" class="level4" data-number="8.2.2.1">
<h4 data-number="8.2.2.1" class="anchored" data-anchor-id="cnvs-signatures-associated-to-factor3"><span class="header-section-number">8.2.2.1</span> CNVs signatures associated to Factor3</h4>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-30-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="somatic-mutations-on-factor3-genes" class="level3" data-number="8.2.3">
<h3 data-number="8.2.3" class="anchored" data-anchor-id="somatic-mutations-on-factor3-genes"><span class="header-section-number">8.2.3</span> Somatic Mutations on Factor3 genes</h3>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-32-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
<section id="mutational-signatures-associated-to-factor3" class="level4" data-number="8.2.3.1">
<h4 data-number="8.2.3.1" class="anchored" data-anchor-id="mutational-signatures-associated-to-factor3"><span class="header-section-number">8.2.3.1</span> Mutational signatures associated to Factor3</h4>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-33-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="mutations-in-a-manhattan-plots" class="level3" data-number="8.2.4">
<h3 data-number="8.2.4" class="anchored" data-anchor-id="mutations-in-a-manhattan-plots"><span class="header-section-number">8.2.4</span> MUTATIONS IN A MANHATTAN PLOTS</h3>
<p>which mutation is enriched in the Factor3????</p>
</section>
<section id="survival" class="level3" data-number="8.2.5">
<h3 data-number="8.2.5" class="anchored" data-anchor-id="survival"><span class="header-section-number">8.2.5</span> survival</h3>
<p>RESIDUAL DISEASE == PEOPLE DIES</p>
<p>Can we develop prognostic markers out of it? Can we identify additional therapeutic targets that could improve this outcome???</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-34-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Contigency table, Vital Status.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-35-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Contigency table, Vital Status.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-36-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Contigency table, Vital Status.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-37-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Contigency table, Vital Status.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-3-worse-outcome_files/figure-html/unnamed-chunk-38-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Contigency table, Vital Status.</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="sec-lesson-learnt-worse-outcome" class="level2" data-number="8.3">
<h2 data-number="8.3" class="anchored" data-anchor-id="sec-lesson-learnt-worse-outcome"><span class="header-section-number">8.3</span> Lessons Learnt</h2>
<p>[[[[PROPER DESCRIPTION OF FINDINGS AND TRANSCRIPTOMICS ENRICHMENTS]]]]</p>
<p>So far, we have learnt:</p>
<ul>
<li>A</li>
</ul>
</section>
<section id="session-information" class="level2" data-number="8.4">
<h2 data-number="8.4" class="anchored" data-anchor-id="session-information"><span class="header-section-number">8.4</span> Session Information</h2>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: openSUSE Tumbleweed

Matrix products: default
BLAS/LAPACK: /home/andrea/miniforge3/envs/moai/lib/libmkl_rt.so.2;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=it_IT.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Brussels
tzcode source: system (glibc)

attached base packages:
 [1] parallel  grid      stats4    stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] vcd_1.4-13                                         
 [2] UpSetR_1.4.0                                       
 [3] umap_0.2.10.0                                      
 [4] stringr_1.5.1                                      
 [5] scales_1.3.0                                       
 [6] RColorBrewer_1.1-3                                 
 [7] qs_0.27.2                                          
 [8] PCAtools_2.14.0                                    
 [9] missMethyl_1.36.0                                  
[10] IlluminaHumanMethylationEPICanno.ilm10b4.hg19_0.6.0
[11] mCSEA_1.22.0                                       
[12] Homo.sapiens_1.3.1                                 
[13] TxDb.Hsapiens.UCSC.hg19.knownGene_3.2.2            
[14] GO.db_3.18.0                                       
[15] OrganismDbi_1.44.0                                 
[16] GenomicFeatures_1.54.4                             
[17] mCSEAdata_1.22.0                                   
[18] org.Hs.eg.db_3.18.0                                
[19] IlluminaHumanMethylation450kanno.ilmn12.hg19_0.6.1 
[20] minfi_1.48.0                                       
[21] bumphunter_1.44.0                                  
[22] locfit_1.5-9.10                                    
[23] iterators_1.0.14                                   
[24] foreach_1.5.2                                      
[25] Biostrings_2.70.3                                  
[26] XVector_0.42.0                                     
[27] SummarizedExperiment_1.32.0                        
[28] MatrixGenerics_1.14.0                              
[29] matrixStats_1.5.0                                  
[30] Gviz_1.46.1                                        
[31] gridExtra_2.3                                      
[32] ggraph_2.2.1                                       
[33] GenomicRanges_1.54.1                               
[34] GenomeInfoDb_1.38.8                                
[35] forcats_1.0.0                                      
[36] EnhancedVolcano_1.20.0                             
[37] ggrepel_0.9.6                                      
[38] ggplot2_3.5.1                                      
[39] edgeR_4.0.16                                       
[40] limma_3.58.1                                       
[41] DT_0.33                                            
[42] dplyr_1.1.4                                        
[43] DOSE_3.28.2                                        
[44] DMRcate_2.16.1                                     
[45] data.table_1.16.4                                  
[46] cowplot_1.1.3                                      
[47] clusterProfiler_4.10.1                             
[48] circlize_0.4.16                                    
[49] BiocSingular_1.18.0                                
[50] BiocParallel_1.36.0                                
[51] AnnotationDbi_1.64.1                               
[52] IRanges_2.36.0                                     
[53] S4Vectors_0.40.2                                   
[54] Biobase_2.62.0                                     
[55] BiocGenerics_0.48.1                                

loaded via a namespace (and not attached):
  [1] igraph_2.1.4                  graph_1.80.0                 
  [3] Formula_1.2-5                 zlibbioc_1.48.2              
  [5] tidyselect_1.2.1              rvest_1.0.4                  
  [7] bit_4.5.0.1                   doParallel_1.0.17            
  [9] clue_0.3-66                   lattice_0.22-6               
 [11] rjson_0.2.23                  nor1mix_1.3-3                
 [13] blob_1.2.4                    rngtools_1.5.2               
 [15] S4Arrays_1.2.1                base64_2.0.2                 
 [17] dichromat_2.0-0.1             scrime_1.3.5                 
 [19] png_0.1-8                     cli_3.6.3                    
 [21] ggplotify_0.1.2               ProtGenerics_1.34.0          
 [23] askpass_1.2.1                 multtest_2.58.0              
 [25] openssl_2.3.1                 BiocIO_1.12.0                
 [27] purrr_1.0.2                   basilisk.utils_1.14.1        
 [29] uwot_0.2.2                    shadowtext_0.1.4             
 [31] curl_6.2.0                    mime_0.12                    
 [33] evaluate_1.0.3                tidytree_0.4.6               
 [35] ComplexHeatmap_2.18.0         stringi_1.8.4                
 [37] backports_1.5.0               XML_3.99-0.18                
 [39] lubridate_1.9.4               httpuv_1.6.15                
 [41] magrittr_2.0.3                OmnipathR_3.10.1             
 [43] rappdirs_0.3.3                splines_4.3.2                
 [45] mclust_6.1.1                  RApiSerialize_0.1.4          
 [47] KMsurv_0.1-5                  jpeg_0.1-10                  
 [49] doRNG_1.8.6.1                 survminer_0.5.0              
 [51] logger_0.4.0                  corrplot_0.95                
 [53] bsseq_1.38.0                  DBI_1.2.3                    
 [55] HDF5Array_1.30.1              reactome.db_1.86.2           
 [57] genefilter_1.84.0             withr_3.0.2                  
 [59] enrichplot_1.22.0             lmtest_0.9-40                
 [61] RBGL_1.78.0                   tidygraph_1.3.1              
 [63] rtracklayer_1.62.0            BiocManager_1.30.25          
 [65] htmlwidgets_1.6.4             fs_1.6.5                     
 [67] biomaRt_2.58.2                labeling_0.4.3               
 [69] SparseArray_1.2.4             cellranger_1.1.0             
 [71] annotate_1.80.0               reticulate_1.40.0            
 [73] VariantAnnotation_1.48.1      zoo_1.8-12                   
 [75] knitr_1.49                    beanplot_1.3.1               
 [77] timechange_0.3.0              patchwork_1.3.0              
 [79] ggtree_3.10.1                 rhdf5_2.46.1                 
 [81] R.oo_1.27.0                   RSpectra_0.16-2              
 [83] irlba_2.3.5.1                 gridGraphics_0.5-1           
 [85] lazyeval_0.2.2                yaml_2.3.10                  
 [87] maxstat_0.7-25                survival_3.8-3               
 [89] BiocVersion_3.18.1            crayon_1.5.3                 
 [91] tidyr_1.3.1                   tweenr_2.0.3                 
 [93] later_1.4.1                   codetools_0.2-20             
 [95] base64enc_0.1-3               GlobalOptions_0.1.2          
 [97] KEGGREST_1.42.0               Rtsne_0.17                   
 [99] shape_1.4.6.1                 ReactomePA_1.46.0            
[101] Rsamtools_2.18.0              filelock_1.0.3               
[103] foreign_0.8-88                pkgconfig_2.0.3              
[105] xml2_1.3.6                    ggpubr_0.6.0                 
[107] GenomicAlignments_1.38.2      aplot_0.2.4                  
[109] BSgenome_1.70.2               ape_5.8-1                    
[111] viridisLite_0.4.2             biovizBase_1.50.0            
[113] xtable_1.8-4                  interp_1.1-6                 
[115] car_3.1-3                     plyr_1.8.9                   
[117] httr_1.4.7                    tools_4.3.2                  
[119] htmlTable_2.4.3               broom_1.0.7                  
[121] checkmate_2.3.2               nlme_3.1-167                 
[123] selectr_0.4-2                 HDO.db_0.99.1                
[125] dbplyr_2.5.0                  ExperimentHub_2.10.0         
[127] survMisc_0.5.6                digest_0.6.37                
[129] permute_0.9-7                 Matrix_1.6-5                 
[131] dir.expiry_1.10.0             farver_2.1.2                 
[133] tzdb_0.4.0                    AnnotationFilter_1.26.0      
[135] reshape2_1.4.4                yulab.utils_0.2.0            
[137] exactRankTests_0.8-35         viridis_0.6.5                
[139] rpart_4.1.24                  glue_1.8.0                   
[141] cachem_1.1.0                  BiocFileCache_2.10.2         
[143] polyclip_1.10-7               Hmisc_5.2-2                  
[145] generics_0.1.3                mvtnorm_1.3-3                
[147] statmod_1.5.0                 ScaledMatrix_1.10.0          
[149] GEOquery_2.70.0               carData_3.0-5                
[151] vroom_1.6.5                   gson_0.1.0                   
[153] dqrng_0.4.1                   siggenes_1.76.0              
[155] basilisk_1.14.3               graphlayouts_1.2.2           
[157] gtools_3.9.5                  readxl_1.4.3                 
[159] preprocessCore_1.64.0         ggsignif_0.6.4               
[161] shiny_1.10.0                  GenomeInfoDbData_1.2.11      
[163] R.utils_2.12.3                rhdf5filters_1.14.1          
[165] RCurl_1.98-1.16               memoise_2.0.1                
[167] rmarkdown_2.29                pheatmap_1.0.12              
[169] R.methodsS3_1.8.2             reshape_0.8.9                
[171] Cairo_1.6-2                   illuminaio_0.44.0            
[173] stringfish_0.16.0             km.ci_0.5-6                  
[175] rstudioapi_0.17.1             cluster_2.1.8                
[177] hms_1.1.3                     munsell_0.5.1                
[179] colorspace_2.1-1              rlang_1.1.5                  
[181] quadprog_1.5-8                DelayedMatrixStats_1.24.0    
[183] sparseMatrixStats_1.14.0      ggforce_0.4.2                
[185] mgcv_1.9-1                    xfun_0.50                    
[187] abind_1.4-8                   GOSemSim_2.28.1              
[189] tibble_3.2.1                  interactiveDisplayBase_1.40.0
[191] treeio_1.26.0                 Rhdf5lib_1.24.2              
[193] readr_2.1.5                   bitops_1.0-9                 
[195] promises_1.3.2                scatterpie_0.2.4             
[197] RSQLite_2.3.9                 qvalue_2.34.0                
[199] fgsea_1.28.0                  DelayedArray_0.28.0          
[201] compiler_4.3.2                prettyunits_1.2.0            
[203] beachmat_2.18.1               graphite_1.48.0              
[205] Rcpp_1.0.14                   AnnotationHub_3.10.1         
[207] MASS_7.3-60.0.1               progress_1.2.3               
[209] R6_2.5.1                      fastmap_1.2.0                
[211] fastmatch_1.1-6               rstatix_0.7.2                
[213] ensembldb_2.26.0              rsvd_1.0.5                   
[215] nnet_7.3-20                   gtable_0.3.6                 
[217] latticeExtra_0.6-30           deldir_2.0-4                 
[219] htmltools_0.5.8.1             RcppParallel_5.1.10          
[221] bit64_4.6.0-1                 lifecycle_1.0.4              
[223] MOFA2_1.12.1                  restfulr_0.0.15              
[225] vctrs_0.6.5                   ggfun_0.1.8                  
[227] pillar_1.10.1                 magick_2.8.5                 
[229] jsonlite_1.8.9                GetoptLong_1.0.5             </code></pre>
</div>
</div>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script data-goatcounter="https://19adc99.goatcounter.com/count" async="" src="//gc.zgo.at/count.js"></script>
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./survival-analysis.html" class="pagination-link" aria-label="Survival Analysis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./kidney-carcinomas-lessons-learnt.html" class="pagination-link" aria-label="Characterization of Kidney Carcinomas: Take Home Message">
        <span class="nav-page-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Characterization of Kidney Carcinomas: Take Home Message</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>