<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>6&nbsp; Factor Analysis – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./survival-analysis.html" rel="next">
<link href="./genomics-and-epigenomics-analysis.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./factor-analysis.html"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./survival-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./kidney-carcinomas-lessons-learnt.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">kidney-carcinomas-lessons-learnt.html</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell Cancer molecular subtypes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cancer-signatures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">cancer-signatures.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./autoencoder.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">autoencoder.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cellular-deconvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Cellular Deconvolution from Bulk RNA-seq</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">6.1</span> On this page</a></li>
  <li><a href="#multi-omics-factor-analysis" id="toc-multi-omics-factor-analysis" class="nav-link" data-scroll-target="#multi-omics-factor-analysis"><span class="header-section-number">6.2</span> Multi-Omics Factor Analysis</a>
  <ul>
  <li><a href="#data-normalization" id="toc-data-normalization" class="nav-link" data-scroll-target="#data-normalization"><span class="header-section-number">6.2.1</span> Data normalization</a></li>
  <li><a href="#multi-omics-data-integration-and-analysis" id="toc-multi-omics-data-integration-and-analysis" class="nav-link" data-scroll-target="#multi-omics-data-integration-and-analysis"><span class="header-section-number">6.2.2</span> Multi-omics data integration and analysis</a></li>
  <li><a href="#sec-factor1" id="toc-sec-factor1" class="nav-link" data-scroll-target="#sec-factor1"><span class="header-section-number">6.2.3</span> Factor 1: KIRC vs KIRP</a>
  <ul>
  <li><a href="#multi-omics-interaction-network" id="toc-multi-omics-interaction-network" class="nav-link" data-scroll-target="#multi-omics-interaction-network"><span class="header-section-number">6.2.3.1</span> Multi-omics interaction network</a></li>
  <li><a href="#pathway-analysis" id="toc-pathway-analysis" class="nav-link" data-scroll-target="#pathway-analysis"><span class="header-section-number">6.2.3.2</span> Pathway analysis</a></li>
  </ul></li>
  <li><a href="#factor-2-kich" id="toc-factor-2-kich" class="nav-link" data-scroll-target="#factor-2-kich"><span class="header-section-number">6.2.4</span> Factor 2: KICH</a>
  <ul>
  <li><a href="#multi-omics-interaction-network-1" id="toc-multi-omics-interaction-network-1" class="nav-link" data-scroll-target="#multi-omics-interaction-network-1"><span class="header-section-number">6.2.4.1</span> Multi-omics interaction network</a></li>
  <li><a href="#pathway-analysis-1" id="toc-pathway-analysis-1" class="nav-link" data-scroll-target="#pathway-analysis-1"><span class="header-section-number">6.2.4.2</span> Pathway analysis</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-factor-analysis" id="toc-sec-lesson-learnt-factor-analysis" class="nav-link" data-scroll-target="#sec-lesson-learnt-factor-analysis"><span class="header-section-number">6.3</span> Lessons Learnt</a></li>
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information"><span class="header-section-number">6.4</span> Session Information</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./factor-analysis.html"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-factor-analysis" class="quarto-section-identifier"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="6.1">
<h2 data-number="6.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">6.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at section Lesson Learnt: <a href="#sec-lesson-learnt-factor-analysis" class="quarto-xref"><span>Section 6.3</span></a>.</p>
<ul>
<li>Here we perform a multi-omics integration of transcriptomics, proteomics, micro-RNAs and epigenomics data for the three different kidney carcinoma types.</li>
<li>We identify and model the hidden factors that capture biological and technical sources of variability.</li>
<li>We correlate the hidden factors with the clinical covariates and identify the factors that discriminates between KICH, KIRC and KIRP.</li>
<li>Finally, we select the top features for each molecular layer of the factor of interest to build a molecular signature for that factor, and the corresponding gene-interaction network.</li>
</ul>
</section>
<section id="multi-omics-factor-analysis" class="level2" data-number="6.2">
<h2 data-number="6.2" class="anchored" data-anchor-id="multi-omics-factor-analysis"><span class="header-section-number">6.2</span> Multi-Omics Factor Analysis</h2>
<section id="data-normalization" class="level3" data-number="6.2.1">
<h3 data-number="6.2.1" class="anchored" data-anchor-id="data-normalization"><span class="header-section-number">6.2.1</span> Data normalization</h3>
<p>In the previous chapters we have analysed independently the transcriptomics (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>), proteomics (see <a href="proteomics-analysis.html" class="quarto-xref"><span>Chapter 3</span></a>), micro-RNAs (see <a href="micro-RNAs-analysis.html" class="quarto-xref"><span>Chapter 4</span></a>), genomics and epigenomics (see <a href="genomics-and-epigenomics-analysis.html" class="quarto-xref"><span>Chapter 5</span></a>) data from three different kidney carcinomas KICH, KIRC and KIRP. This provided quite some insights on how the three kidney carcinomas are different at a molecular level. Also, we observed that different omics layers provided insights on different aspects of the cancers: for example, biopsies from KIRC patients clearly showed a strong signature for enrichment in glycolysis, glycogen metabolism and <em>HIF1A</em>-mediated response to hypoxia (<a href="https://doi.org/10.1126/science.1160809">Warburg effect</a>) that was not so apparent in the other omics.</p>
<p>We observed as well consistent biological signature across different omics layers. For example:</p>
<ul>
<li>A strong EMT signature in biopsies from KIRC patients was detected both at the transcriptomics level and at the epigenomics level.</li>
<li>Evidences of high infiltrating macrophages in KIRP were obtained by deconvoluting the bulk transcriptomics, and a strong signature for macrophages recruitment was also detected in the methylation patterns associated with KIRP.</li>
<li>Significant proportion of somatic non-silent mutations in <em>MET</em> gene was observed in KIRP patients, and the <em>“MET-activated PTK2 signaling”</em> was differentially methylated in those patients as well.</li>
</ul>
<p>While the different omics layers analysed independently draws a consistent depiction of the kidney carcinomas at the molecular level, the interplay between the different omics layers is lost when considering each layer separately. To grasp a better understanding of the interactions between the different omics layers, we can analyse them all together with integrative techniques (such as the <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02015-1">Multi-Omics Factor Analysis</a>), which identifies major dimensions of disease heterogeneity and can generate a multi-omics signature for the kidney carcinomas types.</p>
<p>Here we leverage and integrate all the omics layers that we have analysed independently in the previous chapters, including:</p>
<ul>
<li>transcriptomics</li>
<li>proteomics</li>
<li>micro-RNAs</li>
<li>epigenomics</li>
</ul>
<p>We have decided not to include CNVs and non-silent somatic mutations because the model we are going to use fail to capture significant signal from those binary sparse data.</p>
<p>So far, we have preprocessed each omics readout independently, i.e.: filtering out lowly abundant transcripts, or dropping methylation probes with missing values. The omics data are on different scales, and for proper estimating each omics contribution in the Multi-Omics Factor Analysis we would need to rescale and normalize each omics readout. Moreover, a feature selection step is necessary since the data shape have 4 orders of magnitude of difference, from 190 features for the micro-RNA up to the 141,704 ones of the epigenomics. Briefly, the normalization steps we performed before multi-omics integration are:</p>
<ul>
<li>Transcriptomics counts were scaled and normalized by variance stabilizing transformation. The top 1,000 most variable genes were selected for integration.</li>
<li>Proteomics readouts were already scaled and centered, and all the 234 features were kept for the integration.</li>
<li>micro-RNAs counts were quantile-normalized. All 190 features were kept for the integration.</li>
<li>Beta values of the methylation status of the epigenomics probes were not normalized. The top 1,000 most variable probes were selected for integration.</li>
</ul>
<p>We can visualize the distribution of the multi-omics features across the 887 kidney carcinoma patients. We can immediately notice that we have a good coverage of the patients across the different omics layers:</p>
<ul>
<li>Proteomics data for 79.9% (709 / 887) of patients.</li>
<li>Transcriptomics data for all the patients, 100% (887 / 887).</li>
<li>micro-RNA data for 95,4% (846 / 887) of patients.</li>
<li>Epigenomics data for 72,7% (646 / 887) of patients.</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 1: Distribution of the multi-omics features across the 887 kidney carcinoma patients</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="multi-omics-data-integration-and-analysis" class="level3" data-number="6.2.2">
<h3 data-number="6.2.2" class="anchored" data-anchor-id="multi-omics-data-integration-and-analysis"><span class="header-section-number">6.2.2</span> Multi-omics data integration and analysis</h3>
<p>Now that we have normalized and prepared the multi-omics data, it is time to integrate them and run the Multi-Omics Factor Analysis model on them. MOFA inference is done using the variational Bayes algorithm, which maximises a quantity called the Evidence Lower Bound (ELBO). The ELBO is supposed to increase monotonically up to convergence. For our multi-omics kidney carcinomas data, the model converged after 551 iterations, with an ELBO=-1156553.09 and a deltaELBO=46.776 (0.00033245%).</p>
<p>Let’s now explore the Factors identified with the MOFA model, the amount of variance explained, and their correlation with the clinical covariates. The model explains at least 40% of the variance in each omics layer, from ~47% in proteomics up to 67.1% in the transcriptomics data. We trained the MOFA model to report on 10 hidden factors. If we look at the distribution of the variance captured in the hidden factors, we can see that Factor 1 and Factor 2 captures more than 30% of the variance for each omics layer, and most of the variance for each layer it is captured within these two factors. The only exception is Factor 3, which captures 14.4% of the micro-RNAs varaince, and Factor 5, which captures ~8% of epigenomics variance.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 2: Overview of the variance captured by the MOFA model.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Then, we can look at the correlation between the factors and clinical covariates. First, we correlate the 10 hidden factors with each other, and we see that the factors do not correlate with each other, suggesting that each of them captures a different aspect of the technical or biological variability of the dataset. The only exception could be Factor 5, which seems to weakly correlates with Factors 1, 2, 4, 6, 7 and 8. Second, we correlate the factors with the clinical covariates. We gonna first test for significant association between the factor and the clinical covariate:</p>
<ul>
<li><strong>Factor 1</strong> is significantly associated with the kidney carcinoma types, and it is associated with the clinical covariates <em>cancer type</em>, <em>clinical stage</em>, <em>histological grade</em>, <em>histological type</em>, <em>subtype mRNA</em>, <em>subtype miRNA</em>, <em>subtype integrative</em> and <em>subtype selected</em>.</li>
<li><strong>Factor 2</strong> is also associated with the kidney carcinoma types, and with the clinical covariates <em>cancertype</em>, <em>histological type</em>, <em>subtype selected</em>.</li>
<li><strong>Factor 3</strong> seems slightly associated with the tumor severity and the covariates <em>pathologic tumor grading</em>, <em>subtype selected</em>, <em>tumor cells</em>, <em>tumor nuclei</em>, <em>tumor stage</em> and <em>vital status</em>.</li>
<li><strong>Factor 5</strong> strongly correlated with the epigenomics data, and is significantly associated with <em>clinical stage</em>, <em>subtype DNA methylation</em>, <em>subtype integrative</em> and <em>subtype selected</em>.</li>
</ul>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 3: Correlation of the hidden factors with each other and with the clinical covariates.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Third, we check the Pearson correlation coefficient (r) between the factors and the clinical covariates. We can easily spot a pattern of contrapposition of different directions of correlation coefficient for the following clinical covariates in Factor 1 versus Factor 2: <em>cancer type</em>, <em>histological type</em>, <em>subtype DNA methylation</em>, <em>subtype integrative</em>, <em>subtype other</em> and <em>subtype selected</em>, suggesting that these two first factors capture most of the differences of the three different kidney cariconomas (KICH, KIRCH and KIRP). Given the observations we have done in differences in tumor progression (see <a href="dataset-exploration.html#sec-dataset-exploration" class="quarto-xref"><span>Section 1.3.7</span></a>) and different levels of infitrating immune cells (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>) between these three kidney carcinomas, it makes sense that the same pattern of contrapposition in the directions of correlation coefficient is observed as well for <em>clinical stage</em>, <em>monocyte infiltration</em>, <em>pathologic tumor grade</em> and <em>tumor nuclei</em>.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 4: Pearson correlation coefficient between factors and clinical covariates.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>As we mentioned before, each factor captures a different dimension of heterogeneity in the data. Mathematically, each factor ordinates the samples along a one-dimensional axis centered at zero. Samples with different signs manifest opposite phenotypes along the inferred axis of variation, with higher absolute value indicating a stronger effect. Interpretation of the factor analysis and the relative position of the samples across each factor is similar to the interpretation of the principal components in a PCA.</p>
<p>Indeed, we can confirm that KICH, KIRC and KIRP are separated along the axis of Factor 1 and Factor 2, as observed in the correlation between the hidden factors and the clinical covariates. More in detail, Factor 1 seems to be a gradient that differentiate first KIRC, then KICH and finally KIRP along its axis. Factor 2, instead, strongly separates KICH from KIRC and KIRP. This result is very encouraging for characterizing molecularly the three kidney carcinoma types, and for inferring multi-omics signatures that discriminate them.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Samples distributions across factors and Factor 1 X Factor 2 biplot.</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="sec-factor1" class="level3" data-number="6.2.3">
<h3 data-number="6.2.3" class="anchored" data-anchor-id="sec-factor1"><span class="header-section-number">6.2.3</span> Factor 1: KIRC vs KIRP</h3>
<p>Let’s explore the contribution of each omics layer to Factor 1.</p>
<p>The MOFA model predict a weight for each of the features of the omics layer, but we are going to focus on the principal ones in order to build out molecular signature associated with the different kidney carcinoma types. The weights are useful to get an idea of which genes are driving each factor.</p>
<p>For each omics layer we generate scatter plots of the observations vs factor values for the 20 features with highest positive weight and the 20 ones with the highest negative weight. For each feature we compute a linear regression model to explore the linearity of this relationship, and to get as well some estimates on the correlation (R²) and the likelihood (p-value). In addition to the scatterplots, we also plot the top 100 features with the highest absolute weight (either positive or negative), and build a biclustered heatmap out of it. The heatmap allows to discover trends and clusters of features-sample pairs, and test the distribution of different clinical covariates with the samples and the cancer types.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">Transcriptomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">micro-RNAs</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">Proteomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-4" role="tab" aria-controls="tabset-1-4" aria-selected="false">Epigenomics</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 6: Scatterplots for the top 40 transcriptomics features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 7: Heatmap for the top 100 transcriptomics features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 8: Scatterplots for the top 40 micro-RNAs features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Heatmap for the top 100 micro-RNAs features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 10: Scatterplots for the top 40 proteomics features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 11: Heatmap for the top 100 proteomics features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-4-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-17-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 12: Scatterplots for the top 40 epigenomics features in Factor 1</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-18-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 13: Heatmap for the top 100 epigenomics features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>Interpretation of these high dimensional data is not straightforward, but we can pick on some emerging trends:</p>
<ul>
<li><strong>Transcriptomics</strong>:
<ul>
<li>Most of positive features seems not linear. Negative factors, which are strongly associated with KIRC, are instead linear.</li>
<li>Looking at the heatmap with the top 100 features with highest absolute weight, we can see that these features are mostly associated to KIRC, which clusters in a single large block on the right hand side of the heatmap. KICH and KIRP seems as well quite clustered based on the top 100 transcriptomics features, except for a small clusters of KIRP patients that sits next to next to KIRC where most of the KIRP patients with worse outcome are located.</li>
</ul></li>
<li><strong>micro-RNAs</strong>:
<ul>
<li>Both features with high positive and high negative weights seems to mostly have linear relationships with Factor 1. We can observe as well the KIRC -&gt; KICH -&gt; KIRP gradient detected as well in the Factor 1 X Factor 2 biplot. Some of the features scatterplots also show some linear artefacts, where the samples all sits at the same y-axis position. This is probably due to the fact that in Factor 1 the weight is 0 for most of the micro-RNAs features.</li>
<li>Looking at the heatmap, KIRC is clearly divided into two different clusters, with the smallest one on the left-hand side of the heatmap having most of the patients with worse outcome (<em>vital status</em> == “Dead”). The features with the highest weight responsible for this cluster are <em>hsa-miR-21-3p</em>, <em>hsa-miR-21-5p</em>, <em>hsa-miR-25-3p</em>, <em>hsa-miR-379-5p</em>, <em>hsa-miR-134-5p</em> and the micro-RNA family <em>hsa-miR-199a-3p</em>, <em>hsa-miR-199a-5p</em>, <em>hsa-miR-199b-3p</em>. KICH and KIRP seems to have clear patterns of different micro-RNAs weights that separates them from KIRC.</li>
</ul></li>
<li><strong>Proteomics</strong>:
<ul>
<li>Most of the proteomics features seems linear for Factor 1. There is a clear cluster separating KIRC from KICH and KIRP.</li>
<li>The proteomics data are clearly driven from the metabolic signature of KIRC with high glycolysis (Glyceraldehyde-3-Phosphate Dehydrogenase <em>GAPDH</em>, Lactate Dehydrogenase A and B <em>LDHA</em> and <em>LDHB</em>, Pyruvate Kinase M1/2 <em>PKM2</em>, Glycogen Phosphorylase B, L and M <em>PYGB</em>, <em>PYGL</em>, <em>PYGM</em>) and hypoxic adaptation (Carbonic Anhydrase 9 <em>CA9</em>, Hypoxia Inducible Factor 1 Subunit Alpha <em>HIF1A</em>, Heat-Shock Protein 70 <em>HSP70</em> and pT346: N-Myc Downstream Regulated 1 <em>NDRG1_pT346</em>). In addition to those proteins, we identified also oncogenes <em>p53</em>, <em>Cyclin B1</em>, and <em>PDL1</em>, which is involved in tumor immune evasion</li>
<li>The heatmap of top 100 features with highest absolute weight show clear clusters separating KIRC from KICH and KIRP. KIRP can possibly be separated as well into two smaller subclusters driven by two different groups of proteins.</li>
</ul></li>
<li><strong>Epigenomics</strong>:
<ul>
<li>The scatterplots of the features with highest absolute weights suggested that linearity is not present for all of them. However, KIRC and KIRP are clearly separated, and the KIRC -&gt; KICH -&gt; KIRP gradient seems maintained across all top features.</li>
<li>Looking at the heatmap, KICH methylation profile seems overlapping with both KIRC and KIRP. KIRC samples are divided in 2 clusters with no clear correlation with the clinical covariates that we have included on the top of the heatmap. Most of the top 100 epigenomics features are associated with KIRP.</li>
</ul></li>
</ul>
<p>Overall, based on these results, we expect transcriptomics, proteomics and epigenomics top features to characterize KIRC. KICH methylation profile seems overlapping with both KIRC and KIRP. micro-RNAs top features seems to separate the KIRC patients with had a worse clinical outocome.</p>
<section id="multi-omics-interaction-network" class="level4" data-number="6.2.3.1">
<h4 data-number="6.2.3.1" class="anchored" data-anchor-id="multi-omics-interaction-network"><span class="header-section-number">6.2.3.1</span> Multi-omics interaction network</h4>
<p>In the next step, we retrieve the most important features in Factor 1 for each omics layer and we look manually curated databases for gene-gene interaction in order to refine the molecular signature we are building for the kidney carcinomas. We could follow two very naive strategies:</p>
<ul>
<li>if weights have a normal distribution, set a threshold above/below mean +/- 2sd (red dotted lines).</li>
<li>Select top 10% features with highest / lowest weights for each omics layer (blue dots).</li>
</ul>
<p>Based on the sorted features vs weights distributions, we can see that for Factor 1 the features do not have a normal distribution in any of omics layer. We therefore decide to get the top 10% features with the highest absolute weight to build our gene network.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-19-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 14: Weight distributions of omics features in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>To include the epigenomics information in the gene network, we will map the top methylation probes to their corresponding gene based on the Illumina Methylation 450k chip annotation profile provided by the <a href="https://bioconductor.org/packages/release/bioc/html/minfi.html">minfi R package</a>.</p>
<p>We retrieve the gene interactions networks from two main sources: the <a href="https://string-db.org/">STRING database</a> and <a href="https://omnipathdb.org/">OmniPath</a>. STRING aggregates information on Protein-Protein Interactions (PPIs) from multiple sources. They provide several levels of evidences with corresponding confidence scores. We have decided to retain only Protein-Protein Interactions that were experimentally validated and with a confidence score of at least 700 (in a scale from 0 to 1000). Among the others, OmniPath aggregates information on gene interactions, and it is not restricted to PPIs, but covers several aspects of gene interactions and regulatory networks. From OmniPath, we retrieve:</p>
<ul>
<li><em>post translational interactions</em>: physical interactions of proteins, i.e.: PPIs.</li>
<li><em>transcriptional interactions</em>: gene regulatory interactions, i.e.&nbsp;interactions between transcription factors and the genes they regulates.</li>
<li><em>post transcriptional interaction</em>: i.e.&nbsp;miRNAs-mRNA interactions and the resulting gene expression regulation.</li>
<li><em>miRNAs transcriptional interactions</em>: i.e.&nbsp;transcriptional regulation of micrp-RNAs.</li>
</ul>
<p>Let’s integrate the interaction networks pulled from OmniPath with the high confidence PPI retrieved from STRING into a single, cohesive gene interaction network where we can layer the multi-omics information from the kidney carcinomas.</p>
<p>All these information combined allow us to leverage the contribution of the top features from each omics layer. Let’s plot them on the gene network we have reconstructed.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-23-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 15: KIRC vs KIRP and KICH molecular gene network based on STRING and OmniPath interactions.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>The gene interaction network we have reconstructed provides great insights on the molecular characterization of KIRC and KIRP, based on the magnitude of the Factor 1 weights for each omics feature:</p>
<ul>
<li><strong>KIRC</strong>:
<ul>
<li><em>FLT1</em>, <em>HIF1A</em>, <em>NOTCH4</em>, <em>PDGFRB</em>, <em>VEGFA</em> suggest a hypoxia-driven microenvironment for KIRC.</li>
<li>overexpression of <em>ENPP3</em> transcript and overabundance of the proteins <em>PYGB</em>, <em>PYGL</em>, <em>PYGM</em> suggest higher glycolysis and Warburg effect in KIRC compared to KICH and KIRP.</li>
<li><em>miR-210-3p</em> and <em>miR-143-3p</em> supports the higher hypoxic adaptation and metabolic rewiring of KIRC.</li>
<li><em>ITGA2</em> hypermethylated, and <em>COL3A1</em>, <em>COL5A3</em>, <em>COL6A3</em> suggests a stronger ECM remodelling and EMT signal in KIRC vs KIRP and KICH.</li>
</ul></li>
<li><strong>KIRP</strong>:
<ul>
<li><em>CHL1</em>, <em>CLDN7</em>, <em>KRT7</em>, <em>PIGR</em>, <em>SLC34A2</em>, <em>WFDC2</em>, suggest a larger role of epithelial integrity, tight junctions and differentiation in KIRP and KICH when compared to KIRC.</li>
<li><em>miR-21-5p</em> and <em>miR-10a-5p</em> may promote proliferation and angiogenesis in KIRP.</li>
</ul></li>
</ul>
</section>
<section id="pathway-analysis" class="level4" data-number="6.2.3.2">
<h4 data-number="6.2.3.2" class="anchored" data-anchor-id="pathway-analysis"><span class="header-section-number">6.2.3.2</span> Pathway analysis</h4>
<p>As we have done for the differentially abundant features at the single omics level, we can run a Gene Set Enrichment Analysis (GSEA) on the multi-omics features we have identified in Factor 1.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">KEGG ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">KEGG genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">Reactome ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-4" role="tab" aria-controls="tabset-2-4" aria-selected="false">Reactome genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-5" role="tab" aria-controls="tabset-2-5" aria-selected="false">WikiPathways ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-6" role="tab" aria-controls="tabset-2-6" aria-selected="false">WikiPathways heatplot</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-26-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 16: Dotplot of enriched KEGG pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-27-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 17: Heatplot of enriched KEGG pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-28-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 18: Dotplot of enriched Reactome pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-4-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-29-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 19: Heatplot of enriched Reactome pathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-5-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-30-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 20: Dotplot of enriched WikiPathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-6-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-31-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 21: Heatplot of enriched WikiPathways in Factor 1.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>We gathered the following insights from the gene set enrichment analyses:</p>
<ul>
<li>We could not find any enriched relevant GO terms.</li>
<li>We identified two KEGG pathways: <em>HIF-1 singnaling pathways</em>, with 8/10 genes present, and the generic <em>Metabolic pathway</em>, with 13 / 20 genes present. Both of these pathways are associated with the Warburg effect present in KIRC biopsies.</li>
<li>We have identified 15 Reactome pathways with an adjusted p-value =&lt; 0.05, but their gene coverage (gene ratio) was always low (~10% or less). Among them, however, we have identified <em>extracellular matrix organization</em>, <em>MET promotes cell motility</em>, <em>Regulation of gene expression by Hypoxia-inducible Factor</em>, <em>Signaling by NOTCH</em> and <em>Signaling by VEGF</em>, which are all common molecular signature that differentiate KIRC from KIRP.</li>
<li>We have identified 19 Wikipathways with an adjusted p-value =&lt; 0.05. Similarly to the Reactome pathwats, their gene coverage (gene ratio) was low (~17% or less). However, the pathways we identified as enriched are of particular interest and were highly accurate in identified the molecular signatures of KIRC and KIRP. Among the others, we identified: <em>Clear cell renal cell carcinoma pathways</em>, <em>Focal adhesion: PI3K-Akt-mTOR-signaling</em>, <em>Warburg effect modulated by deubiquitinating enzymes and their substrates</em>, <em>Warburg effect modulated by deubiquitinating enzymes and their substrates</em>, <em>miRNA targets in ECM and membrane receptors</em>, <em>Notch signaling</em>, <em>Genes controlling nephrogenesis</em>, <em>Macrophage-stimulating protein (MSP) signaling</em>.</li>
</ul>
<p>While the multi-omics factorial model did not rescued all the genes involved in the key signature pathways of KIRC and KIRP, the figure that emerged was more complete than the one reconstructed from each single omics layer. While not all features in a pathway may be highly disregulated (and having a high absolute weight in the MOFA Factor), the combination of transcriptomics, proteomics, micro-RNA and epigenomics signal allowed the reconstruction of a molecular network of interactions that properly describe KIRC and KIRP. For each feature, we have information on its weight on the model, the omic origin (e.g: transcriptomics evidence), their molecular function (e.g.: transcription factor, transmembrane receptor, etc.) and the nature of the molecular interaction with the other features in the model. Taken all together, the KIRC vs KIRP gene network may facilitate the discovery of kidney carcinoma specific therapeutic targets (see <span class="quarto-unresolved-ref">?sec-target-discovery</span>).</p>
</section>
</section>
<section id="factor-2-kich" class="level3" data-number="6.2.4">
<h3 data-number="6.2.4" class="anchored" data-anchor-id="factor-2-kich"><span class="header-section-number">6.2.4</span> Factor 2: KICH</h3>
<p>Let’s explore the contribution of each omics layer to Factor 2. For each omics layer we generate the scatter plots and a biclustered heatmap for the top 40 and 100 features with higher weights, as we have done for Factor 1 (see <a href="#sec-factor1" class="quarto-xref"><span>Section 6.2.3</span></a>).</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">Transcriptomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">micro-RNAs</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-3" role="tab" aria-controls="tabset-3-3" aria-selected="false">Proteomics</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-4" role="tab" aria-controls="tabset-3-4" aria-selected="false">Epigenomics</a></li></ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-32-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 22: Scatterplots for the top 40 transcriptomics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-33-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 23: Heatmap for the top 100 transcriptomics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-34-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 24: Scatterplots for the top 40 micro-RNAs features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-35-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 25: Heatmap for the top 100 micro-RNAs features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-36-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 26: Scatterplots for the top 40 proteomics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-37-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 27: Heatmap for the top 100 proteomics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-4-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-38-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 28: Scatterplots for the top 40 epigenomics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-39-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 19: Heatmap for the top 100 epigenomics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>Let’s look at the emerging trends from the multi-omics factor model in Factor2:</p>
<ul>
<li><strong>Transcriptomics</strong>:
<ul>
<li>Most of the positive features, that are correlated with KICH, seems linear, in contrast to the negative ones. As also observed in the Factor 1 X Factor 2 biplot, KICH biopsies are clearly separated from KIRC and KIRP biopsies, which in general clusters together. Proto-oncogene <em>KIT</em> is strongly associated with KICH in Factor 2, which is consistent with what we observed in the proteomics analyses (see <a href="proteomics-analysis.html" class="quarto-xref"><span>Chapter 3</span></a>).</li>
<li>Looking at the heatmap with the top 100 features with highest absolute weight, we can see that just 16 of them are associated with KICH, but their weight are very high. Based on Factor 2 weigths, expression profiles are similar in KIRC and KIRP, and the clusters of these two kidney carcinoma types are not well defined. A small subcluster of KIRC and KIRP biopsies sits next to the KICH cluster, has a different profile than the rest of KIRC and KIRP samples and seems enriched in patients with worse prognosis (<em>vital status</em>).</li>
</ul></li>
<li><strong>micro-RNAs</strong>:
<ul>
<li>The scatterplots indicates that the separation between KICH and KIRC or KIRP samples is less clear in the micro-RNAs when compared to the transcriptomics data. The linearity assumption seems respected in many cases, and we observe some linear artefacts as observed as well in Factor 1 (see <a href="#sec-factor1" class="quarto-xref"><span>Section 6.2.3</span></a>).</li>
<li>The heatmap indicates again a clear solid cluster of KICH biopsies. KIRC and KIRP seems to have different expression profiles based on Factor 2 weights. KIRC samples clusters in two separated groups, the smallest one with a higher number of patients with worse outcome</li>
</ul></li>
<li><strong>Proteomics</strong>
<ul>
<li>Most of the proteomics features seems not linear for Factor 2, but they clearly separates KICH biopsies from KIRC and KIRP. For features strongly associate with KIRC (e.g.: <em>PYGB</em>, <em>HIF1ALPHA</em>, <em>LDHA</em>), there is a clear separation between KIRC and KIRP biopsies as well. The features with the highest positive weight (strongly associated with KIRC) are <em>CKIT</em>, <em>CLAUDIN7</em>, <em>ECADHERIN</em>, and this is consistent with the idea that KICH tumors generally retain their epitelial differentiation.</li>
<li>The heatmap indicates again a clear solid cluster of KICH biopsies. KIRC and KIRP seems to have different expression profiles based on Factor 2 weights.</li>
</ul></li>
<li><strong>Epigenomics</strong>:
<ul>
<li>The scatterplots of the features with highest absolute weights suggested that linearity is present in most of these features.</li>
<li>Looking at the heatmap, KICH biopsies have a strong profile of 44 probes with high positive weight in Factor 2. The top 100 features with highest absolute weight for Factor 2 fail to further discriminate between KIRC and KIRP biopsies, which clusters together in interspersed groups.</li>
</ul></li>
</ul>
<p>Overall, based on these results, we expect Factor 2 transcriptomics, proteomics and epigenomics top features to characterize KICH biopsies.</p>
<section id="multi-omics-interaction-network-1" class="level4" data-number="6.2.4.1">
<h4 data-number="6.2.4.1" class="anchored" data-anchor-id="multi-omics-interaction-network-1"><span class="header-section-number">6.2.4.1</span> Multi-omics interaction network</h4>
<p>In the next step, we retrieve the most important features in Factor 2 for each omics layer and we look manually curated databases for gene-gene interaction in order to refine the molecular signature we are building for the kidney carcinomas.</p>
<p>Based on the sorted features vs weights distributions, we can see that for Factor 2 the features do not have a normal distribution in any of omics layer. We therefore decide to get the top 10% features with the highest absolute weight to build our gene network.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-40-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 20: Weight distributions of omics features in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>As we have done for Factor 1 (see <a href="#sec-factor1" class="quarto-xref"><span>Section 6.2.3</span></a>), we layer the multi-omics features identified with the Factor Analysis on top of a gene interaction network build by merging STRING and gene networks from OmniPath. To help to generate a simplified, more readable gene network, we can also remove the PPI interactions retrieved from STRING, and use only a gene interaction networks reconstructed from the OmniPath interaction networks.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-43-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 21: KICH molecular gene network based on OmniPath interactions.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>The gene interaction network we have reconstructed provides great insights on the molecular characterization of KICH, based on the magnitude of the Factor 2 weights for each omics feature:</p>
<ul>
<li>Overexpressed transcripts for genes <em>INPP5J</em>, <em>KIT</em>, <em>NTN1</em>, <em>SCNN1A</em> suggest a strong differentiated epithelial phenotype.</li>
<li>Suppression of <em>HNF4</em> transcript and lower abundance of <em>HIF1A</em>, <em>LDHA</em> and <em>NDRG1</em> proteins suggest reduced metabolic activity and reliance on glycolysis in KICH compared to KIRC and KIRP.</li>
<li>Hypometilation of homeobox genes <em>HOXA7</em>, <em>HOXB3</em>, <em>HOXB5</em>, <em>HOXC4</em> may allow activation of differentiation pathways, maintaining epithelial identity.</li>
<li><em>miR-200c-3p</em> could be involved in maintaining the epithelial integrity and inhibit EMT in KICH.</li>
<li>Suppressed <em>miR-21-5p</em> could explain the limited pathways associated with immune evasion and inflammation in KICH.</li>
</ul>
</section>
<section id="pathway-analysis-1" class="level4" data-number="6.2.4.2">
<h4 data-number="6.2.4.2" class="anchored" data-anchor-id="pathway-analysis-1"><span class="header-section-number">6.2.4.2</span> Pathway analysis</h4>
<p>As we have done for the differentially abundant features at the single omics level, we can run a Gene Set Enrichment Analysis (GSEA) on the multi-omics features we have identified in Factor 2.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-4-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-1" role="tab" aria-controls="tabset-4-1" aria-selected="true">GO BP ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-2" role="tab" aria-controls="tabset-4-2" aria-selected="false">GO BP genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-3" role="tab" aria-controls="tabset-4-3" aria-selected="false">Reactome ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-4-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-4" role="tab" aria-controls="tabset-4-4" aria-selected="false">Reactome genes</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-5-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-5" role="tab" aria-controls="tabset-4-5" aria-selected="false">WikiPathways ratios</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-6-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-6" role="tab" aria-controls="tabset-4-6" aria-selected="false">WikiPathways heatplot</a></li></ul>
<div class="tab-content">
<div id="tabset-4-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-4-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-46-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 22: dotplot of enriched GO Biological Process terms in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-47-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 23: Heatplot of enriched GO Biological Process terms in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-48-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 24: Dotplot of enriched Reactome pathways in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-4" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-4-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-49-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 25: Heatplot of enriched Reactome pathways in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-5" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-5-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-50-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 26: Dotplot of enriched WikiPathways in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-6" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-6-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="factor-analysis_files/figure-html/unnamed-chunk-51-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 27: Heatplot of enriched WikiPathways in Factor 2.</span></figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>The combination of transcriptomics, proteomics, micro-RNA and epigenomics signal allowed the reconstruction of a molecular network of interactions that properly describe KICH. For each feature, we have information on its weight on the model, the omic origin (e.g: transcriptomics evidence), their molecular function (e.g.: transcription factor, transmembrane receptor, etc.) and the nature of the molecular interaction with the other features in the model.</p>
</section>
</section>
</section>
<section id="sec-lesson-learnt-factor-analysis" class="level2" data-number="6.3">
<h2 data-number="6.3" class="anchored" data-anchor-id="sec-lesson-learnt-factor-analysis"><span class="header-section-number">6.3</span> Lessons Learnt</h2>
<p>So far, we have learnt:</p>
<ul>
<li>A</li>
</ul>
<p>Based on genomics and epigenomics data, we have learnt:</p>
<ul>
<li>Copy Number Variants:
<ul>
<li>KICH has significantly more genome instability and CNV events than KIRC and KIRP</li>
<li>whole genome duplication seems more likely to occur than loss of chromosomes</li>
<li>there is a link between CNVs signatures and cancer advancement, which is a common feature across all kidney carcinomas and not limited to KIRC, KIRP or KICH.</li>
</ul></li>
<li>Somatic Mutations:
<ul>
<li>KIRC is the kidney carcinoma with the highest amount of non-silent somatic mutations.</li>
<li>KICH:
<ul>
<li>~30% of the patients (21 / 66) had a non-silent mutation in TP53.</li>
<li>9% of patients (6 / 66) having a mutation on PTEN.</li>
</ul></li>
<li>KIRC:
<ul>
<li>47% of patients (170 / 365) with mutation in VHL</li>
<li>~40% of patients (145 / 365) with mutation on PBRM1</li>
<li>~20% of patients (72 / 365) with non-silent somatic mutations on TTN</li>
</ul></li>
<li>KIRP:
<ul>
<li>16% of KIRP patients (45 / 280) has non-silent somatic mutations on TTN</li>
<li>9% of KIRP patients (25 / 280) with mutation on MUC16</li>
<li>8% of KIRP patients (22 / 280) with mutation on MET</li>
</ul></li>
</ul></li>
<li>Epigenomics:
<ul>
<li>Exploratory analysis suggests:
<ul>
<li>Principal Component Analysis can discriminates between KICH, KIRC and KIRP.</li>
<li><em>subtype DNAmeth</em> co-correlates with residual disease, more advanced cancer and worse prognosis, suggesting that DNA methylations and epigenomics plays a major role in cancer evolution and disease outcome.</li>
</ul></li>
<li>Differential Gene Expression analysis:
<ul>
<li>strong methylation signature that discriminates KICH from KIRC and KIRP (&gt; 4,000 probes).</li>
<li>GSEA on differentially methylated probes supports the idea that:
<ul>
<li>KICH biopsies mostly retained their epithelial differentiation that was instead lost in KIRC and in KIRP.</li>
<li>Higher abundance of infiltrating Macrophages M0 and M2 in KIRP.</li>
<li>A pro-inflammatory, hypoxic microenvironment in KIRC.</li>
</ul></li>
<li>GSEA on Differentially Methylated function Regions support the idea that:
<ul>
<li>General theme of epithelial differentiation retention in KICH.</li>
<li>A stronger Epithelial-to-Mesenchimal Transition in KIRC, with a signature of extra-cellular matrix remodelling and cell migration.</li>
<li><em>MET</em> pathway is differentially methylated in KIRP, which is consistent with the higher chance of <em>MET</em> non-silent somatic mutation in this kind of kidney carcinoma.</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="session-information" class="level2" data-number="6.4">
<h2 data-number="6.4" class="anchored" data-anchor-id="session-information"><span class="header-section-number">6.4</span> Session Information</h2>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: openSUSE Tumbleweed

Matrix products: default
BLAS/LAPACK: /home/andrea/miniforge3/envs/moai/lib/libmkl_rt.so.2;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=it_IT.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Brussels
tzcode source: system (glibc)

attached base packages:
 [1] parallel  grid      stats4    stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] UpSetR_1.4.0                                      
 [2] tidygraph_1.3.1                                   
 [3] umap_0.2.10.0                                     
 [4] stringr_1.5.1                                     
 [5] STRINGdb_2.14.3                                   
 [6] scales_1.3.0                                      
 [7] RColorBrewer_1.1-3                                
 [8] pathview_1.42.0                                   
 [9] org.Hs.eg.db_3.18.0                               
[10] MOFA2_1.12.1                                      
[11] infiltR_0.1.0                                     
[12] IlluminaHumanMethylation450kanno.ilmn12.hg19_0.6.1
[13] minfi_1.48.0                                      
[14] bumphunter_1.44.0                                 
[15] locfit_1.5-9.10                                   
[16] iterators_1.0.14                                  
[17] foreach_1.5.2                                     
[18] Biostrings_2.70.3                                 
[19] XVector_0.42.0                                    
[20] igraph_2.1.4                                      
[21] gridExtra_2.3                                     
[22] ggraph_2.2.1                                      
[23] ggplot2_3.5.1                                     
[24] forcats_1.0.0                                     
[25] edgeR_4.0.16                                      
[26] limma_3.58.1                                      
[27] DT_0.33                                           
[28] dplyr_1.1.4                                       
[29] DOSE_3.28.2                                       
[30] DESeq2_1.42.1                                     
[31] SummarizedExperiment_1.32.0                       
[32] MatrixGenerics_1.14.0                             
[33] matrixStats_1.5.0                                 
[34] GenomicRanges_1.54.1                              
[35] GenomeInfoDb_1.38.8                               
[36] data.table_1.16.4                                 
[37] cowplot_1.1.3                                     
[38] clusterProfiler_4.10.1                            
[39] BiocSingular_1.18.0                               
[40] BiocParallel_1.36.0                               
[41] AnnotationDbi_1.64.1                              
[42] IRanges_2.36.0                                    
[43] S4Vectors_0.40.2                                  
[44] Biobase_2.62.0                                    
[45] BiocGenerics_0.48.1                               

loaded via a namespace (and not attached):
  [1] vroom_1.6.5               progress_1.2.3           
  [3] ggpp_0.5.8-1              HDF5Array_1.30.1         
  [5] vctrs_0.6.5               RApiSerialize_0.1.4      
  [7] shape_1.4.6.1             digest_0.6.37            
  [9] png_0.1-8                 proxy_0.4-27             
 [11] OmnipathR_3.10.1          ggrepel_0.9.6            
 [13] corrplot_0.95             magick_2.8.5             
 [15] MASS_7.3-60.0.1           reshape_0.8.9            
 [17] reshape2_1.4.4            qvalue_2.34.0            
 [19] withr_3.0.2               psych_2.4.12             
 [21] xfun_0.50                 ggfun_0.1.8              
 [23] ggpubr_0.6.0              survival_3.8-3           
 [25] doRNG_1.8.6.1             memoise_2.0.1            
 [27] MatrixModels_0.5-3        gson_0.1.0               
 [29] GlobalOptions_0.1.2       tidytree_0.4.6           
 [31] gtools_3.9.5              KEGGgraph_1.62.0         
 [33] Formula_1.2-5             prettyunits_1.2.0        
 [35] KEGGREST_1.42.0           httr_1.4.7               
 [37] rstatix_0.7.2             restfulr_0.0.15          
 [39] hash_2.2.6.3              rhdf5filters_1.14.1      
 [41] stringfish_0.16.0         rhdf5_2.46.1             
 [43] generics_0.1.3            reactome.db_1.86.2       
 [45] dir.expiry_1.10.0         curl_6.2.0               
 [47] zlibbioc_1.48.2           ScaledMatrix_1.10.0      
 [49] polyclip_1.10-7           doSNOW_1.0.20            
 [51] GenomeInfoDbData_1.2.11   quadprog_1.5-8           
 [53] SparseArray_1.2.4         doParallel_1.0.17        
 [55] xtable_1.8-4              evaluate_1.0.3           
 [57] S4Arrays_1.2.1            BiocFileCache_2.10.2     
 [59] preprocessCore_1.64.0     hms_1.1.3                
 [61] irlba_2.3.5.1             qs_0.27.2                
 [63] colorspace_2.1-1          filelock_1.0.3           
 [65] polynom_1.4-1             readxl_1.4.3             
 [67] reticulate_1.40.0         magrittr_2.0.3           
 [69] readr_2.1.5               Rgraphviz_2.46.0         
 [71] later_1.4.1               viridis_0.6.5            
 [73] ggtree_3.10.1             limSolve_1.5.7.1         
 [75] lattice_0.22-6            genefilter_1.84.0        
 [77] SparseM_1.84-2            XML_3.99-0.18            
 [79] shadowtext_0.1.4          class_7.3-23             
 [81] pillar_1.10.1             nlme_3.1-167             
 [83] caTools_1.18.3            compiler_4.3.2           
 [85] beachmat_2.18.1           RSpectra_0.16-2          
 [87] stringi_1.8.4             lubridate_1.9.4          
 [89] GenomicAlignments_1.38.2  plyr_1.8.9               
 [91] crayon_1.5.3              abind_1.4-8              
 [93] BiocIO_1.12.0             gridGraphics_0.5-1       
 [95] chron_2.3-62              graphlayouts_1.2.2       
 [97] bit_4.5.0.1               fastmatch_1.1-6          
 [99] codetools_0.2-20          openssl_2.3.1            
[101] MCPcounter_1.2.0          e1071_1.7-16             
[103] GetoptLong_1.0.5          multtest_2.58.0          
[105] splines_4.3.2             circlize_0.4.16          
[107] Rcpp_1.0.14               basilisk_1.14.3          
[109] quantreg_6.00             dbplyr_2.5.0             
[111] sparseMatrixStats_1.14.0  HDO.db_0.99.1            
[113] cellranger_1.1.0          knitr_1.49               
[115] blob_1.2.4                clue_0.3-66              
[117] fs_1.6.5                  checkmate_2.3.2          
[119] DelayedMatrixStats_1.24.0 logger_0.4.0             
[121] ggsignif_0.6.4            ggplotify_0.1.2          
[123] tibble_3.2.1              sqldf_0.4-11             
[125] Matrix_1.6-5              statmod_1.5.0            
[127] tzdb_0.4.0                lpSolve_5.6.23           
[129] tweenr_2.0.3              pkgconfig_2.0.3          
[131] pheatmap_1.0.12           tools_4.3.2              
[133] cachem_1.1.0              RSQLite_2.3.9            
[135] rvest_1.0.4               viridisLite_0.4.2        
[137] DBI_1.2.3                 graphite_1.48.0          
[139] fastmap_1.2.0             rmarkdown_2.29           
[141] Rsamtools_2.18.0          broom_1.0.7              
[143] patchwork_1.3.0           graph_1.80.0             
[145] carData_3.0-5             scrime_1.3.5             
[147] selectr_0.4-2             snow_0.4-4               
[149] farver_2.1.2              mgcv_1.9-1               
[151] scatterpie_0.2.4          gsubfn_0.7               
[153] yaml_2.3.10               rtracklayer_1.62.0       
[155] illuminaio_0.44.0         cli_3.6.3                
[157] purrr_1.0.2               siggenes_1.76.0          
[159] GEOquery_2.70.0           lifecycle_1.0.4          
[161] askpass_1.2.1             uwot_0.2.2               
[163] ggpmisc_0.6.1             backports_1.5.0          
[165] annotate_1.80.0           timechange_0.3.0         
[167] gtable_0.3.6              rjson_0.2.23             
[169] ape_5.8-1                 jsonlite_1.8.9           
[171] bitops_1.0-9              bit64_4.6.0-1            
[173] Rtsne_0.17                yulab.utils_0.2.0        
[175] base64_2.0.2              ReactomePA_1.46.0        
[177] proto_1.0.0               RcppParallel_5.1.10      
[179] GOSemSim_2.28.1           lazyeval_0.2.2           
[181] htmltools_0.5.8.1         enrichplot_1.22.0        
[183] GO.db_3.18.0              rappdirs_0.3.3           
[185] basilisk.utils_1.14.1     glue_1.8.0               
[187] RCurl_1.98-1.16           treeio_1.26.0            
[189] mclust_6.1.1              mnormt_2.1.1             
[191] quantiseqr_1.10.0         R6_2.5.1                 
[193] tidyr_1.3.1               gplots_3.2.0             
[195] labeling_0.4.3            GenomicFeatures_1.54.4   
[197] cluster_2.1.8             rngtools_1.5.2           
[199] Rhdf5lib_1.24.2           beanplot_1.3.1           
[201] aplot_0.2.4               DelayedArray_0.28.0      
[203] tidyselect_1.2.1          plotrix_3.8-4            
[205] ggforce_0.4.2             xml2_1.3.6               
[207] car_3.1-3                 rsvd_1.0.5               
[209] munsell_0.5.1             KernSmooth_2.23-26       
[211] nor1mix_1.3-3             ComplexHeatmap_2.18.0    
[213] htmlwidgets_1.6.4         fgsea_1.28.0             
[215] biomaRt_2.58.2            rlang_1.1.5              
[217] Cairo_1.6-2              </code></pre>
</div>
</div>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script data-goatcounter="https://19adc99.goatcounter.com/count" async="" src="//gc.zgo.at/count.js"></script>
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./genomics-and-epigenomics-analysis.html" class="pagination-link" aria-label="Genomics and Epigenomics Analysis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./survival-analysis.html" class="pagination-link" aria-label="Survival Analysis">
        <span class="nav-page-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>