<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Gather Datasets and QC | Multi-omics Integration for Target Discovery</title>
  <meta name="description" content="Multi-omics integration for target discovery in cancer" />
  <meta name="generator" content="bookdown 0.40 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Gather Datasets and QC | Multi-omics Integration for Target Discovery" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Multi-omics integration for target discovery in cancer" />
  <meta name="github-repo" content="19ADC99/cancer_code" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Gather Datasets and QC | Multi-omics Integration for Target Discovery" />
  
  <meta name="twitter:description" content="Multi-omics integration for target discovery in cancer" />
  

<meta name="author" content="Andrea Del Cortona" />


<meta name="date" content="2024-09-27" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="horizontal-integration.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Multi-omics for Target Discovery</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="gather-datasets-and-qc.html"><a href="gather-datasets-and-qc.html"><i class="fa fa-check"></i><b>1</b> Gather Datasets and QC</a>
<ul>
<li class="chapter" data-level="1.1" data-path="gather-datasets-and-qc.html"><a href="gather-datasets-and-qc.html#gather-resources"><i class="fa fa-check"></i><b>1.1</b> Gather resources</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="horizontal-integration.html"><a href="horizontal-integration.html"><i class="fa fa-check"></i><b>2</b> Horizontal integration</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Multi-omics Integration for Target Discovery</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="gather-datasets-and-qc" class="section level1 hasAnchor" number="1">
<h1><span class="header-section-number">Chapter 1</span> Gather Datasets and QC<a href="gather-datasets-and-qc.html#gather-datasets-and-qc" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="gather-resources" class="section level2 hasAnchor" number="1.1">
<h2><span class="header-section-number">1.1</span> Gather resources<a href="gather-datasets-and-qc.html#gather-resources" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We can gather cancer multi-omics datasets from large public data resources such as TCGA (<a href="https://www.cancer.gov/ccg/research/genome-sequencing/tcga">The Cancer Genome Atlas</a>) and the GDC (<a href="https://gdc.cancer.gov/">Genomic Data Commons</a>), as well as smaller-scale datasets generated by individual labs from the <a href="https://xena.ucsc.edu/">USDC Xena platform</a>. From the Xena platform, we can find the data from <a href="https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html">Pan-Cancer Atlas consortium</a>, which provides textbook high quality datasets that can be explored for target discovery in cancer.</p>
<p>TCGA PanCan collects 12,839 samples from 28 organs, 69 primary sites spanning 32 different cancer types.</p>
<p>The metadata available for the samples and the subjects include:</p>
<ul>
<li>biopsys type (metastatic, primary tumor, solid tissue normal) –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/TCGA_phenotype_denseDataOnlyDownload.tsv.gz">data</a></li>
<li>cancer subtype based on canonical molecular classification (methylation, miRNA, mRNA, proteins, etc.) –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/TCGASubtype.20170308.tsv.gz">data</a></li>
<li>cancer subtype based on the immune models –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/Subtype_Immune_Model_Based.txt.gz">data</a></li>
<li>overall survival information –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/Survival_SupplementalTable_S1_20171025_xena_sp">data</a></li>
</ul>
<p>For these samples, there are available rich clinical metadata and matched multiomics datasets.</p>
<p>We can retrieved the following multiomics readouts:</p>
<ul>
<li>copy number variants at the gene level –&gt; <a href="https://tcga-xena-hub.s3.us-east-1.amazonaws.com/download/TCGA.PANCAN.sampleMap%2FGistic2_CopyNumber_Gistic2_all_thresholded.by_genes.gz">data</a></li>
<li>DNA methylation data –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/jhu-usc.edu_PANCAN_HumanMethylation450.betaValue_whitelisted.tsv.synapse_download_5096262.xena.gz">data</a></li>
<li>gene expression (RNAseq) –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/EB%2B%2BAdjustPANCAN_IlluminaHiSeq_RNASeqV2.geneExp.xena.gz">data</a></li>
<li>micro RNAs (miRNA) –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/pancanMiRs_EBadjOnProtocolPlatformWithoutRepsWithUnCorrectMiRs_08_04_16.xena.gz">data</a></li>
<li>protein expression –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/TCGA-RPPA-pancan-clean.xena.gz">data</a></li>
<li>somatic mutations –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/mc3.v0.2.8.PUBLIC.nonsilentGene.xena.gz">data</a></li>
</ul>
<p>All the multiomics readouts are at the gene level, except for the DNA methylation data, that will have to be mapped and normalized at the gene level. Luckily, a <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/probeMap%2FilluminaMethyl450_hg19_GPL16304_TCGAlegacy">mapping file</a> is available as well.</p>
<p>Xena provides as well already precomputed enrichments and cell types deconvolutions:</p>
<ul>
<li>gene programs that are canonical drug targets –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/Pancan12_GenePrograms_drugTargetCanon_in_Pancan33.tsv.gz">data</a></li>
<li>homologous recombination deficiency (HRD) –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/TCGA.HRD_withSampleID.txt.gz">data</a></li>
<li>immune signaling –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/TCGA_pancancer_10852whitelistsamples_68ImmuneSigs.xena.gz">data</a></li>
<li>ssGSEA PARADIGM annotations –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/PanCan33_ssGSEA_1387GeneSets_NonZero_sample_level.txt.gz">data</a></li>
<li>stemness score based on DNA methylation signals –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/StemnessScores_DNAmeth_20170210.tsv.gz">data</a></li>
<li>stemnsess score based on the RNAseq expression of 103 key genes –&gt; <a href="https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/StemnessScores_RNAexp_20170127.2.tsv.gz">data</a></li>
</ul>
<p>I assure you that integrating all these data will be fun, and it will provide deep insights on each cancer we decided to analyse.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="gather-datasets-and-qc.html#cb1-1" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">&quot;hello!!!</span><span class="sc">\n</span><span class="st">&quot;</span>)</span></code></pre></div>
<pre><code>## hello!!!</code></pre>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="horizontal-integration.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/01-gather-datasets-and-QC.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
