<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>4&nbsp; micro-RNAs Analysis – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./genomics-and-epigenomics-analysis.html" rel="next">
<link href="./proteomics-analysis.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-fixedcolumns-1.13.6/css/fixedColumns.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedcolumns-1.13.6/js/dataTables.fixedColumns.min.js"></script>
<link href="site_libs/dt-ext-fixedheader-1.13.6/css/fixedHeader.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedheader-1.13.6/js/dataTables.fixedHeader.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./micro-RNAs-analysis.html"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./survival-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./kidney-carcinomas-lessons-learnt.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">kidney-carcinomas-lessons-learnt.html</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell Cancer molecular subtypes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cancer-signatures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">cancer-signatures.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./autoencoder.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">autoencoder.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cellular-deconvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Cellular Deconvolution from Bulk RNA-seq</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">4.1</span> On this page</a></li>
  <li><a href="#micro-rnas-data-overview-and-qc" id="toc-micro-rnas-data-overview-and-qc" class="nav-link" data-scroll-target="#micro-rnas-data-overview-and-qc"><span class="header-section-number">4.2</span> micro-RNAs data overview and QC</a>
  <ul>
  <li><a href="#micro-rnas-counts-overview" id="toc-micro-rnas-counts-overview" class="nav-link" data-scroll-target="#micro-rnas-counts-overview"><span class="header-section-number">4.2.1</span> micro-RNAs counts overview</a></li>
  </ul></li>
  <li><a href="#dimesionality-reduction-and-dataset-exploration" id="toc-dimesionality-reduction-and-dataset-exploration" class="nav-link" data-scroll-target="#dimesionality-reduction-and-dataset-exploration"><span class="header-section-number">4.3</span> Dimesionality Reduction and Dataset Exploration</a>
  <ul>
  <li><a href="#umap-on-micro-rnas-data" id="toc-umap-on-micro-rnas-data" class="nav-link" data-scroll-target="#umap-on-micro-rnas-data"><span class="header-section-number">4.3.1</span> UMAP on micro-RNAs data</a></li>
  <li><a href="#principal-component-analysis-pca" id="toc-principal-component-analysis-pca" class="nav-link" data-scroll-target="#principal-component-analysis-pca"><span class="header-section-number">4.3.2</span> Principal Component Analysis (PCA)</a></li>
  </ul></li>
  <li><a href="#differential-gene-expression-analysis" id="toc-differential-gene-expression-analysis" class="nav-link" data-scroll-target="#differential-gene-expression-analysis"><span class="header-section-number">4.4</span> Differential gene expression analysis</a>
  <ul>
  <li><a href="#identification-of-differentially-expressed-micro-rnas" id="toc-identification-of-differentially-expressed-micro-rnas" class="nav-link" data-scroll-target="#identification-of-differentially-expressed-micro-rnas"><span class="header-section-number">4.4.1</span> Identification of differentially expressed micro-RNAs</a></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-micro-RNAs" id="toc-sec-lesson-learnt-micro-RNAs" class="nav-link" data-scroll-target="#sec-lesson-learnt-micro-RNAs"><span class="header-section-number">4.5</span> Lessons Learnt</a></li>
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information"><span class="header-section-number">4.6</span> Session Information</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./micro-RNAs-analysis.html"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="4.1">
<h2 data-number="4.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">4.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at Lesson Learnt: <a href="#sec-lesson-learnt-micro-RNAs" class="quarto-xref"><span>Section 4.5</span></a>.</p>
<ul>
<li>Here we investigate the micro-RNAs data across the three Kidney Cancers.</li>
<li>First we filter lowly expressed micro-RNAs and we do some exploratory analyses on samples, micro-RNAs expression and clinical covariates.</li>
<li>We then run a formal Differential Gene Expression analysis to identify micro-RNAs that have different expression levels across the three Kidney cancer types.</li>
</ul>
</section>
<section id="micro-rnas-data-overview-and-qc" class="level2" data-number="4.2">
<h2 data-number="4.2" class="anchored" data-anchor-id="micro-rnas-data-overview-and-qc"><span class="header-section-number">4.2</span> micro-RNAs data overview and QC</h2>
<section id="micro-rnas-counts-overview" class="level3" data-number="4.2.1">
<h3 data-number="4.2.1" class="anchored" data-anchor-id="micro-rnas-counts-overview"><span class="header-section-number">4.2.1</span> micro-RNAs counts overview</h3>
<p>For TCGA Kidney cancer micro-RNAs data, we will perform analyses similar to the ones performed for the transcriptomics one (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>) and proteomics (see <a href="proteomics-analysis.html" class="quarto-xref"><span>Chapter 3</span></a>). We first start with an overview of the datasets and QC.</p>
<p>Let’s check the gene counts distributions across samples. We have counts information for 846 samples and a total of 743 micro-RNAs. There are no missing-values, so the count matrix has 100% occupancy. Looking at the counts distributions across the three kidney carcinomas, we observe an over-abundance of micro-RNAs with an average expression of 0 across the samples.</p>
<p>To best capture the biological signal, it would be beneficial to discard the micro-RNAs that have an expression of zero across all or most of the samples, so that we can focus on the ones that were expressed in the biopsies of the patients with kidney carcinomas.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 1: micro-RNAs expression distributions before filtering.</figcaption>
</figure>
</div>
</div>
</div>
<p>Filtering of lowly expressed micro-RNAs resulted in the exclusion of 553 of the 743 micro-RNAs.</p>
<p>In Figure 2 we can see the expression profiles across the kidney carcinomas samples of the 190 remaining micro-RNAs. Now, the average (log2) expression across samples is around 5.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 2: micro-RNAs expression distributions after filtering.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="dimesionality-reduction-and-dataset-exploration" class="level2" data-number="4.3">
<h2 data-number="4.3" class="anchored" data-anchor-id="dimesionality-reduction-and-dataset-exploration"><span class="header-section-number">4.3</span> Dimesionality Reduction and Dataset Exploration</h2>
<section id="umap-on-micro-rnas-data" class="level3" data-number="4.3.1">
<h3 data-number="4.3.1" class="anchored" data-anchor-id="umap-on-micro-rnas-data"><span class="header-section-number">4.3.1</span> UMAP on micro-RNAs data</h3>
<p>Let’s check the samples clustering on a UMAP. The UMAP based on micro-RNAs data has a worse clustering of the samples to the different kidney carcinomas when compared to the UMAP based on the transcriptomics data (see <a href="transcriptomics-analysis.html#sec-UMAP-transcriptomics" class="quarto-xref"><span>Section 2.3.1</span></a>) and the UMAP based on the proteomics data (see <a href="proteomics-analysis.html#sec-UMAP-proteomics" class="quarto-xref"><span>Section 3.3.1</span></a>). Many KIRC samples in fact are interspersed between the KIRP cluster and the KIRC cluster, suggesting a more complex relationship between micro-RNAs expression and the different kidney carcinomas classification.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 3: Kidney micro-RNAs UMAP.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="principal-component-analysis-pca" class="level3" data-number="4.3.2">
<h3 data-number="4.3.2" class="anchored" data-anchor-id="principal-component-analysis-pca"><span class="header-section-number">4.3.2</span> Principal Component Analysis (PCA)</h3>
<p>As we did for the transcriptomics data (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>) and the proteomics one (see <a href="proteomics-analysis.html" class="quarto-xref"><span>Chapter 3</span></a>), the next step in the dataset exploration is to perform the Principal Component Analysis.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 4: PCA, exploration.</figcaption>
</figure>
</div>
</div>
</div>
<p>The first 19 Principal Components capture more than 80% of the variance in the Kidney cancers micro-RNAs datasets, with the first two components (PC1 and PC2) capturing almost 40% of the variance.</p>
<p>When we project the samples in the PC1 and PC2, we can see that the PC1 has a gradient of samples that go from KIRC to KIRP to KICH, which overlaps with the tail of KIRP samples. The second component PC2, instead, separated KIRC and KIRP from KICH. The first two principal components seems to separate the thee types of kidney carcinomas.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 5: PCA, biplot PC1 x PC2.</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 6: PCA, pairs plots.</figcaption>
</figure>
</div>
</div>
</div>
<p>Let’s check the loadings (i.e.: 19 Principal Components capture more than 80% of the variance in the) for the top 4 Principal Components. These indicate which micro-RNAs are the more responsible to explain the position of the samples along the components, and the direction of this separation.</p>
<p>Looking at the top most variable genes, the following 8 micro-RNAs are the top loadings for the first 4 Principal Components:</p>
<ol type="1">
<li><em>hsa-miR-10b-5p</em></li>
<li><em>hsa-miR-126-3p</em></li>
<li><em>hsa-miR-143-5p</em></li>
<li><em>hsa-miR-144-5p</em></li>
<li><em>hsa-miR-183-5p</em></li>
<li><em>hsa-miR-200c-3p</em></li>
<li><em>hsa-miR-204-5p</em></li>
<li><em>hsa-miR-9-5p</em></li>
</ol>
<p>Let’s now check the expression of the five top genes identified with the PCA across the cancer types:</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 7: PCA, loadings.</figcaption>
</figure>
</div>
</div>
</div>
<p>The miR-200 family is suggested to represent a <a href="https://pubmed.ncbi.nlm.nih.gov/31635993/">good prognostic marker in clear cell renal cell carcinoma</a>. In our analysis, hsa-miR-200c-3p seems to be overexpressed in KICH when compared to KIRC and KIRP. Lower expression of hsa-miR-200c-3p are associated with worse prognosis and carcinomas, in line with the better prognosis for KICH patients observed in our previous analysis (see <a href="dataset-exploration.html#sec-dataset-exploration" class="quarto-xref"><span>Section 1.3.7</span></a>).</p>
<p>Let’s check the Pearson correlation with other clinical covariates.</p>
<p><em>Cancer type</em> correlates with PC2, that, as we saw in the biplot in Figure 5, clearly separates KIRC and KIRP from KICH. <em>Subtype miRNA</em> and most of the other molecular subtypes assigned by TCGA strongly correlated with PC1 and PC2. <em>Subtype miRNA</em> also correlates with PC5. We observed little to no correlation with <em>vital status</em> or residual disease (<em>follow up tumor status</em>).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 8: PCA, correlations between clinical covariates and Principal Components.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="differential-gene-expression-analysis" class="level2" data-number="4.4">
<h2 data-number="4.4" class="anchored" data-anchor-id="differential-gene-expression-analysis"><span class="header-section-number">4.4</span> Differential gene expression analysis</h2>
<p>In addition to cancer type, we saw that age, ethnicity (and race) and age had somewhat a correlation with the cancer types.</p>
<p>We may want to include this covariates in the differential gene expression analysis in order to include their contribution into the model.</p>
<p>For micro-RNAs analyses, we use two arbitrary thresholds to retain genes that are significantly differentially expressed across the each comparison: an absolute logFold-Change (logFC) higher than 1, and an adjusted p-value lower than 0.05. This results in:</p>
<ul>
<li>KIRC_vs_KICH, 157 differentially expressed micro-RNAs, 118 upregulated and 49 downregulated</li>
<li>KIRP_vs_KICH, 155 differentially expressed micro-RNAs, 87 upregulated and 68 downregulated</li>
<li>KIRC_vs_KIRP, 163 differentially expressed micro-RNAs, 105 upregulated and 55 downregulated</li>
</ul>
<p>As usual, we can visualize the logFC p-values relationships for all the micro-RNAs with a volcano plot.</p>
<section id="identification-of-differentially-expressed-micro-rnas" class="level3" data-number="4.4.1">
<h3 data-number="4.4.1" class="anchored" data-anchor-id="identification-of-differentially-expressed-micro-rnas"><span class="header-section-number">4.4.1</span> Identification of differentially expressed micro-RNAs</h3>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 9: Volcano plots of each contrats reporting the micro-RNAs differentially expressed.</figcaption>
</figure>
</div>
</div>
</div>
<p>We can also check the logFC and adjusted p-value for the top 8 micro-RNAs that are the top loadings for the first 4 Principal Components and see how these micro-RNAs were differentially expressed across the comparisons.</p>
<p>We can confirm what we have naively observed at the level of micro-RNAs expression in Figure 7 while investigating the PCA loadings: <em>hsa-miR-200c-3p</em> and <em>hsa-miR-204c-5p</em> are the top loadings for PC2, which separates KIRC and KIRP from KICH, and indeed, these micro-RNAs are the most differentially expressed in the comparisons of KIRC and KIRP versus KICH, with <em>hsa-miR-200c-3p</em> strongly upregulated in KICH (&gt; 9 fold difference), while <em>hsa-miR-204c-5p</em> is strongly downregulates in KICH (&gt;5 fold difference).</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-be88325ca1307854b84b" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-be88325ca1307854b84b">{"x":{"filter":"none","vertical":false,"class":"display","extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 1: top 8 loadings across DGE comparisons.<\/caption>","data":[["hsa-miR-200c-3p","hsa-miR-183-5p","hsa-miR-204-5p","hsa-miR-126-3p","hsa-miR-143-5p","hsa-miR-10b-5p","hsa-miR-9-5p","hsa-miR-144-5p"],[-9.529999999999999,-4.42,5.2,2.16,2.2,0.96,1.4,0.05],[8.62e-190,5.57e-78,7.01e-71,2.45e-49,9.960000000000001e-39,1.11e-14,4.45e-05,0.85],[-9.140000000000001,-2.19,6.24,-1.64,-1.2,-0.93,1.13,-0.97],[2.76e-169,1.82e-21,1.29e-87,5.21e-28,3.34e-12,1.02e-12,0.00174,0.000152],[-0.39,-2.23,-1.04,3.8,3.39,1.89,0.26,1.02],[0.00674,2.49e-63,1.54e-11,1.09e-245,2.18e-178,8.18e-117,0.19,3.28e-13]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th rowspan=\"2\">Gene<\/th>\n      <th colspan=\"2\">KIRC_vs_KICH<\/th>\n      <th colspan=\"2\">KIRP_vs_KICH<\/th>\n      <th colspan=\"2\">KIRC_vs_KIRP<\/th>\n    <\/tr>\n    <tr>\n      <th>logFC<\/th>\n      <th>adj. p-value<\/th>\n      <th>logFC<\/th>\n      <th>adj. p-value<\/th>\n      <th>logFC<\/th>\n      <th>adj. p-value<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"micro.RNA","targets":0},{"name":"KIRC_vs_KICH.logFC","targets":1},{"name":"KIRC_vs_KICH.adj..p.value","targets":2},{"name":"KIRP_vs_KICH.logFC","targets":3},{"name":"KIRP_vs_KICH.adj..p.value","targets":4},{"name":"KIRC_vs_KIRP.logFC","targets":5},{"name":"KIRC_vs_KIRP.adj..p.value","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>The UpSet plot shows that KICH has a strong signature of 30 micro-RNAs that are differentially abundant when compared to KIRC or KIRP, including <em>hsa-miR-34a-5p</em>, <em>hsa-miR-146a-5p</em>, <em>hsa-miR-146b-5p</em>, <em>hsa-miR-181a-3p</em>, <em>hsa-miR-204-5p</em> and <em>hsa-miR-200c-3p</em>. KIRC, instead, has a signature of 20 micro-RNAs, including <em>hsa-miR-155-5p</em>.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="micro-RNAs-analysis_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 10: UpSet reporting the micro-RNAs differentially expressed upregulated and downregulated in common across all contrasts.</figcaption>
</figure>
</div>
</div>
</div>
<p>Tables of differentially expressed micro-RNAs</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">KIRC vs KICH</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">KIRP vs KICH</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">KIRC vs KIRP</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-8b57fe923d67980056aa" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-8b57fe923d67980056aa">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 2: Differentially expressed micro-RNAs in KIRC vs KICH comparison.<\/caption>","data":[["hsa-miR-200c-3p","hsa-miR-221-3p","hsa-miR-200b-3p","hsa-miR-200a-5p","hsa-miR-455-5p","hsa-miR-140-5p","hsa-miR-455-3p","hsa-miR-222-3p","hsa-miR-210-3p","hsa-miR-21-5p","hsa-miR-374a-3p","hsa-miR-26b-5p","hsa-miR-24-2-5p","hsa-miR-338-3p","hsa-miR-183-5p","hsa-miR-29a-3p","hsa-miR-10a-5p","hsa-miR-130a-3p","hsa-miR-342-3p","hsa-miR-204-5p","hsa-miR-98-5p","hsa-miR-186-5p","hsa-miR-200a-3p","hsa-miR-19b-3p","hsa-miR-429","hsa-miR-182-5p","hsa-miR-30a-3p","hsa-miR-203a","hsa-miR-155-5p","hsa-miR-26b-3p","hsa-miR-505-3p","hsa-miR-194-5p","hsa-miR-192-5p","hsa-miR-30c-2-3p","hsa-miR-484","hsa-miR-145-3p","hsa-miR-126-3p","hsa-miR-532-5p","hsa-miR-181b-5p","hsa-miR-181a-3p","hsa-miR-34a-5p","hsa-miR-29b-2-5p","hsa-miR-15a-5p","hsa-miR-195-5p","hsa-miR-423-5p","hsa-miR-320a","hsa-let-7i-5p","hsa-miR-142-3p","hsa-miR-197-3p","hsa-miR-143-5p","hsa-miR-375","hsa-miR-145-5p","hsa-miR-181a-5p","hsa-miR-19a-3p","hsa-miR-23a-3p","hsa-miR-324-5p","hsa-miR-452-5p","hsa-miR-2355-5p","hsa-miR-423-3p","hsa-miR-146b-5p","hsa-miR-30d-5p","hsa-miR-365a-3p","hsa-miR-151a-5p","hsa-miR-365b-3p","hsa-let-7a-3p","hsa-miR-574-3p","hsa-miR-16-5p","hsa-miR-223-3p","hsa-miR-21-3p","hsa-miR-582-3p","hsa-miR-140-3p","hsa-miR-589-5p","hsa-miR-30a-5p","hsa-miR-30b-5p","hsa-miR-146a-5p","hsa-miR-1287-5p","hsa-miR-199a-5p","hsa-miR-660-5p","hsa-miR-30e-3p","hsa-miR-28-5p","hsa-miR-125a-5p","hsa-miR-22-3p","hsa-miR-28-3p","hsa-miR-450b-5p","hsa-miR-199a-3p","hsa-miR-199b-3p","hsa-miR-134-5p","hsa-miR-127-5p","hsa-miR-193a-5p","hsa-miR-328-3p","hsa-miR-30c-5p","hsa-miR-27b-3p","hsa-miR-653-5p","hsa-miR-542-3p","hsa-miR-142-5p","hsa-miR-103a-3p","hsa-miR-27a-3p","hsa-miR-196a-5p","hsa-miR-532-3p","hsa-miR-339-5p","hsa-miR-185-5p","hsa-miR-10b-5p","hsa-miR-132-3p","hsa-miR-126-5p","hsa-let-7c-5p","hsa-miR-92a-3p","hsa-miR-106b-5p","hsa-miR-374a-5p","hsa-miR-374b-5p","hsa-miR-148b-3p","hsa-miR-151a-3p","hsa-let-7g-3p","hsa-miR-218-5p","hsa-miR-708-3p","hsa-miR-152-3p","hsa-miR-99b-3p","hsa-miR-181c-5p","hsa-miR-424-5p","hsa-miR-143-3p","hsa-miR-324-3p","hsa-miR-101-3p","hsa-miR-29c-5p","hsa-miR-99b-5p","hsa-miR-652-3p","hsa-miR-379-5p","hsa-let-7d-5p","hsa-miR-20a-5p","hsa-miR-29c-3p","hsa-miR-30e-5p","hsa-miR-139-5p","hsa-miR-15b-5p","hsa-miR-27b-5p","hsa-miR-196b-5p","hsa-miR-744-5p","hsa-miR-17-3p","hsa-miR-146b-3p","hsa-miR-191-5p","hsa-miR-625-3p","hsa-miR-486-5p","hsa-miR-181c-3p","hsa-let-7b-5p","hsa-miR-24-3p","hsa-miR-17-5p","hsa-miR-9-5p","hsa-miR-664a-3p","hsa-miR-32-5p","hsa-miR-125b-5p","hsa-miR-361-5p","hsa-miR-361-3p","hsa-let-7b-3p","hsa-miR-150-5p","hsa-let-7i-3p","hsa-miR-127-3p","hsa-miR-92b-3p","hsa-miR-107","hsa-let-7e-5p","hsa-miR-1307-5p","hsa-miR-1307-3p","hsa-miR-500a-3p","hsa-miR-425-5p","hsa-miR-10b-3p","hsa-miR-106b-3p","hsa-miR-331-3p","hsa-miR-3607-3p","hsa-miR-378a-3p","hsa-miR-99a-5p","hsa-miR-181a-2-3p","hsa-miR-100-5p","hsa-miR-501-3p","hsa-miR-193b-3p","hsa-miR-598-3p","hsa-miR-125b-2-3p","hsa-let-7d-3p","hsa-miR-128-3p","hsa-miR-93-5p","hsa-miR-584-5p","hsa-let-7f-5p","hsa-miR-29b-3p","hsa-miR-148a-3p","hsa-miR-629-5p","hsa-miR-451a","hsa-miR-20b-5p","hsa-let-7g-5p","hsa-miR-26a-5p","hsa-miR-25-3p","hsa-miR-144-5p","hsa-miR-23b-3p","hsa-miR-22-5p","hsa-miR-378a-5p","hsa-let-7a-5p"],[-9.535,-4.069,-4.033,-3.736,3.378,2.265,3.921,-3.064,4.133,2.568,2.004,1.952,2.103,2.555,-4.415,-1.687,-2.625,2.091,2.257,5.2,1.529,1.494,-2.578,1.717,-2.549,-3.628,-1.812,-4.362,3.127,1.562,1.645,3.845,4.108,-1.849,1.522,2.227,2.158,-1.527,1.572,1.498,1.767,1.555,1.204,1.854,1.295,1.615,1.072,2.356,1.391,2.196,-3.619,2.183,1.303,1.419,1.045,1.195,2.005,1.231,0.868,2.179,-0.908,1.435,0.826,1.434,1.03,1.121,1.09,1.705,1.47,-2.135,0.929,-0.867,-1.019,-1.038,1.477,0.914,1.988,-1.149,-0.741,0.645,0.803,-0.796,0.6820000000000001,1.471,1.748,1.746,1.766,1.651,1.013,0.953,0.906,-0.758,1.981,1.349,1.289,0.6870000000000001,0.747,1.113,-1.006,0.985,-0.836,0.961,0.6,0.974,1.02,-0.618,0.622,0.586,0.717,0.5610000000000001,-0.508,-0.8,-1.196,1.192,0.829,0.5620000000000001,0.8149999999999999,1.17,1.052,-0.617,-0.584,0.653,0.527,-0.516,1.076,0.447,0.545,-0.5659999999999999,-0.486,0.864,-0.5570000000000001,0.469,1.062,0.47,-0.384,0.884,-0.47,0.593,-1.039,0.5620000000000001,0.496,0.335,0.38,1.395,0.397,0.351,-0.464,-0.222,0.344,-0.316,0.6,0.34,0.582,-0.331,-0.228,0.265,0.371,-0.271,-0.283,0.312,0.264,-0.234,-0.242,0.378,0.321,0.295,0.195,-0.331,0.273,-0.352,-0.232,0.199,-0.133,-0.144,0.116,-0.201,0.176,-0.13,0.143,0.097,-0.182,0.105,0.034,0.018,-0.017,0.05,0.014,-0.017,-0.013,-0.006],[5.931,6.76,8.129,7.628,5.351,5.979,8.430999999999999,5.435,10.352,17.506,9.413,9.039,4.028,7.524,10.37,12.34,15.047,6.02,6.484,7.457,5.639,7.736,7.115,6.138,5.258,12.161,15.031,7.962,7.983,4.434,5.087,10.379,11.818,7.598,5.838,4.507,11.847,9.619,8.026,7.169,7.724,5.149,6.954,4.708,5.712,8.542,8.48,10.546,7.696,5.37,5.704,9.83,10.295,3.973,11.285,4.058,4.975,5.478,6.617,8.538,13.347,5.408,6.181,5.408,4.144,5.375,8.802,6.618,10.183,7.593,10.284,5.669,16.997,8.999000000000001,6.352,3.854,7.06,5.335,12.73,6.409,9.327999999999999,16.007,11.611,4.143,8.429,8.423,5.431,5.011,7.465,5.827,10.658,9.946,5.045,7.893,5.417,13.548,9.550000000000001,8.212,5.255,4.543,5.69,16.513,5.763,8.103,10.62,12.349,7.3,4.779,5.838,7.288,10.596,5.018,4.654,5.076,7.534,5.353,4.85,7.29,15.043,3.87,13.585,5.306,14.606,4.035,6.199,7.699,7.415,10.394,13.143,5.682,6.826,3.74,9.593,4.422,7.122,5.777,8.616,6.607,7.179,4.597,13.256,10.054,7.999,8.113,4.96,4.481,8.669,7.428,6.522,4.349,7.958,5.731,6.024,5.602,5.72,9.930999999999999,8.311999999999999,9.314,7.887,6.242,4.696,7.273,4.531,6.487,7.734,9.132999999999999,9.414,11.378,5.582,4.933,4.116,3.975,8.045999999999999,6.407,11.487,5.359,12.617,8.994999999999999,14.106,5.788,9.077,4.289,9.007999999999999,10.346,12.218,7.396,10.179,4.249,5.504,14.373],[-39.168,-30.965,-30.943,-28.329,27.396,27.333,24.057,-23.867,23.769,23.165,22.696,22.557,21.977,21.103,-20.997,-20.043,-19.956,19.859,19.833,19.782,19.553,18.75,-18.713,18.553,-18.14,-17.795,-17.716,-17.618,17.277,17.001,16.998,16.436,16.356,-16.322,16.232,16.087,15.911,-15.261,15.014,14.968,14.933,14.908,14.302,14.272,14.225,14.137,14.136,14.1,14.009,13.819,-13.71,13.698,13.303,13.089,13.005,12.794,12.494,12.388,11.96,11.624,-11.517,11.385,11.385,11.379,11.368,11.274,11.005,10.934,10.835,-10.78,10.609,-10.597,-10.567,-10.539,10.528,10.128,10.112,-9.856,-9.640000000000001,9.581,9.574,-9.455,9.285,9.273999999999999,9.199999999999999,9.17,9.115,9.061,9.052,8.864000000000001,8.773999999999999,-8.692,8.683999999999999,8.635999999999999,8.635,8.516999999999999,8.5,8.340999999999999,-8.234999999999999,8.063000000000001,-8.005000000000001,7.95,7.931,7.86,7.637,-7.635,7.58,7.512,7.486,7.441,-7.391,-7.247,-7.208,7.169,7.141,6.938,6.809,6.726,6.547,-6.53,-6.454,6.234,6.006,-5.847,5.768,5.765,5.729,-5.665,-5.6,5.433,-5.387,5.198,5.13,5.037,-5.002,4.93,-4.884,4.794,-4.718,4.687,4.674,4.513,4.444,4.169,4.096,3.998,-3.689,-3.586,3.549,-3.543,3.429,3.242,3.233,-3.075,-2.889,2.71,2.678,-2.552,-2.529,2.479,2.428,-2.406,-2.383,2.228,2.163,2.055,2.035,-1.942,1.937,-1.921,-1.562,1.521,-1.487,-1.439,1.333,-1.325,1.219,-1.195,1.176,1.038,-0.731,0.6820000000000001,0.412,0.266,-0.233,0.212,0.163,-0.146,-0.09,-0.063],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.002,0.004,0.007,0.008,0.011,0.012,0.013,0.015,0.016,0.017,0.026,0.031,0.04,0.042,0.052,0.053,0.055,0.119,0.129,0.137,0.15,0.183,0.185,0.223,0.232,0.24,0.3,0.465,0.495,0.681,0.79,0.8159999999999999,0.832,0.871,0.884,0.928,0.949],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.002,0.002,0.003,0.005,0.008,0.008999999999999999,0.013,0.014,0.016,0.018,0.019,0.02,0.03,0.036,0.046,0.048,0.059,0.06,0.062,0.132,0.142,0.151,0.164,0.199,0.2,0.24,0.248,0.255,0.316,0.488,0.517,0.707,0.8159999999999999,0.838,0.85,0.885,0.894,0.9330000000000001,0.949],[428.948,312.157,311.833,273.9,260.354,259.442,212.249,209.54,208.146,199.573,192.949,190.986,182.851,170.706,169.24,156.186,155.006,153.691,153.341,152.646,149.553,138.834,138.346,136.232,130.812,126.336,125.313,124.051,119.676,116.168,116.121,109.073,108.078,107.648,106.538,104.75,102.58,94.717,91.77200000000001,91.23399999999999,90.815,90.521,83.45699999999999,83.10599999999999,82.571,81.55800000000001,81.548,81.137,80.105,77.956,76.724,76.596,72.208,69.86199999999999,68.95099999999999,66.678,63.493,62.373,57.936,54.54,53.464,52.162,52.158,52.097,51.994,51.064,48.455,47.775,46.826,46.306,44.692,44.586,44.298,44.043,43.941,40.266,40.129,37.841,35.943,35.437,35.371,34.353,32.91,32.819,32.193,31.945,31.487,31.036,30.967,29.424,28.7,28.044,27.981,27.597,27.588,26.657,26.523,25.29,24.474,23.176,22.741,22.333,22.195,21.673,20.056,20.045,19.654,19.173,18.988,18.671,18.324,17.333,17.066,16.807,16.614,15.268,14.425,13.89,12.761,12.657,12.187,10.856,9.516,8.613,8.172000000000001,8.153,7.953,7.605,7.25,6.361,6.12,5.153,4.814,4.351,4.182,3.837,3.618,3.194,2.837,2.698,2.637,1.916,1.613,0.455,0.16,-0.229,-1.396,-1.766,-1.898,-1.918,-2.31,-2.925,-2.954,-3.447,-3.996,-4.492,-4.579,-4.905,-4.962,-5.088,-5.212,-5.263,-5.32,-5.674,-5.817,-6.042,-6.083,-6.268,-6.278,-6.309,-6.932,-6.994,-7.046,-7.115,-7.262,-7.272,-7.408,-7.436,-7.458,-7.612,-7.882,-7.917,-8.065,-8.115,-8.122999999999999,-8.127000000000001,-8.137,-8.138999999999999,-8.146000000000001,-8.148]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>microRNA<\/th>\n      <th>logFC<\/th>\n      <th>AveExpr<\/th>\n      <th>t<\/th>\n      <th>P.Value<\/th>\n      <th>adj.P.Val<\/th>\n      <th>B<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"microRNA","targets":0},{"name":"logFC","targets":1},{"name":"AveExpr","targets":2},{"name":"t","targets":3},{"name":"P.Value","targets":4},{"name":"adj.P.Val","targets":5},{"name":"B","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-56dc6409056b7babdaa7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-56dc6409056b7babdaa7">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 3: Differentially expressed micro-RNAs KIRP vs KICH comparison.<\/caption>","data":[["hsa-miR-200c-3p","hsa-miR-21-5p","hsa-miR-222-3p","hsa-miR-455-5p","hsa-miR-221-3p","hsa-miR-204-5p","hsa-miR-203a","hsa-miR-582-3p","hsa-miR-30a-3p","hsa-miR-200a-5p","hsa-miR-374a-3p","hsa-miR-34a-5p","hsa-miR-19b-3p","hsa-let-7b-3p","hsa-miR-423-3p","hsa-miR-455-3p","hsa-miR-126-5p","hsa-miR-29a-3p","hsa-miR-132-3p","hsa-miR-200b-3p","hsa-miR-130a-3p","hsa-miR-98-5p","hsa-miR-1287-5p","hsa-miR-24-2-5p","hsa-miR-10a-5p","hsa-miR-181c-3p","hsa-miR-140-5p","hsa-miR-181c-5p","hsa-miR-324-5p","hsa-miR-103a-3p","hsa-miR-146a-5p","hsa-miR-181a-5p","hsa-miR-423-5p","hsa-miR-30e-3p","hsa-miR-181b-5p","hsa-miR-21-3p","hsa-miR-744-5p","hsa-miR-20a-5p","hsa-miR-26b-5p","hsa-miR-181a-3p","hsa-miR-27a-3p","hsa-miR-126-3p","hsa-miR-505-3p","hsa-miR-223-3p","hsa-miR-23a-3p","hsa-miR-140-3p","hsa-miR-194-5p","hsa-miR-28-5p","hsa-miR-146b-5p","hsa-miR-342-3p","hsa-miR-30b-5p","hsa-miR-532-5p","hsa-miR-19a-3p","hsa-miR-183-5p","hsa-let-7g-3p","hsa-miR-210-3p","hsa-miR-625-3p","hsa-miR-151a-3p","hsa-miR-30e-5p","hsa-miR-375","hsa-miR-197-3p","hsa-let-7i-3p","hsa-miR-22-3p","hsa-miR-192-5p","hsa-miR-424-5p","hsa-miR-106b-5p","hsa-miR-10b-3p","hsa-miR-339-5p","hsa-miR-629-5p","hsa-miR-101-3p","hsa-miR-182-5p","hsa-let-7i-5p","hsa-miR-200a-3p","hsa-miR-15a-5p","hsa-miR-29c-3p","hsa-miR-328-3p","hsa-let-7b-5p","hsa-miR-186-5p","hsa-miR-15b-5p","hsa-miR-10b-5p","hsa-miR-30a-5p","hsa-miR-151a-5p","hsa-miR-128-3p","hsa-miR-143-5p","hsa-miR-148a-3p","hsa-miR-143-3p","hsa-miR-1307-3p","hsa-miR-374a-5p","hsa-miR-320a","hsa-miR-30c-2-3p","hsa-miR-532-3p","hsa-miR-155-5p","hsa-miR-139-5p","hsa-miR-429","hsa-miR-338-3p","hsa-miR-17-3p","hsa-miR-374b-5p","hsa-miR-100-5p","hsa-miR-331-3p","hsa-miR-652-3p","hsa-let-7c-5p","hsa-miR-146b-3p","hsa-miR-93-5p","hsa-miR-664a-3p","hsa-miR-484","hsa-miR-30c-5p","hsa-miR-3607-3p","hsa-miR-584-5p","hsa-miR-152-3p","hsa-let-7d-3p","hsa-miR-24-3p","hsa-miR-193a-5p","hsa-miR-17-5p","hsa-miR-125a-5p","hsa-miR-486-5p","hsa-miR-16-5p","hsa-miR-29c-5p","hsa-miR-29b-2-5p","hsa-miR-361-3p","hsa-miR-29b-3p","hsa-miR-361-5p","hsa-miR-106b-3p","hsa-miR-199a-5p","hsa-miR-142-3p","hsa-miR-28-3p","hsa-miR-125b-2-3p","hsa-miR-196a-5p","hsa-miR-145-5p","hsa-miR-30d-5p","hsa-miR-144-5p","hsa-miR-142-5p","hsa-let-7d-5p","hsa-let-7a-3p","hsa-miR-365a-3p","hsa-miR-365b-3p","hsa-miR-99b-5p","hsa-miR-218-5p","hsa-miR-99a-5p","hsa-miR-451a","hsa-miR-185-5p","hsa-miR-9-5p","hsa-miR-23b-3p","hsa-miR-598-3p","hsa-miR-26a-5p","hsa-miR-191-5p","hsa-miR-92a-3p","hsa-miR-1307-5p","hsa-let-7f-5p","hsa-miR-196b-5p","hsa-miR-125b-5p","hsa-let-7a-5p","hsa-miR-26b-3p","hsa-miR-574-3p","hsa-miR-708-3p","hsa-miR-378a-5p","hsa-miR-27b-3p","hsa-miR-378a-3p","hsa-let-7g-5p","hsa-miR-32-5p","hsa-miR-324-3p","hsa-miR-589-5p","hsa-miR-20b-5p","hsa-miR-193b-3p","hsa-miR-379-5p","hsa-miR-27b-5p","hsa-miR-92b-3p","hsa-let-7e-5p","hsa-miR-452-5p","hsa-miR-660-5p","hsa-miR-99b-3p","hsa-miR-425-5p","hsa-miR-148b-3p","hsa-miR-199b-3p","hsa-miR-199a-3p","hsa-miR-195-5p","hsa-miR-107","hsa-miR-542-3p","hsa-miR-653-5p","hsa-miR-181a-2-3p","hsa-miR-450b-5p","hsa-miR-145-3p","hsa-miR-2355-5p","hsa-miR-127-5p","hsa-miR-500a-3p","hsa-miR-22-5p","hsa-miR-25-3p","hsa-miR-150-5p","hsa-miR-501-3p","hsa-miR-127-3p","hsa-miR-134-5p"],[-9.144,3.602,-3.587,3.43,-3.534,6.245,-5.648,-3.895,-1.904,-2.447,1.638,2.17,1.671,-1.591,1.284,2.784,-2.115,-1.433,1.283,-2.143,1.724,1.274,1.433,1.507,-1.968,1.755,1.206,1.74,1.342,1.145,1.935,1.335,1.237,-1.045,1.416,1.824,1.254,1.258,1.134,1.303,1.115,-1.64,1.165,1.86,0.95,0.991,2.645,0.746,2.041,1.237,-1.063,-1.07,1.135,-2.188,-1.146,1.802,1.28,-0.71,-0.878,-2.626,0.987,-1.042,-0.819,2.418,-1.665,0.771,-1.02,1.141,-0.869,-0.831,-1.869,0.661,-1.193,0.724,-0.832,0.877,-0.859,0.635,-0.8139999999999999,-0.9330000000000001,-0.74,0.552,0.756,-1.198,-0.9,-1.189,0.785,0.5600000000000001,0.8179999999999999,-0.8090000000000001,-0.866,1.281,-1.108,-0.958,0.804,0.51,0.632,-1.091,-0.65,-0.5629999999999999,0.85,1.119,0.536,0.593,0.571,0.629,-1.03,-0.912,0.678,-0.5,0.41,0.609,0.459,0.448,-1.175,-0.528,-0.537,0.523,0.464,-0.522,0.297,-0.447,0.897,0.738,0.317,0.5620000000000001,0.555,-0.662,-0.324,-0.973,0.608,-0.316,0.368,0.491,0.491,-0.341,-0.625,0.523,-0.859,-0.355,1.133,-0.278,-0.46,-0.209,-0.285,-0.238,0.403,0.404,0.5679999999999999,-0.34,-0.231,0.237,0.242,0.402,-0.347,-0.186,0.311,0.159,0.168,0.179,-0.154,0.289,-0.34,0.34,0.158,-0.171,0.154,0.247,-0.168,0.113,0.168,0.094,-0.23,-0.225,-0.151,0.089,0.17,-0.244,-0.098,0.135,-0.117,-0.07000000000000001,0.128,0.076,-0.057,-0.034,-0.07199999999999999,-0.038,-0.02,-0.018],[5.931,17.506,5.435,5.351,6.76,7.457,7.962,7.593,15.031,7.628,9.413,7.724,6.138,4.349,6.617,8.430999999999999,8.103,12.34,5.763,8.129,6.02,5.639,3.854,4.028,15.047,4.597,5.979,4.85,4.058,13.548,6.352,10.295,5.712,12.73,8.026,10.183,4.422,7.415,9.039,7.169,9.550000000000001,11.847,5.087,6.618,11.285,10.284,10.379,6.409,8.538,6.484,8.999000000000001,9.619,3.973,10.37,5.018,10.352,6.607,10.596,13.143,5.704,7.696,5.731,16.007,11.818,7.29,7.3,4.696,4.543,5.788,13.585,12.161,8.48,7.115,6.954,10.394,5.827,13.256,7.736,6.826,16.513,16.997,6.181,6.407,5.37,14.106,15.043,9.314,4.779,8.542,7.598,5.255,7.983,5.682,5.258,7.524,7.122,5.838,11.378,4.531,4.035,10.62,5.777,11.487,4.96,5.838,10.658,6.487,5.359,7.534,8.045999999999999,10.054,7.465,7.999,9.327999999999999,7.179,8.802,5.306,5.149,6.522,8.994999999999999,7.428,7.273,7.06,10.546,11.611,3.975,8.212,9.83,13.347,7.396,5.417,7.699,4.144,5.408,5.408,14.606,4.654,9.132999999999999,9.077,5.69,8.113,10.179,4.116,10.346,8.616,12.349,8.311999999999999,12.617,9.593,8.669,14.373,4.434,5.375,5.076,5.504,9.946,7.734,9.007999999999999,4.481,3.87,5.669,4.289,4.933,6.199,3.74,5.602,9.930999999999999,4.975,5.335,5.353,6.242,7.288,8.423,8.429,4.708,5.72,7.893,5.045,9.414,4.143,4.507,5.478,5.011,7.887,4.249,12.218,7.958,5.582,6.024,5.431],[-35.813,30.988,-26.637,26.517,-25.641,22.65,-21.75,-18.753,-17.747,-17.695,17.693,17.485,17.22,-17.035,16.875,16.285,-16.278,-16.229,16.18,-15.678,15.618,15.535,15.139,15.016,-14.267,13.956,13.872,13.856,13.698,13.533,13.148,13.001,12.959,-12.954,12.894,12.823,12.796,12.603,12.495,12.415,12.1,-11.53,11.473,11.37,11.275,10.795,10.78,10.565,10.383,10.363,-10.287,-10.195,9.984,-9.92,-9.893000000000001,9.882999999999999,9.869999999999999,-9.843999999999999,-9.641,-9.484999999999999,9.477,-9.464,-9.279999999999999,9.18,-9.125999999999999,8.949999999999999,-8.946999999999999,8.904999999999999,-8.839,-8.752000000000001,-8.742000000000001,8.318,-8.259,8.192,-7.938,7.777,-7.719,7.601,-7.514,-7.362,-7.323,7.256,7.191,-7.185,-7.066,-7.052,7.042,6.846,6.828,-6.806,-6.757,6.75,-6.645,-6.499,6.335,6.331,6.29,-6.106,-6.098,-6.079,6.064,5.95,5.876,5.828,5.81,5.807,-5.79,-5.731,5.568,-5.324,5.258,5.183,5.119,5.095,-5.089,-5.088,-4.888,4.78,4.564,-4.564,4.558,-4.383,4.351,4.211,4.107,4.097,3.966,-3.958,-3.917,-3.9,3.883,-3.883,3.868,3.716,3.713,-3.707,-3.589,3.475,-3.296,-3.24,3.228,-3.058,-2.95,-2.922,-2.824,-2.803,2.778,2.67,2.616,-2.578,-2.527,2.459,2.324,2.303,-2.276,-2.028,2.003,1.847,1.823,1.809,-1.795,1.786,-1.768,1.739,1.67,-1.512,1.502,1.466,-1.373,1.33,1.269,1.189,-1.151,-1.131,-1.112,1.069,1.038,-1.019,-0.975,0.8120000000000001,-0.8090000000000001,-0.671,0.667,0.649,-0.467,-0.456,-0.391,-0.257,-0.104,-0.08699999999999999],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.001,0.002,0.003,0.004,0.005,0.005,0.006,0.008,0.008999999999999999,0.01,0.012,0.014,0.02,0.022,0.023,0.043,0.046,0.065,0.06900000000000001,0.07099999999999999,0.073,0.074,0.077,0.082,0.095,0.131,0.133,0.143,0.17,0.184,0.205,0.235,0.25,0.259,0.267,0.285,0.3,0.309,0.33,0.417,0.419,0.502,0.505,0.516,0.64,0.648,0.696,0.798,0.917,0.931],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.002,0.002,0.003,0.004,0.005,0.006,0.007,0.007,0.01,0.012,0.013,0.015,0.018,0.025,0.027,0.028,0.052,0.055,0.078,0.082,0.08400000000000001,0.08599999999999999,0.08699999999999999,0.09,0.095,0.11,0.15,0.152,0.162,0.191,0.205,0.228,0.259,0.275,0.282,0.289,0.308,0.322,0.329,0.35,0.44,0.44,0.524,0.524,0.533,0.658,0.662,0.707,0.806,0.922,0.931],[381.8,312.496,249.393,247.659,235.017,192.341,179.739,138.941,125.779,125.109,125.08,122.403,119.007,116.663,114.639,107.258,107.175,106.57,105.966,99.80800000000001,99.08799999999999,98.08199999999999,93.328,91.867,83.13,79.57899999999999,78.62,78.44499999999999,76.67100000000001,74.825,70.583,68.989,68.52800000000001,68.47199999999999,67.831,67.063,66.77500000000001,64.714,63.578,62.736,59.451,53.674,53.112,52.095,51.156,46.529,46.385,44.365,42.675,42.494,41.797,40.954,39.056,38.485,38.252,38.155,38.042,37.813,36.033,34.694,34.625,34.509,32.951,32.112,31.657,30.213,30.186,29.845,29.306,28.607,28.528,25.195,24.74,24.232,22.331,21.151,20.733,19.888,19.269,18.212,17.942,17.479,17.037,16.999,16.199,16.104,16.038,14.749,14.635,14.489,14.176,14.133,13.463,12.55,11.549,11.525,11.276,10.185,10.136,10.027,9.94,9.281000000000001,8.864000000000001,8.590999999999999,8.494,8.477,8.380000000000001,8.052,7.166,5.883,5.544,5.163,4.844,4.724,4.697,4.692,3.723,3.215,2.231,2.231,2.206,1.438,1.301,0.714,0.293,0.251,-0.267,-0.296,-0.457,-0.522,-0.588,-0.588,-0.644,-1.21,-1.221,-1.245,-1.669,-2.063,-2.661,-2.843,-2.881,-3.409,-3.731,-3.811,-4.09,-4.149,-4.218,-4.509,-4.652,-4.75,-4.879,-5.047,-5.368,-5.416,-5.478,-6.009,-6.059,-6.359,-6.402,-6.428,-6.452,-6.468,-6.5,-6.55,-6.669,-6.919,-6.934,-6.988,-7.119,-7.177,-7.256,-7.354,-7.398,-7.421,-7.443,-7.489,-7.522,-7.541,-7.585,-7.731,-7.733,-7.835,-7.838,-7.85,-7.951,-7.956,-7.984,-8.026999999999999,-8.055,-8.057]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>microRNA<\/th>\n      <th>logFC<\/th>\n      <th>AveExpr<\/th>\n      <th>t<\/th>\n      <th>P.Value<\/th>\n      <th>adj.P.Val<\/th>\n      <th>B<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"microRNA","targets":0},{"name":"logFC","targets":1},{"name":"AveExpr","targets":2},{"name":"t","targets":3},{"name":"P.Value","targets":4},{"name":"adj.P.Val","targets":5},{"name":"B","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-1d9474bf94b344caac86" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-1d9474bf94b344caac86">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 4: Differentially expressed micro-RNAs in KIRC vs KIRP comparison.<\/caption>","data":[["hsa-miR-126-3p","hsa-miR-126-5p","hsa-miR-143-5p","hsa-miR-145-5p","hsa-miR-145-3p","hsa-miR-16-5p","hsa-miR-424-5p","hsa-miR-10b-5p","hsa-miR-195-5p","hsa-miR-200b-3p","hsa-miR-338-3p","hsa-miR-26b-3p","hsa-let-7b-3p","hsa-miR-143-3p","hsa-miR-210-3p","hsa-let-7i-3p","hsa-miR-2355-5p","hsa-miR-140-5p","hsa-let-7b-5p","hsa-miR-139-5p","hsa-miR-10b-3p","hsa-miR-17-3p","hsa-miR-29c-5p","hsa-miR-429","hsa-miR-452-5p","hsa-miR-186-5p","hsa-miR-183-5p","hsa-miR-199a-3p","hsa-miR-199b-3p","hsa-miR-629-5p","hsa-miR-155-5p","hsa-miR-484","hsa-miR-200a-3p","hsa-miR-181c-3p","hsa-miR-1307-3p","hsa-miR-29b-2-5p","hsa-miR-99b-5p","hsa-let-7d-5p","hsa-miR-200a-5p","hsa-miR-653-5p","hsa-miR-142-3p","hsa-miR-26b-5p","hsa-miR-21-5p","hsa-miR-134-5p","hsa-miR-30c-2-3p","hsa-miR-132-3p","hsa-miR-128-3p","hsa-miR-342-3p","hsa-miR-582-3p","hsa-miR-574-3p","hsa-miR-589-5p","hsa-miR-182-5p","hsa-miR-148a-3p","hsa-miR-324-3p","hsa-miR-450b-5p","hsa-miR-660-5p","hsa-miR-744-5p","hsa-miR-361-5p","hsa-miR-127-5p","hsa-miR-3607-3p","hsa-miR-181c-5p","hsa-miR-542-3p","hsa-miR-20a-5p","hsa-miR-365a-3p","hsa-miR-365b-3p","hsa-miR-30d-5p","hsa-let-7a-3p","hsa-miR-455-3p","hsa-miR-320a","hsa-miR-192-5p","hsa-miR-27b-3p","hsa-miR-24-2-5p","hsa-miR-148b-3p","hsa-miR-1287-5p","hsa-miR-423-3p","hsa-miR-15a-5p","hsa-miR-103a-3p","hsa-miR-625-3p","hsa-miR-199a-5p","hsa-miR-99b-3p","hsa-let-7i-5p","hsa-miR-203a","hsa-miR-194-5p","hsa-miR-10a-5p","hsa-miR-28-3p","hsa-miR-505-3p","hsa-miR-93-5p","hsa-miR-708-3p","hsa-miR-92a-3p","hsa-miR-584-5p","hsa-miR-185-5p","hsa-miR-532-5p","hsa-miR-142-5p","hsa-miR-30e-5p","hsa-miR-100-5p","hsa-miR-144-5p","hsa-miR-125a-5p","hsa-miR-27a-3p","hsa-miR-196a-5p","hsa-miR-374a-3p","hsa-let-7d-3p","hsa-miR-221-3p","hsa-miR-222-3p","hsa-miR-197-3p","hsa-miR-331-3p","hsa-miR-107","hsa-miR-204-5p","hsa-miR-30e-3p","hsa-miR-379-5p","hsa-miR-150-5p","hsa-miR-151a-5p","hsa-miR-375","hsa-miR-193a-5p","hsa-miR-29b-3p","hsa-miR-130a-3p","hsa-miR-27b-5p","hsa-miR-218-5p","hsa-miR-34a-5p","hsa-miR-23b-3p","hsa-miR-127-3p","hsa-miR-26a-5p","hsa-miR-146a-5p","hsa-miR-98-5p","hsa-miR-500a-3p","hsa-let-7g-3p","hsa-miR-181a-2-3p","hsa-miR-29a-3p","hsa-miR-151a-3p","hsa-miR-30a-5p","hsa-miR-125b-2-3p","hsa-miR-451a","hsa-miR-101-3p","hsa-miR-30c-5p","hsa-miR-29c-3p","hsa-miR-19a-3p","hsa-miR-21-3p","hsa-let-7a-5p","hsa-miR-15b-5p","hsa-miR-196b-5p","hsa-miR-378a-5p","hsa-miR-501-3p","hsa-miR-106b-3p","hsa-miR-32-5p","hsa-miR-664a-3p","hsa-miR-181a-3p","hsa-miR-191-5p","hsa-miR-106b-5p","hsa-miR-200c-3p","hsa-miR-99a-5p","hsa-let-7f-5p","hsa-miR-324-5p","hsa-miR-598-3p","hsa-let-7g-5p","hsa-miR-28-5p","hsa-miR-181b-5p","hsa-miR-92b-3p","hsa-miR-146b-3p","hsa-miR-152-3p","hsa-miR-339-5p","hsa-let-7c-5p","hsa-miR-361-3p","hsa-miR-20b-5p","hsa-miR-23a-3p","hsa-miR-425-5p","hsa-miR-532-3p","hsa-let-7e-5p","hsa-miR-24-3p","hsa-miR-223-3p","hsa-miR-125b-5p","hsa-miR-17-5p","hsa-miR-30a-3p","hsa-miR-374b-5p","hsa-miR-9-5p","hsa-miR-146b-5p","hsa-miR-140-3p","hsa-miR-328-3p","hsa-miR-423-5p","hsa-miR-486-5p","hsa-miR-652-3p","hsa-miR-19b-3p","hsa-miR-455-5p","hsa-miR-22-5p","hsa-miR-181a-5p","hsa-miR-374a-5p","hsa-miR-22-3p","hsa-miR-30b-5p","hsa-miR-25-3p","hsa-miR-1307-5p","hsa-miR-193b-3p","hsa-miR-378a-3p"],[3.798,3.088,3.394,2.844,2.344,1.618,2.835,1.894,2.006,-1.89,1.75,1.325,1.276,2.241,2.331,1.382,1.301,1.059,1.355,1.971,1.284,-0.894,1.189,-1.591,1.758,0.859,-2.227,1.974,1.976,0.967,1.846,0.951,-1.385,-1.193,-1.057,1.032,0.868,0.762,-1.288,2.225,1.618,0.8179999999999999,-1.035,1.784,-1.041,-0.6830000000000001,-0.9,1.02,1.76,0.878,-0.713,-1.759,1.043,-0.797,1.336,-0.981,-0.783,-0.519,1.524,1.408,-0.925,1.179,-0.713,0.9429999999999999,0.9429999999999999,-0.584,0.663,1.137,0.797,1.69,-0.573,0.596,0.467,-0.519,-0.416,0.481,-0.458,-0.6870000000000001,1.091,0.449,0.41,1.286,1.2,-0.657,0.366,0.48,-0.42,0.79,-0.38,0.711,-0.481,-0.457,0.681,0.392,0.76,1.023,0.355,-0.368,0.5580000000000001,0.365,0.367,-0.535,0.523,0.404,0.408,-0.317,-1.045,0.303,0.735,0.672,0.274,-0.993,0.405,0.391,0.366,0.311,-0.571,-0.403,0.292,0.602,0.227,-0.458,0.255,-0.359,0.346,0.292,-0.254,0.202,-0.278,-0.363,0.677,0.247,0.277,0.266,0.284,-0.355,0.226,0.258,0.494,0.334,0.311,0.213,0.183,-0.196,0.195,-0.185,-0.148,-0.391,-0.228,-0.228,-0.147,0.228,-0.125,-0.101,0.156,-0.16,-0.235,0.151,-0.156,0.171,-0.12,-0.184,0.095,0.145,-0.14,0.111,-0.074,-0.155,-0.124,-0.079,0.092,0.08500000000000001,0.262,0.138,-0.062,0.076,0.058,0.136,0.047,0.046,-0.051,0.04,-0.033,0.026,0.023,0.025,0.018,-0.033,-0.012,0.008999999999999999],[11.847,8.103,5.37,9.83,4.507,8.802,7.29,16.513,4.708,8.129,7.524,4.434,4.349,15.043,10.352,5.731,5.478,5.979,13.256,5.682,4.696,7.122,5.306,5.258,4.975,7.736,10.37,8.429,8.423,5.788,7.983,5.838,7.115,4.597,9.314,5.149,14.606,7.699,7.628,5.045,10.546,9.039,17.506,5.431,7.598,5.763,6.407,6.484,7.593,5.375,5.669,12.161,14.106,3.87,4.143,5.335,4.422,7.428,5.011,6.487,4.85,7.893,7.415,5.408,5.408,13.347,4.144,8.430999999999999,8.542,11.818,9.946,4.028,7.288,3.854,6.617,6.954,13.548,6.607,7.06,5.353,8.48,7.962,10.379,15.047,11.611,5.087,11.487,5.076,12.349,5.359,5.69,9.619,5.417,13.143,11.378,7.396,9.327999999999999,9.550000000000001,8.212,9.413,8.045999999999999,6.76,5.435,7.696,4.531,5.72,7.457,12.73,6.199,7.958,6.181,5.704,7.465,8.994999999999999,6.02,3.74,4.654,7.724,10.179,6.024,10.346,6.352,5.639,7.887,5.018,9.414,12.34,10.596,16.997,3.975,9.077,13.585,10.658,10.394,3.973,10.183,14.373,6.826,9.593,5.504,5.582,7.273,4.481,4.96,7.169,8.616,7.3,5.931,9.132999999999999,12.617,4.058,4.116,9.007999999999999,6.409,8.026,5.602,5.777,7.534,4.543,10.62,6.522,4.289,11.285,6.242,5.255,9.930999999999999,10.054,6.618,8.669,7.999,15.031,5.838,8.113,8.538,10.284,5.827,5.712,7.179,4.035,6.138,5.351,4.249,10.295,4.779,16.007,8.999000000000001,12.218,8.311999999999999,4.933,7.734],[48.8,43.45,37.215,31.106,29.514,28.478,28.4,27.31,26.902,-25.269,25.196,25.133,24.961,24.298,23.358,22.947,22.815,22.278,22.254,21.613,20.584,-20.288,19.799,-19.733,19.095,18.782,-18.46,18.099,18.085,17.963,17.77,17.668,-17.516,-17.34,-17.318,17.243,17.241,17.142,-17.026,16.996,16.875,16.471,-16.269,16.043,-16.005,-15.752,-15.651,15.622,15.49,15.399,-15.187,-15.032,14.965,-14.685,14.678,-14.667,-14.611,-14.581,14.57,14.466,-13.46,13.153,-13.051,13.049,13.042,-12.911,12.741,12.158,12.155,11.725,-11.442,10.854,10.794,-10.021,-10.002,9.951000000000001,-9.893000000000001,-9.683999999999999,9.670999999999999,9.66,9.430999999999999,9.051,8.94,-8.699999999999999,8.673999999999999,8.651,-8.417999999999999,8.285,-8.182,8.164999999999999,-8.028,-7.962,7.951,7.862,7.776,7.497,7.372,-7.302,7.288,7.214,7.141,-7.097,7.093,7.091,6.993,-6.988,-6.924,6.877,6.873,6.689,6.578,-6.555,6.303,6.259,6.062,6.007,-6.002,-5.935,5.873,5.824,5.805,-5.684,5.68,-5.594,5.454,5.328,-5.266,5.112,-5.029,-4.837,4.75,4.75,4.676,4.636,4.562,-4.555,4.508,4.346,4.16,4.002,3.844,3.817,3.636,-3.513,3.393,-3.35,-3.149,-2.799,-2.771,-2.757,-2.741,2.67,-2.657,-2.613,2.597,-2.596,-2.282,2.267,-2.225,2.225,-2.158,-2.075,2.056,2.001,-2.001,1.977,-1.746,-1.727,-1.718,-1.612,1.564,1.549,1.365,1.28,-1.243,1.232,1.104,1.08,0.921,0.858,-0.724,0.601,-0.58,0.579,0.485,0.436,0.428,-0.412,-0.116,0.108],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.002,0.005,0.006,0.006,0.006,0.008,0.008,0.008999999999999999,0.01,0.01,0.023,0.024,0.026,0.026,0.031,0.038,0.04,0.046,0.046,0.048,0.081,0.08400000000000001,0.08599999999999999,0.107,0.118,0.122,0.173,0.201,0.214,0.218,0.27,0.28,0.357,0.391,0.469,0.548,0.5620000000000001,0.5629999999999999,0.628,0.663,0.669,0.681,0.908,0.914],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.002,0.007,0.007,0.008,0.008,0.01,0.01,0.011,0.012,0.012,0.027,0.028,0.031,0.031,0.037,0.045,0.047,0.053,0.053,0.055,0.092,0.096,0.097,0.12,0.131,0.135,0.19,0.219,0.233,0.235,0.29,0.299,0.379,0.413,0.492,0.572,0.581,0.581,0.645,0.677,0.68,0.6879999999999999,0.913,0.914],[557.525,487.468,401.62,314.122,291.001,275.952,274.819,259.001,253.09,229.493,228.446,227.536,225.072,215.559,202.173,196.349,194.474,186.925,186.576,177.62,163.4,159.361,152.708,151.819,143.255,139.086,134.838,130.11,129.922,128.341,125.841,124.511,122.554,120.306,120.027,119.058,119.035,117.782,116.307,115.928,114.398,109.322,106.808,104.02,103.549,100.463,99.23099999999999,98.878,97.282,96.181,93.648,91.807,91.011,87.717,87.63,87.501,86.843,86.498,86.36799999999999,85.15600000000001,73.741,70.37,69.261,69.229,69.16200000000001,67.73699999999999,65.917,59.781,59.75,55.348,52.52,46.803,46.234,39.097,38.927,38.478,37.958,36.124,36.006,35.916,33.936,30.748,29.839,27.899,27.688,27.507,25.672,24.647,23.86,23.732,22.702,22.212,22.133,21.477,20.848,18.852,17.982,17.501,17.4,16.894,16.4,16.107,16.083,16.067,15.411,15.385,14.962,14.654,14.629,13.443,12.744,12.597,11.051,10.792,9.627000000000001,9.307,9.276999999999999,8.897,8.544,8.266999999999999,8.16,7.491,7.47,7.004,6.256,5.601,5.281,4.507,4.098,3.178,2.772,2.769,2.432,2.251,1.915,1.887,1.678,0.975,0.199,-0.43,-1.04,-1.143,-1.805,-2.239,-2.649,-2.792,-3.436,-4.469,-4.547,-4.583,-4.627,-4.817,-4.852,-4.967,-5.01,-5.012,-5.771,-5.806,-5.899,-5.901,-6.045,-6.22,-6.261,-6.372,-6.372,-6.418,-6.848,-6.879,-6.896,-7.072,-7.148,-7.171,-7.439,-7.551,-7.597,-7.61,-7.76,-7.786,-7.945,-8.000999999999999,-8.106999999999999,-8.189,-8.201000000000001,-8.202,-8.252000000000001,-8.275,-8.278,-8.285,-8.363,-8.364000000000001]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>microRNA<\/th>\n      <th>logFC<\/th>\n      <th>AveExpr<\/th>\n      <th>t<\/th>\n      <th>P.Value<\/th>\n      <th>adj.P.Val<\/th>\n      <th>B<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"microRNA","targets":0},{"name":"logFC","targets":1},{"name":"AveExpr","targets":2},{"name":"t","targets":3},{"name":"P.Value","targets":4},{"name":"adj.P.Val","targets":5},{"name":"B","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
</div>
</section>
</section>
<section id="sec-lesson-learnt-micro-RNAs" class="level2" data-number="4.5">
<h2 data-number="4.5" class="anchored" data-anchor-id="sec-lesson-learnt-micro-RNAs"><span class="header-section-number">4.5</span> Lessons Learnt</h2>
<p>Based on micro-RNAs data, we have learnt:</p>
<ul>
<li>Exploratory analysis suggests:
<ul>
<li>PC2 (12.83% variance) separates the three kidney carcinomas types.</li>
</ul></li>
<li>Differential Gene Expression analysis:
<ul>
<li>here KIRC and KIRP seems more similar to each other (30 DGE microRNAs in common vs KICH).</li>
<li>KIRC have 20 dysregulated miRNAs when compared to KIRP and KICH (including <em>hsa-miR-155-5p</em>)</li>
<li><em>hsa-miR-155-5p</em> (among the others) may contribute to regulate the metabolic reprogramming in KIRC by promoting glycolysis via <em>PKM2</em>, <em>LDHA</em>/<em>LDHB</em>, and <em>GAPDH.</em></li>
</ul></li>
</ul>
</section>
<section id="session-information" class="level2" data-number="4.6">
<h2 data-number="4.6" class="anchored" data-anchor-id="session-information"><span class="header-section-number">4.6</span> Session Information</h2>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: openSUSE Tumbleweed

Matrix products: default
BLAS/LAPACK: /home/andrea/miniforge3/envs/moai/lib/libmkl_rt.so.2;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=it_IT.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Brussels
tzcode source: system (glibc)

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] UpSetR_1.4.0           umap_0.2.10.0          stringr_1.5.1         
 [4] scales_1.3.0           RColorBrewer_1.1-3     PCAtools_2.14.0       
 [7] matrixStats_1.5.0      org.Hs.eg.db_3.18.0    gridExtra_2.3         
[10] forcats_1.0.0          EnhancedVolcano_1.20.0 ggrepel_0.9.6         
[13] ggplot2_3.5.1          edgeR_4.0.16           limma_3.58.1          
[16] DT_0.33                dplyr_1.1.4            DOSE_3.28.2           
[19] data.table_1.16.4      cowplot_1.1.3          clusterProfiler_4.10.1
[22] BiocSingular_1.18.0    BiocParallel_1.36.0    AnnotationDbi_1.64.1  
[25] IRanges_2.36.0         S4Vectors_0.40.2       Biobase_2.62.0        
[28] BiocGenerics_0.48.1   

loaded via a namespace (and not attached):
  [1] jsonlite_1.8.9            magrittr_2.0.3           
  [3] farver_2.1.2              rmarkdown_2.29           
  [5] fs_1.6.5                  zlibbioc_1.48.2          
  [7] vctrs_0.6.5               DelayedMatrixStats_1.24.0
  [9] memoise_2.0.1             RCurl_1.98-1.16          
 [11] askpass_1.2.1             ggtree_3.10.1            
 [13] htmltools_0.5.8.1         S4Arrays_1.2.1           
 [15] SparseArray_1.2.4         gridGraphics_0.5-1       
 [17] sass_0.4.9                bslib_0.8.0              
 [19] htmlwidgets_1.6.4         plyr_1.8.9               
 [21] cachem_1.1.0              igraph_2.1.4             
 [23] lifecycle_1.0.4           pkgconfig_2.0.3          
 [25] rsvd_1.0.5                Matrix_1.6-5             
 [27] R6_2.5.1                  fastmap_1.2.0            
 [29] gson_0.1.0                GenomeInfoDbData_1.2.11  
 [31] MatrixGenerics_1.14.0     digest_0.6.37            
 [33] aplot_0.2.4               enrichplot_1.22.0        
 [35] colorspace_2.1-1          patchwork_1.3.0          
 [37] RSpectra_0.16-2           dqrng_0.4.1              
 [39] irlba_2.3.5.1             crosstalk_1.2.1          
 [41] RSQLite_2.3.9             beachmat_2.18.1          
 [43] labeling_0.4.3            httr_1.4.7               
 [45] polyclip_1.10-7           abind_1.4-8              
 [47] compiler_4.3.2            bit64_4.6.0-1            
 [49] withr_3.0.2               viridis_0.6.5            
 [51] DBI_1.2.3                 ggforce_0.4.2            
 [53] MASS_7.3-60.0.1           openssl_2.3.1            
 [55] DelayedArray_0.28.0       HDO.db_0.99.1            
 [57] tools_4.3.2               ape_5.8-1                
 [59] scatterpie_0.2.4          glue_1.8.0               
 [61] nlme_3.1-167              GOSemSim_2.28.1          
 [63] shadowtext_0.1.4          reshape2_1.4.4           
 [65] fgsea_1.28.0              generics_0.1.3           
 [67] gtable_0.3.6              tidyr_1.3.1              
 [69] tidygraph_1.3.1           ScaledMatrix_1.10.0      
 [71] XVector_0.42.0            pillar_1.10.1            
 [73] yulab.utils_0.2.0         splines_4.3.2            
 [75] tweenr_2.0.3              treeio_1.26.0            
 [77] lattice_0.22-6            bit_4.5.0.1              
 [79] tidyselect_1.2.1          GO.db_3.18.0             
 [81] locfit_1.5-9.10           Biostrings_2.70.3        
 [83] knitr_1.49                xfun_0.50                
 [85] graphlayouts_1.2.2        statmod_1.5.0            
 [87] stringi_1.8.4             lazyeval_0.2.2           
 [89] ggfun_0.1.8               yaml_2.3.10              
 [91] evaluate_1.0.3            codetools_0.2-20         
 [93] ggraph_2.2.1              tibble_3.2.1             
 [95] qvalue_2.34.0             ggplotify_0.1.2          
 [97] cli_3.6.3                 reticulate_1.40.0        
 [99] jquerylib_0.1.4           munsell_0.5.1            
[101] Rcpp_1.0.14               GenomeInfoDb_1.38.8      
[103] png_0.1-8                 parallel_4.3.2           
[105] blob_1.2.4                sparseMatrixStats_1.14.0 
[107] bitops_1.0-9              viridisLite_0.4.2        
[109] tidytree_0.4.6            purrr_1.0.2              
[111] crayon_1.5.3              rlang_1.1.5              
[113] fastmatch_1.1-6           KEGGREST_1.42.0          </code></pre>
</div>
</div>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./proteomics-analysis.html" class="pagination-link" aria-label="Proteomics Analysis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./genomics-and-epigenomics-analysis.html" class="pagination-link" aria-label="Genomics and Epigenomics Analysis">
        <span class="nav-page-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>