<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>3&nbsp; Proteomics Analysis – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./micro-RNAs-analysis.html" rel="next">
<link href="./transcriptomics-analysis.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-fixedcolumns-1.13.6/css/fixedColumns.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedcolumns-1.13.6/js/dataTables.fixedColumns.min.js"></script>
<link href="site_libs/dt-ext-fixedheader-1.13.6/css/fixedHeader.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedheader-1.13.6/js/dataTables.fixedHeader.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<script src="site_libs/dt-plugin-ellipsis-1.13.6/source.min.js"></script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./proteomics-analysis.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./survival-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./kidney-carcinomas-lessons-learnt.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">kidney-carcinomas-lessons-learnt.html</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell Cancer molecular subtypes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cancer-signatures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">cancer-signatures.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./autoencoder.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">autoencoder.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cellular-deconvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Cellular Deconvolution from Bulk RNA-seq</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">3.1</span> On this page</a></li>
  <li><a href="#proteomics-data-overview-and-qc" id="toc-proteomics-data-overview-and-qc" class="nav-link" data-scroll-target="#proteomics-data-overview-and-qc"><span class="header-section-number">3.2</span> Proteomics data overview and QC</a>
  <ul>
  <li><a href="#spectral-counts-overview" id="toc-spectral-counts-overview" class="nav-link" data-scroll-target="#spectral-counts-overview"><span class="header-section-number">3.2.1</span> Spectral counts overview</a></li>
  <li><a href="#data-imputation" id="toc-data-imputation" class="nav-link" data-scroll-target="#data-imputation"><span class="header-section-number">3.2.2</span> Data imputation</a></li>
  </ul></li>
  <li><a href="#dimesionality-reduction-and-dataset-exploration" id="toc-dimesionality-reduction-and-dataset-exploration" class="nav-link" data-scroll-target="#dimesionality-reduction-and-dataset-exploration"><span class="header-section-number">3.3</span> Dimesionality Reduction and Dataset Exploration</a>
  <ul>
  <li><a href="#sec-UMAP-proteomics" id="toc-sec-UMAP-proteomics" class="nav-link" data-scroll-target="#sec-UMAP-proteomics"><span class="header-section-number">3.3.1</span> UMAP on proteomics data</a></li>
  <li><a href="#principal-component-analysis-pca" id="toc-principal-component-analysis-pca" class="nav-link" data-scroll-target="#principal-component-analysis-pca"><span class="header-section-number">3.3.2</span> Principal Component Analysis (PCA)</a></li>
  </ul></li>
  <li><a href="#differential-protein-abundance-analysis" id="toc-differential-protein-abundance-analysis" class="nav-link" data-scroll-target="#differential-protein-abundance-analysis"><span class="header-section-number">3.4</span> Differential protein abundance analysis</a>
  <ul>
  <li><a href="#identification-of-differentially-abundant-proteins" id="toc-identification-of-differentially-abundant-proteins" class="nav-link" data-scroll-target="#identification-of-differentially-abundant-proteins"><span class="header-section-number">3.4.1</span> Identification of differentially abundant proteins</a></li>
  <li><a href="#comparative-enrichment-analyses-across-cancer-types" id="toc-comparative-enrichment-analyses-across-cancer-types" class="nav-link" data-scroll-target="#comparative-enrichment-analyses-across-cancer-types"><span class="header-section-number">3.4.2</span> Comparative enrichment analyses across cancer types</a>
  <ul>
  <li><a href="#go-biological-process-terms" id="toc-go-biological-process-terms" class="nav-link" data-scroll-target="#go-biological-process-terms"><span class="header-section-number">3.4.2.1</span> GO Biological Process terms</a></li>
  <li><a href="#go-cellular-compartments-terms" id="toc-go-cellular-compartments-terms" class="nav-link" data-scroll-target="#go-cellular-compartments-terms"><span class="header-section-number">3.4.2.2</span> GO Cellular Compartments terms</a></li>
  <li><a href="#go-molecular-function-terms" id="toc-go-molecular-function-terms" class="nav-link" data-scroll-target="#go-molecular-function-terms"><span class="header-section-number">3.4.2.3</span> GO Molecular Function terms</a></li>
  <li><a href="#kegg-pathways" id="toc-kegg-pathways" class="nav-link" data-scroll-target="#kegg-pathways"><span class="header-section-number">3.4.2.4</span> KEGG pathways</a></li>
  <li><a href="#reactome-pathways" id="toc-reactome-pathways" class="nav-link" data-scroll-target="#reactome-pathways"><span class="header-section-number">3.4.2.5</span> Reactome pathways</a></li>
  <li><a href="#wikipathways" id="toc-wikipathways" class="nav-link" data-scroll-target="#wikipathways"><span class="header-section-number">3.4.2.6</span> WikiPathways</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-proteomics" id="toc-sec-lesson-learnt-proteomics" class="nav-link" data-scroll-target="#sec-lesson-learnt-proteomics"><span class="header-section-number">3.5</span> Lessons Learnts</a></li>
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information"><span class="header-section-number">3.6</span> Session Information</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./proteomics-analysis.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-proteomics-analysis" class="quarto-section-identifier"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">3.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at Lesson Learnt: <a href="#sec-lesson-learnt-proteomics" class="quarto-xref"><span>Section 3.5</span></a>.</p>
<ul>
<li>Here we investigate the proteomics data across the three Kidney Cancers.</li>
<li>First, we perform a QC on protein spectra abundance, their distribution across samples and we impute eventual missing values.</li>
<li>Then, we do some exploratory analyses on samples, protein bundances and clinical covariates.</li>
<li>We then run a formal Differential Protein Abundance analysis to identify proteins that have different expression levels across the three Kidney cancer types.</li>
<li>We perform Gene Set Enrichment Analyses on the differentially abundant proteins to investigate biological and molecular themes that discriminates between the three Kidney cancer types.</li>
</ul>
</section>
<section id="proteomics-data-overview-and-qc" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="proteomics-data-overview-and-qc"><span class="header-section-number">3.2</span> Proteomics data overview and QC</h2>
<section id="spectral-counts-overview" class="level3" data-number="3.2.1">
<h3 data-number="3.2.1" class="anchored" data-anchor-id="spectral-counts-overview"><span class="header-section-number">3.2.1</span> Spectral counts overview</h3>
<p>For TCGA Kidney cancer proteomics data, we will perform analyses similar to the ones performed for the transcriptomics one (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>). We first start with an overview of the datasets and QC.</p>
<p>Let’s check the spectral counts distributions across samples, and the protein completeness across samples.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 1: Proteomics spectral counts missing values.</figcaption>
</figure>
</div>
</div>
</div>
<p>Looking at the distributions, we can conclude that the spectral intensities were already normalized and scaled, since the spectral values are centered around 0 and have a normal distribution.</p>
<p>We have 258 proteins in total, across 709 samples (79.9% of samples had proteomics data).</p>
<ul>
<li>24 proteins were missing in all samples (100% of patients with proteomics data available)</li>
<li>1 protein was missing in 440 samples (62.1% of patients with proteomics data available)</li>
<li>16 proteins were missing in 269 samples (37.9% of patients with proteomics data available)</li>
<li>6 protein were missing in only 1 sample 211 proteins were detected for all patients with proteomics data available)</li>
</ul>
<p>We can obviously drop the 24 proteins for which we have not spectral counts for none of the samples. For the remaining proteins for which we do not have full occupancy, we will apply a standard imputation strategy.</p>
</section>
<section id="data-imputation" class="level3" data-number="3.2.2">
<h3 data-number="3.2.2" class="anchored" data-anchor-id="data-imputation"><span class="header-section-number">3.2.2</span> Data imputation</h3>
<p>Missing values imputation for label-free proteomics is a large and complex <a href="https://pubs.acs.org/doi/10.1021/pr501138h">field of study</a>.</p>
<p>In our Kidney cancer proteomics samples the number of proteins detected is small and the distribution of missing values is oddly discrete. This suggests that some kind of filtering was already applied during the preprocessing steps, together with scaling and normalization of the proteins spectral counts. The absence can therefore as well be ascribed to biological reasons as well, and not only to technical problems with its detection.</p>
<p>To preserve this putative biological signal, I replace in each sample the missing values with the minimum spectral count available in that sample.</p>
<p>Let’s check how the spectral distributions changed.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 2: Proteomics spectral counts after imputation.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="dimesionality-reduction-and-dataset-exploration" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="dimesionality-reduction-and-dataset-exploration"><span class="header-section-number">3.3</span> Dimesionality Reduction and Dataset Exploration</h2>
<section id="sec-UMAP-proteomics" class="level3" data-number="3.3.1">
<h3 data-number="3.3.1" class="anchored" data-anchor-id="sec-UMAP-proteomics"><span class="header-section-number">3.3.1</span> UMAP on proteomics data</h3>
<p>Let’s check the samples clustering on a UMAP. We have almost a perfect clustering of the samples in their corresponding cancer subtypes, with just 2 KICH samples clusting with KIRC samples.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 3: Kidney proteomics UMAP.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="principal-component-analysis-pca" class="level3" data-number="3.3.2">
<h3 data-number="3.3.2" class="anchored" data-anchor-id="principal-component-analysis-pca"><span class="header-section-number">3.3.2</span> Principal Component Analysis (PCA)</h3>
<p>As we did for the transcriptomics data (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>), the next step in the dataset exploration is to perform the Principal Component Analysis.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 4: PCA, exploration.</figcaption>
</figure>
</div>
</div>
</div>
<p>The first 21 Principal Components capture more than 80% of the variance in the Kidney cancers transcriptomics dataset, with the first two components (PC1 and PC2) capturing a bit more than 40% of the variance.</p>
<p>When we project the samples in the PC1 and PC2, we can see that the PC1 separates KIRC from KICH and KIRP, which instead cluster together. The second component PC2, instead, seems to separate KICH and KIRP samples. The separation between KICH and KIRP samples along component PC2 is clearer in the proteomics data than in the transcriptomics data.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 5: PCA, biplot PC1 x PC2.</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 6: PCA, pairs plots.</figcaption>
</figure>
</div>
</div>
</div>
<p>Let’s check the loadings (i.e.: 21 Principal Components capture more than 80% of the variance in the) for the top 4 Principal Components. These indicate which genes are the more responsible to explain the position of the samples along the components, and the direction of this separation.</p>
<p>Looking at the top 1% most variable proteins (~ 24 proteins), the following 7 proteins are the top loadings for the first 4 Principal Components:</p>
<ol type="1">
<li><em>ATM</em></li>
<li><em>CLAUDIN7</em></li>
<li><em>EPPK1</em></li>
<li><em>GAPDH</em></li>
<li><em>HSP70</em></li>
<li><em>MYH11</em></li>
<li><em>NDRG1_pT346</em></li>
</ol>
<p>Let’s now check the expression of the five top genes identified with the PCA across the cancer types:</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 7: PCA, loadings.</figcaption>
</figure>
</div>
</div>
</div>
<p><em>CLAUDIN-7</em> is a known diagnostic immunohistochemical marker for <a href="https://www.sciencedirect.com/science/article/pii/S0893395222023766">classifing chromophobe renal carcinoma (KICH)</a>, and indeed seems overexpressed in KICH when compared to KIRC and KIRP.</p>
<p>Let’s check the Pearson correlation with other clinical covariates.</p>
<p><em>Histological grade</em>, <em>subtype mRNA</em> and <em>subtype miRNA</em> strongly correlates with PC1. <em>Cancer type</em>, instead, correlates more with PC2. Interestingly, <em>tumor stage</em>, <em>follow up tumor status</em> and <em>vital status</em> correlates with PC5 (~4% of observed variance), which does not correlates with <em>cancer type</em> but does correlate with <em>subtype protein</em>. This suggest that there is a good correlation between the differences in protein abundances across the kidney carcinomas that correlate with a poorer prognosis, and this signal does not correlate with the kidney carcinoma type (<em>cancer type</em>), nor with the gene expression (<em>subtype mRNA</em> and <em>subtype miRNA</em>).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 8: PCA, correlations between clinical covariates and Principal Components.</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="differential-protein-abundance-analysis" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="differential-protein-abundance-analysis"><span class="header-section-number">3.4</span> Differential protein abundance analysis</h2>
<p>In addition to cancer type, we saw that age, ethnicity (and race) and age had somewhat a correlation with the cancer types.</p>
<p>We may want to include this covariates in the differential gene expression analysis in order to include their contribution into the model.</p>
<section id="identification-of-differentially-abundant-proteins" class="level3" data-number="3.4.1">
<h3 data-number="3.4.1" class="anchored" data-anchor-id="identification-of-differentially-abundant-proteins"><span class="header-section-number">3.4.1</span> Identification of differentially abundant proteins</h3>
<p>For proteomics analyses, we use two arbitrary thresholds to retain genes that are significantly differentially expressed across the each comparison: an absolute logFold-Change (logFC) higher than 1, and an adjusted p-value lower than 0.05. This results in:</p>
<ul>
<li>KIRC_vs_KICH, 197 differentially abundant proteins, 105 upregulated and 92 downregulated</li>
<li>KIRP_vs_KICH, 186 differentially abundant proteins, 97 upregulated and 89 downregulated</li>
<li>KIRC_vs_KIRP, 207 differentially abundant proteins, 102 upregulated and 105 downregulated</li>
</ul>
<p>As usual, we can visualize the logFC p-values relationships for all the proteins with a volcano plot.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 9: Volcano plots of each contrats reporting the proteins differentially expressed.</figcaption>
</figure>
</div>
</div>
</div>
<p>We can also check the logFC and adjusted p-value for the top 7 proteins that are the top loadings for the first 4 Principal Components and see how these proteins were differentially abundant across the comparisons. We can confirm what we have naively observed at the level of protein abundance in Figure 7 while investigating the PCA loadings: <em>CLAUDIN7</em> and <em>NDRG1_pT346</em> are the top loadings for PC1, which separates KIRC from KICH and KIRP, and indeed, these proteins are the most differentially abundant in the comparisons of KIRC versus KIRP or KICH.</p>
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-aedb36e44f224e82764c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-aedb36e44f224e82764c">{"x":{"filter":"none","vertical":false,"class":"display","extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 1: top 7 loadings across DGE comparisons.<\/caption>","data":[["CLAUDIN7","NDRG1_pT346","ATM","HSP70","EPPK1","GAPDH","MYH11"],[-2.08,3.27,1.08,1.01,1.65,1.68,-0.29],[7e-92,4.71e-84,1.87e-27,3.9e-19,4.24e-19,3.36e-17,0.228],[-0.9399999999999999,1.14,1.46,-0.11,1.16,0.28,-1.81],[4.35e-21,2.95e-12,1.71e-39,0.392,5.19e-09,0.202,2.45e-12],[-1.15,2.12,-0.37,1.12,0.48,1.4,1.52],[4.93e-70,1.33e-82,4.82e-09,4.85e-47,4.89e-05,4.79e-26,3.49e-22]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th rowspan=\"2\">Gene<\/th>\n      <th colspan=\"2\">KIRC_vs_KICH<\/th>\n      <th colspan=\"2\">KIRP_vs_KICH<\/th>\n      <th colspan=\"2\">KIRC_vs_KIRP<\/th>\n    <\/tr>\n    <tr>\n      <th>logFC<\/th>\n      <th>adj. p-value<\/th>\n      <th>logFC<\/th>\n      <th>adj. p-value<\/th>\n      <th>logFC<\/th>\n      <th>adj. p-value<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"protein","targets":0},{"name":"KIRC_vs_KICH.logFC","targets":1},{"name":"KIRC_vs_KICH.adj..p.value","targets":2},{"name":"KIRP_vs_KICH.logFC","targets":3},{"name":"KIRP_vs_KICH.adj..p.value","targets":4},{"name":"KIRC_vs_KIRP.logFC","targets":5},{"name":"KIRC_vs_KIRP.adj..p.value","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>The UpSet plot shows that KIRC has a strong signature of 19 proteins that are differentially abundant when compared to KIRP or KICH. Moreover, KICH overexpresses <em>c-KIT</em>, <em>ERα_pS118</em> and <em>CDH1</em> and downregulates <em>ATM</em>, <em>NDRG1_pT346</em> and <em>PARP1</em> when compared to KIRC or KIRP.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 10: UpSet reporting the proteins differentially expressed upregulated and downregulated in common across all contrasts.</figcaption>
</figure>
</div>
</div>
</div>
<p>Tables of differentially expressed proteins.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">KIRC vs KICH</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">KIRP vs KICH</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">KIRC vs KIRP</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-999c47e1d210c3228d56" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-999c47e1d210c3228d56">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 2: Differentially expressed proteins in KIRC vs KICH comparison.<\/caption>","data":[["PYGBAB2","OXPHOSCOMPLEXVSUBUNITB","CKIT","PR","GYGGLYCOGENIN1","PYGM","X1433EPSILON","COMPLEXIISUBUNIT30","PKM2","LDHA","ERALPHA_pS118","PYGB","LDHB","ADAR1","HIF1ALPHA","GATA6","GYS_pS641","P53","GYS","ERCC1","MSH2","CLAUDIN7","PYGL","IRS1","MITOCHONDRIA","CIAP","NDRG1_pT346","CHK2_pT68","ASNS","CHK2","PKCDELTA_pS664","EEF2K","PARP1","HER3_pY1289","PEA15_pS116","CA9","P21","CD20","ECADHERIN","BAD_pS112","PI3KP110ALPHA","CHK1","CYCLINE1","ARAF","RAD50","PARPCLEAVED","SF2","PDL1","CRAF","SCD1","SMAC","P38_pT180Y182","CMYC","GSK3_pS9","RICTOR_pT1135","INPP4B","TUBERIN","STATHMIN","EGFR_pY1068","P38MAPK","PEA15","TAZ","FOXM1","MEK1_pS217S221","JNK_pT183Y185","FOXO3A_pS318S321","P16INK4A","PCADHERIN","CYCLINB1","SNAIL","PDK1","EIF4E","NCADHERIN","ERALPHA","EGFR_pY1173","AKT_pS473","P70S6K1","RAD51","ANNEXIN1","RAB25","ATM","S6_pS240S244","KU80","P70S6K_pT389","SHC_pY317","PCNA","XBP1","BECLIN","X4EBP1","PI3KP85","CD26","TRANSGLUTAMINASE","CD31","LKB1","BCL2A1","X4EBP1_pT37T46","FOXO3A","MAPK_pT202Y204","COLLAGENVI","XRCC1","HSP70","EPPK1","RAPTOR","EGFR","IGF1R_pY1135Y1136","SMAD1","CD49B","GAPDH","PDCD4","X4EBP1_pS65","P27_pT198","DUSP4","DVL3","MTOR_pS2448","JNK2","MRE11","SETD2","GATA3","GSK3ALPHABETA_pS21S9","AMPKALPHA_pT172","LCK","YAP_pS127","SRC","S6_pS235S236","AKT_pT308","ETS1","BETACATENIN","CRAF_pS338","P27_pT157","PRAS40_pT246","STAT3_pY705","ACC1","CHK1_pS345","CASPASE3","BRAF","PTEN","GSK3ALPHABETA","JAK2","CAVEOLIN1","ACC_pS79","ARID1A","RBM15","SRC_pY527","YB1_pS102","X53BP1","DIRAS3","ERK2","BRD4","TFRC","X1433ZETA","IGFBP2","CASPASE8","MIG6","S6","BID","CYCLIND1","P90RSK_pT359S363","PKCPANBETAII_pS660","RAB11","IRF1","PAI1","BIM","HEREGULIN","PDK1_pS241","CJUN_pS73","ANNEXINVII","P27","RB_pS807S811","RICTOR","SHP2_pY542","X1433BETA","BAX","CMET_pY1235","DJ1","BCLXL","TUBERIN_pT1462","CDK1","CMET","BAP1C4","STAT5ALPHA","MTOR","P62LCKLIGAND","ERCC5","P90RSK","HER2_pY1248","AKT","PAXILLIN","AR","COG3","AMPKALPHA","CDK1_pY15","PKCALPHA","TSC1","ACVRL1","SRC_pY416","NF2","EIF4G","PRDX1","FASN","RB","BRCA2","HER3","SYK","PREX1","JAB1","CHK1_pS296","G6PD","MYOSINIIA_pS1943","CYCLINE2","YAP","NOTCH1","SMAD4","BCL2","MYH11","HER2","X4EBP1_pT70","BRAF_pS445","CABL","NRAS","EEF2","GAB2","PKCALPHA_pS657","BAK","ACETYLATUBULINLYS40","YB1","NFKBP65_pS536","MEK1","FIBRONECTIN","VEGFR2","CASPASE7CLEAVEDD198","MSH6","SMAD3","ARAF_pS299","TIGAR"],[1.544,1.411,-3.441,-0.67,1.296,1.466,-0.735,1.46,2.549,2.39,-1.401,1.683,1.697,-0.719,1.875,1.198,2.424,-1.353,2.197,0.593,-0.858,-2.08,2.22,-0.75,1.316,-0.644,3.267,-0.618,-1.398,-0.762,-0.529,1.111,1.02,-0.504,1.374,1.577,-0.749,0.5639999999999999,-1.871,0.775,-0.599,0.336,-0.647,-0.554,0.525,-0.444,-0.476,-0.662,-0.324,-0.512,-1.13,1.408,-0.867,1.012,-0.362,-1.112,-0.768,-0.413,1.535,0.548,0.657,0.458,-0.486,-0.571,-0.408,-0.264,-0.845,-0.28,-0.958,-0.395,-0.357,-0.36,0.475,-0.592,0.714,1.035,0.418,-0.404,0.9340000000000001,-0.65,1.085,0.997,0.626,-0.555,0.444,-0.25,-0.461,-0.314,0.634,0.454,0.893,1.33,0.361,-0.292,0.395,0.839,-0.273,1.02,0.59,0.322,1.01,1.645,0.483,0.423,-0.232,-0.257,-0.305,1.683,0.608,0.33,-0.195,-0.447,0.354,0.282,0.273,-0.189,-0.196,-0.421,0.589,0.795,0.377,0.553,-0.345,0.593,0.653,0.376,0.5649999999999999,-0.123,-0.151,0.5580000000000001,0.389,-0.484,-0.202,-0.208,-0.375,0.33,-0.184,0.186,0.727,-0.408,-0.116,0.435,0.5659999999999999,0.213,-0.334,0.26,0.372,0.317,-0.545,0.217,-0.365,0.142,0.316,-0.271,0.212,0.118,-0.15,-0.341,0.363,0.14,0.507,-0.204,-0.187,-0.202,0.14,-0.12,0.149,0.227,-0.41,0.239,-0.08699999999999999,0.226,-0.082,-0.146,-0.133,-0.142,0.133,-0.11,-0.112,0.246,-0.122,0.292,-0.101,0.115,0.102,0.118,0.147,0.156,0.081,-0.143,-0.176,0.147,-0.097,0.079,0.139,0.132,-0.08699999999999999,0.074,0.116,-0.046,0.077,0.148,-0.168,-0.096,-0.07099999999999999,-0.031,0.099,-0.064,0.054,0.04,0.034,-0.045,0.06900000000000001,-0.287,-0.064,0.028,-0.048,0.023,-0.028,-0.063,-0.076,0.053,-0.052,-0.073,0.031,0.054,0.033,-0.039,0.032,0.019,0.012,0.006,0.004,-0.003],[-0.391,-0.474,0.379,-0.314,-0.546,-0.44,0.255,-0.443,0.234,0.138,-0.28,-0.304,-0.3,-0.418,-0.185,-0.605,0.158,0.052,0.017,-0.107,-0.4,-0.828,0.033,-0.151,-0.533,-0.078,2.691,0.067,-0.469,-0.655,0.147,-0.283,-1.317,0.169,0.022,-0.371,0.025,0.327,-1.63,0.09,-0.194,-0.123,-0.766,0.082,-0.279,0.133,0.045,0.083,-0.215,0.323,-0.186,0.393,0.092,-0.005,0.167,-0.015,-0.276,0.058,1.255,-0.058,-0.143,0.27,-0.442,0.26,0.089,0.095,-0.528,-0.169,-2.021,0.135,0.151,-0.328,0.272,-1.414,0.41,0.293,-0.141,0.07000000000000001,0.183,-0.721,-0.118,-0.147,-0.669,0.08599999999999999,0.181,-0.194,0.056,0.014,-0.234,-0.026,0.961,0.571,0.49,0.066,0.503,-0.27,0.094,0.387,0.581,-0.205,0.4,0.132,0.113,0.554,0.235,-0.253,0.199,1.009,-0.359,-0.236,-0.148,-0.471,-0.505,-0.025,-0.102,0.143,0.008999999999999999,-0.379,0.023,-0.182,0.11,0.14,-0.005,-0.15,0.131,0.04,-0.26,-0.057,0.07000000000000001,0.163,-0.073,-1.051,-0.175,-0.216,-0.038,0.039,-0.176,-0.101,0.643,-0.756,-0.121,-0.662,0.148,0.113,-0.694,0.343,0.137,-0.547,-1.353,-0.308,-0.924,0.225,0.599,-0.58,-0.007,0.11,0.14,0.197,0.62,0.16,0.501,-0.235,0.357,0.193,0.058,0.14,0.168,-0.782,0.083,0.277,0.098,0.026,0.08599999999999999,0.077,-0.189,0.271,-0.04,0.478,-0.334,-0.55,-0.171,-0.004,0.105,0.158,0.154,-0.154,0.176,0.1,-0.134,0.72,-0.422,0.674,0.015,0.316,0.2,0.056,-0.374,0.213,-2.113,0.13,0.165,0.431,-0.203,-0.15,0.295,0.029,0.373,0.06,-0.268,0.003,0.094,0.156,0.519,1.796,-0.226,-0.073,0.016,-0.021,0.284,-1.011,0.338,0.633,0.64,0.095,-0.479,0.122,0.206,-0.235,0.29,-0.555,-0.89,0.183,0.353,-0.11],[52.186,46.552,-44.76,-41.303,38.408,35.135,-33.574,31.862,30.8,30.555,-30.001,29.878,28.96,-28.676,27.83,26.872,26.844,-26.704,26.358,25.115,-24.626,-23.897,23.765,-23.479,22.833,-22.561,22.514,-21.94,-21.532,-21.381,-20.968,19.983,19.332,-19.12,18.85,18.443,-18.33,18.25,-18.099,17.688,-17.507,17.447,-16.985,-16.976,16.274,-15.738,-15.633,-15.493,-15.426,-15.137,-14.885,14.782,-14.561,14.414,-14.338,-14.272,-14.263,-14.105,14.092,13.996,13.983,13.753,-13.533,-13.469,-13.388,-13.184,-12.91,-12.789,-12.773,-12.75,-12.701,-12.247,12.218,-12.189,12.061,11.778,11.74,-11.688,11.688,-11.601,11.44,11.437,11.354,-11.342,11.272,-11.192,-10.993,-10.913,10.844,10.596,10.55,10.518,10.516,-10.434,10.35,10.237,-9.831,9.789,9.77,9.715,9.303000000000001,9.292,9.286,9.176,-9.09,-9.09,-8.907,8.750999999999999,8.727,8.651999999999999,-8.615,-8.43,8.385,8.318,8.202,-8.179,-8.131,-8.108000000000001,7.982,7.67,7.659,7.575,-7.564,7.549,7.515,7.488,7.382,-7.364,-7.187,7.076,6.933,-6.79,-6.771,-6.769,-6.724,6.704,-6.672,6.594,6.548,-6.532,-6.5,6.319,6.194,5.936,-5.713,5.668,5.65,5.488,-5.364,5.331,-5.208,5.142,5.086,-5.066,5.034,5.024,-4.863,-4.826,4.823,4.746,4.492,-4.446,-4.391,-4.29,4.26,-4.226,4.189,4.044,-4.037,3.934,-3.86,3.853,-3.801,-3.713,-3.627,-3.581,3.484,-3.268,-3.211,3.205,-3.036,2.998,-2.924,2.873,2.622,2.568,2.564,2.437,2.401,-2.384,-2.365,2.262,-2.244,2.203,2.107,2.095,-2.05,1.934,1.866,-1.846,1.843,1.822,-1.816,-1.716,-1.598,-1.543,1.457,-1.442,1.398,1.384,1.319,-1.31,1.273,-1.259,-1.113,1.066,-1.048,1.009,-0.9320000000000001,-0.875,-0.785,0.784,-0.761,-0.6870000000000001,0.641,0.627,0.531,-0.513,0.396,0.362,0.325,0.176,0.149,-0.097],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.001,0.002,0.003,0.004,0.004,0.008999999999999999,0.01,0.011,0.015,0.017,0.017,0.018,0.024,0.025,0.028,0.035,0.037,0.041,0.053,0.062,0.065,0.066,0.06900000000000001,0.07000000000000001,0.08699999999999999,0.111,0.123,0.146,0.15,0.163,0.167,0.188,0.19,0.204,0.208,0.266,0.287,0.295,0.313,0.352,0.382,0.432,0.433,0.447,0.493,0.522,0.531,0.595,0.608,0.6929999999999999,0.717,0.745,0.861,0.881,0.923],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.002,0.002,0.003,0.004,0.005,0.005,0.011,0.013,0.013,0.019,0.021,0.021,0.022,0.029,0.03,0.034,0.043,0.044,0.048,0.063,0.073,0.076,0.077,0.08,0.08,0.099,0.126,0.14,0.165,0.168,0.182,0.186,0.208,0.21,0.224,0.228,0.29,0.311,0.318,0.336,0.376,0.406,0.457,0.457,0.469,0.515,0.543,0.55,0.614,0.624,0.708,0.73,0.755,0.868,0.885,0.923],[549.125,486.681,465.994,424.941,389.454,348.174,328.101,305.84,291.927,288.703,281.398,279.782,267.641,263.884,252.666,239.935,239.57,237.708,233.1,216.598,210.105,200.466,198.715,194.946,186.434,182.86,182.249,174.736,169.425,167.461,162.107,149.449,141.177,138.497,135.106,130.014,128.607,127.622,125.743,120.679,118.466,117.734,112.125,112.015,103.627,97.334,96.102,94.47799999999999,93.703,90.386,87.521,86.354,83.861,82.223,81.372,80.642,80.538,78.80200000000001,78.654,77.599,77.459,74.943,72.57299999999999,71.887,71.015,68.842,65.958,64.70099999999999,64.536,64.3,63.789,59.145,58.854,58.558,57.273,54.46,54.092,53.58,53.574,52.73,51.162,51.138,50.334,50.223,49.552,48.784,46.904,46.151,45.51,43.221,42.798,42.514,42.489,41.749,40.988,39.977,36.408,36.045,35.879,35.409,31.937,31.844,31.794,30.887,30.186,30.185,28.705,27.46,27.273,26.684,26.395,24.964,24.617,24.112,23.236,23.06,22.705,22.534,21.606,19.353,19.274,18.682,18.606,18.502,18.267,18.078,17.343,17.224,16.022,15.284,14.348,13.423,13.304,13.288,13.006,12.88,12.68,12.189,11.901,11.808,11.609,10.514,9.771000000000001,8.279999999999999,7.037,6.793,6.695,5.831,5.187,5.013,4.391,4.064,3.784,3.687,3.531,3.481,2.714,2.536,2.525,2.168,1.028,0.826,0.59,0.164,0.039,-0.103,-0.256,-0.841,-0.868,-1.27,-1.552,-1.577,-1.772,-2.099,-2.408,-2.572,-2.908,-3.627,-3.808,-3.829,-4.348,-4.462,-4.678,-4.824,-5.508,-5.646,-5.656,-5.972,-6.058,-6.1,-6.145,-6.381,-6.42,-6.512,-6.717,-6.743,-6.836,-7.065,-7.193,-7.23,-7.237,-7.274,-7.286,-7.463,-7.658,-7.743,-7.872,-7.893,-7.956,-7.975,-8.063000000000001,-8.074,-8.122999999999999,-8.140000000000001,-8.313000000000001,-8.364000000000001,-8.384,-8.423999999999999,-8.499000000000001,-8.550000000000001,-8.624000000000001,-8.625,-8.643000000000001,-8.696999999999999,-8.727,-8.736000000000001,-8.792,-8.801,-8.855,-8.867000000000001,-8.880000000000001,-8.917999999999999,-8.922000000000001,-8.928000000000001]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Protein<\/th>\n      <th>logFC<\/th>\n      <th>AveExpr<\/th>\n      <th>t<\/th>\n      <th>P.Value<\/th>\n      <th>adj.P.Val<\/th>\n      <th>B<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"Protein","targets":0},{"name":"logFC","targets":1},{"name":"AveExpr","targets":2},{"name":"t","targets":3},{"name":"P.Value","targets":4},{"name":"adj.P.Val","targets":5},{"name":"B","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-d91bcd5640000a0e840b" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d91bcd5640000a0e840b">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 3: Differentially expressed proteins KIRP vs KICH comparison.<\/caption>","data":[["CKIT","ERALPHA_pS118","PARP1","XRCC1","HER3_pY1289","KU80","CHK1","IRS1","P38MAPK","ARAF_pS299","ANNEXIN1","EIF4E","FASN","ECADHERIN","ADAR1","MEK1","X1433EPSILON","DVL3","ATM","EEF2K","RICTOR_pT1135","ARAF","ASNS","RAPTOR","BAD_pS112","MEK1_pS217S221","ANNEXINVII","PI3KP85","PKCDELTA_pS664","ERCC1","JNK_pT183Y185","CIAP","CMYC","RAB25","P53","PRDX1","TUBERIN","GAB2","AMPKALPHA","CLAUDIN7","P90RSK","TFRC","GSK3_pS9","P38_pT180Y182","CD26","CYCLINE1","ACC1","P70S6K1","SMAD3","SCD1","CD20","PKCALPHA","SF2","CAVEOLIN1","STATHMIN","SMAC","VEGFR2","PKCPANBETAII_pS660","ETS1","CHK2_pT68","PYGBAB2","SNAIL","PKCALPHA_pS657","CRAF","OXPHOSCOMPLEXVSUBUNITB","MYH11","AMPKALPHA_pT172","MTOR_pS2448","NDRG1_pT346","NOTCH1","S6_pS240S244","PAXILLIN","PDK1","PEA15","CASPASE3","INPP4B","BAP1C4","AR","GYGGLYCOGENIN1","RAD50","BAX","X4EBP1_pS65","P27","RB","IGFBP2","ACC_pS79","P21","MYOSINIIA_pS1943","DIRAS3","YAP","AKT_pS473","YAP_pS127","PARPCLEAVED","EPPK1","JAK2","RBM15","CDK1","SMAD4","YB1_pS102","LKB1","ERK2","MSH2","PYGM","X4EBP1_pT70","ACETYLATUBULINLYS40","FOXO3A_pS318S321","GATA6","RB_pS807S811","DUSP4","AKT_pT308","BCLXL","JAB1","S6_pS235S236","RICTOR","COMPLEXIISUBUNIT30","EGFR_pY1173","RAD51","AKT","STAT3_pY705","P27_pT198","G6PD","ERCC5","CYCLIND1","NF2","P70S6K_pT389","PDCD4","IRF1","S6","PREX1","CD31","SHC_pY317","SRC","BRAF","COG3","RAB11","TRANSGLUTAMINASE","PYGB","NCADHERIN","BECLIN","MITOCHONDRIA","DJ1","FIBRONECTIN","FOXM1","TSC1","TAZ","PEA15_pS116","HIF1ALPHA","CHK1_pS345","CJUN_pS73","CRAF_pS338","TIGAR","LDHB","XBP1","X4EBP1","PDL1","IGF1R_pY1135Y1136","ARID1A","PR","LDHA","PDK1_pS241","CABL","BCL2","MTOR","GYS","PKM2","EIF4G","CDK1_pY15","GYS_pS641","PYGL","CA9","MAPK_pT202Y204","BRD4","CHK2","CMET","CMET_pY1235","BRCA2","SMAD1","X1433BETA","P27_pT157","MIG6","X53BP1","ERALPHA","BRAF_pS445","BID","X4EBP1_pT37T46","EGFR","PCNA","GSK3ALPHABETA","EGFR_pY1068","BCL2A1","CYCLINB1","COLLAGENVI","PTEN","HER2_pY1248","YB1","GSK3ALPHABETA_pS21S9","SRC_pY416","PCADHERIN","LCK","PRAS40_pT246","JNK2","PI3KP110ALPHA","SETD2","PAI1","CD49B","MRE11","GAPDH","SHP2_pY542","CYCLINE2","MSH6","P62LCKLIGAND","HSP70","TUBERIN_pT1462","FOXO3A","CASPASE7CLEAVEDD198","HEREGULIN","BIM","HER3","NRAS","STAT5ALPHA","X1433ZETA","ACVRL1","GATA3","P90RSK_pT359S363","CASPASE8","SYK","BETACATENIN","CHK1_pS296","SRC_pY527","BAK","P16INK4A","EEF2","NFKBP65_pS536","HER2"],[-3.53,-1.251,1.358,0.739,-0.5570000000000001,1.136,0.396,-0.647,0.79,0.547,1.504,-0.518,1.079,-1.785,-0.426,-1.029,-0.358,0.6870000000000001,1.459,0.846,-0.374,-0.471,-0.894,0.709,0.584,-0.5610000000000001,0.375,0.5610000000000001,-0.312,0.285,-0.366,-0.339,-0.7,-0.654,-0.584,0.443,-0.609,-1.082,-0.668,-0.9350000000000001,0.428,-1.076,0.735,0.994,0.882,-0.396,0.733,0.363,-0.358,-0.338,0.29,0.605,-0.279,-1.015,-0.261,-0.668,-0.701,-0.601,-0.427,-0.239,0.244,-0.252,0.55,-0.171,0.245,-1.805,0.82,0.268,1.143,-0.2,0.6850000000000001,-0.448,-0.219,0.361,-0.232,-0.587,0.256,0.465,0.243,0.229,0.415,0.268,0.25,-0.172,-0.486,0.433,-0.28,-0.303,0.311,-0.196,0.592,-0.485,-0.186,1.162,-0.184,0.438,0.242,0.214,0.225,-0.174,-0.389,-0.204,0.242,-0.151,0.613,-0.114,0.249,0.311,0.293,0.478,-0.2,0.24,0.419,-0.54,0.243,0.311,-0.179,-0.231,0.281,0.112,0.334,-0.17,-0.111,0.295,-0.225,0.319,-0.135,0.241,0.253,0.153,0.175,-0.203,-0.246,0.148,-0.331,0.552,0.246,0.169,0.124,0.246,-0.166,-0.318,-0.15,-0.18,0.137,0.282,0.258,-0.114,0.125,-0.063,-0.122,0.218,-0.155,0.216,-0.157,-0.092,-0.063,0.058,0.274,-0.163,-0.08,0.187,0.138,0.263,0.253,-0.129,0.222,0.27,0.272,0.248,0.294,0.16,-0.098,0.093,-0.057,-0.111,-0.074,-0.057,0.052,-0.155,-0.145,0.117,0.107,0.097,0.185,-0.103,0.049,-0.058,0.228,-0.08,-0.155,0.123,0.098,-0.076,0.094,0.139,-0.118,0.039,-0.08400000000000001,0.126,0.053,-0.054,-0.036,0.168,-0.051,-0.034,0.28,-0.08599999999999999,0.049,0.047,0.118,-0.109,0.038,0.026,-0.044,-0.036,0.035,0.061,0.021,0.052,-0.024,0.02,-0.025,-0.014,0.013,0.025,-0.018,-0.004,0.015,-0.007,-0.005,0.004,0.002,0],[0.379,-0.28,-1.317,-0.205,0.169,-0.669,-0.123,-0.151,-0.058,0.353,0.183,-0.328,-2.113,-1.63,-0.418,0.206,0.255,-0.505,-0.118,-0.283,0.167,0.082,-0.469,0.113,0.09,0.26,0.14,-0.026,0.147,-0.107,0.089,-0.078,0.092,-0.721,0.052,0.213,-0.276,0.338,0.72,-0.828,0.158,-1.353,-0.005,0.393,0.961,-0.766,-1.051,-0.141,0.183,0.323,0.327,0.674,0.045,0.643,0.058,-0.186,0.29,0.197,0.04,0.067,-0.391,0.135,0.633,-0.215,-0.474,1.796,-0.182,-0.025,2.691,0.094,-0.147,0.176,0.151,-0.143,-0.216,-0.015,-0.334,0.1,-0.546,-0.279,0.026,-0.236,0.168,0.13,-0.924,-0.756,0.025,0.06,0.343,0.003,0.293,0.14,0.133,0.132,-0.101,-0.662,-0.04,0.156,0.113,0.066,0.137,-0.4,-0.44,-0.073,0.095,0.095,-0.605,-0.782,-0.471,0.131,-0.189,0.295,-0.15,0.083,-0.443,0.41,0.07000000000000001,-0.154,-0.073,-0.148,0.373,0.105,0.11,0.056,0.08599999999999999,-0.359,0.16,-0.58,-0.15,0.49,0.181,-0.005,-0.038,-0.134,0.62,0.571,-0.304,0.272,0.014,-0.533,0.077,-0.235,-0.442,0.015,0.27,0.022,-0.185,-0.175,0.058,-0.057,-0.11,-0.3,0.056,-0.234,0.083,0.235,-0.121,-0.314,0.138,0.193,-0.021,0.519,-0.171,0.017,0.234,-0.374,-0.422,0.158,0.033,-0.371,0.387,-0.547,-0.655,0.478,0.08599999999999999,0.165,-0.253,0.098,0.07000000000000001,0.599,-0.694,-1.414,0.016,-0.007,-0.27,0.554,-0.194,-0.176,1.255,0.503,-2.021,0.581,0.039,0.154,-0.479,0.023,0.2,-0.169,0.11,0.163,-0.102,-0.194,0.008999999999999999,0.501,0.199,0.143,1.009,0.277,-0.268,-0.89,-0.004,0.4,0.271,0.094,-0.555,0.357,-0.235,0.431,0.284,-0.55,-0.308,0.316,-0.379,0.14,0.225,-0.203,-0.26,0.029,0.148,0.64,-0.528,-1.011,0.122,-0.226],[-42.468,-24.773,23.795,20.608,-19.564,19.064,19.003,-18.747,18.683,17.741,17.406,-16.306,16.05,-15.972,-15.706,-15.212,-15.116,15.049,14.231,14.075,-13.692,-13.353,-12.745,12.591,12.329,-12.239,12.217,12.121,-11.439,11.167,-11.097,-10.991,-10.873,-10.8,-10.659,10.629,-10.465,-10.348,-10.275,-9.935,9.897,-9.797000000000001,9.692,9.656000000000001,9.643000000000001,-9.621,9.513,9.432,-9.263999999999999,-9.234,8.677,8.628,-8.496,-8.456,-8.256,-8.134,-7.961,-7.876,-7.857,-7.848,7.622,-7.534,7.529,-7.527,7.487,-7.317,7.313,7.303,7.284,-7.262,7.261,-7.224,-7.212,7.108,-6.979,-6.973,6.768,6.718,6.655,6.579,6.541,6.5,6.481,-6.434,-6.417,6.411,-6.333,-6.297,6.257,-6.234,6.225,-6.139,-6.1,6.07,-6.03,5.889,5.878,5.812,5.788,-5.766,-5.463,-5.412,5.363,-5.36,5.317,-5.276,5.169,5.139,5.114,5.089,-5.044,5.024,4.934,-4.918,4.904,4.852,-4.791,-4.671,4.625,4.594,4.562,-4.538,-4.392,4.342,-4.253,4.236,-4.211,4.178,4.169,4.125,4.123,-4.117,-4.087,4.072,-4.062,4.038,4.035,4.014,3.973,3.94,-3.916,-3.872,-3.859,-3.833,3.805,3.577,3.54,-3.534,3.514,-3.467,-3.439,3.434,-3.423,3.417,-3.403,-3.33,-3.296,3.282,3.236,-3.197,-3.189,3.181,3.167,2.923,2.827,-2.815,2.766,2.762,2.691,2.684,2.613,2.554,-2.553,2.54,-2.466,-2.453,-2.409,-2.329,2.316,-2.302,-2.288,2.237,2.175,2.132,2.089,-2.072,2.015,-1.95,1.936,-1.928,-1.91,1.882,1.844,-1.803,1.801,1.744,-1.651,1.635,-1.569,1.481,1.475,-1.46,-1.402,1.38,-1.368,-1.358,1.345,-1.305,1.183,1.14,1.12,-0.925,0.9,0.875,-0.78,-0.778,0.715,0.697,0.639,0.627,-0.539,0.505,-0.446,-0.435,0.426,0.245,-0.223,-0.161,0.149,-0.101,-0.077,0.047,0.026,0.005],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.002,0.002,0.004,0.005,0.005,0.006,0.006,0.007,0.007,0.008999999999999999,0.011,0.011,0.011,0.014,0.014,0.016,0.02,0.021,0.022,0.022,0.026,0.03,0.033,0.037,0.039,0.044,0.052,0.053,0.054,0.057,0.06,0.066,0.07199999999999999,0.07199999999999999,0.082,0.099,0.102,0.117,0.139,0.141,0.145,0.161,0.168,0.172,0.175,0.179,0.192,0.237,0.255,0.263,0.356,0.369,0.382,0.436,0.437,0.475,0.486,0.523,0.531,0.59,0.614,0.656,0.664,0.67,0.806,0.823,0.872,0.882,0.92,0.9389999999999999,0.962,0.979,0.996],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.002,0.002,0.002,0.002,0.002,0.002,0.002,0.005,0.007,0.007,0.008,0.008,0.01,0.01,0.013,0.015,0.015,0.015,0.019,0.019,0.022,0.026,0.027,0.028,0.029,0.033,0.038,0.042,0.047,0.049,0.055,0.064,0.066,0.067,0.06900000000000001,0.073,0.08,0.08599999999999999,0.08599999999999999,0.097,0.118,0.121,0.138,0.163,0.164,0.168,0.186,0.193,0.196,0.199,0.202,0.216,0.265,0.284,0.292,0.392,0.405,0.418,0.473,0.473,0.512,0.522,0.5590000000000001,0.5649999999999999,0.625,0.647,0.6879999999999999,0.6929999999999999,0.697,0.835,0.849,0.895,0.901,0.9360000000000001,0.951,0.971,0.983,0.996],[439.181,212.492,199.553,157.935,144.598,138.279,137.506,134.292,133.488,121.825,117.719,104.499,101.478,100.557,97.449,91.747,90.651,89.88500000000001,80.697,78.968,74.789,71.157,64.755,63.166,60.492,59.581,59.358,58.389,51.67,49.066,48.402,47.41,46.301,45.621,44.319,44.049,42.553,41.491,40.838,37.832,37.506,36.639,35.739,35.428,35.321,35.126,34.211,33.533,32.133,31.889,27.409,27.026,25.994,25.691,24.17,23.253,21.978,21.357,21.218,21.156,19.544,18.928,18.895,18.875,18.596,17.434,17.404,17.338,17.206,17.055,17.051,16.804,16.722,16.029,15.176,15.135,13.819,13.496,13.104,12.627,12.393,12.138,12.024,11.739,11.633,11.599,11.125,10.913,10.675,10.539,10.483,9.981999999999999,9.753,9.581,9.349,8.548,8.486000000000001,8.119999999999999,7.985,7.863,6.235,5.967,5.712,5.7,5.477,5.269,4.731,4.583,4.457,4.333,4.116,4.015,3.584,3.507,3.443,3.194,2.912,2.359,2.153,2.012,1.872,1.763,1.129,0.918,0.544,0.476,0.371,0.236,0.2,0.021,0.013,-0.013,-0.131,-0.192,-0.231,-0.326,-0.338,-0.422,-0.583,-0.708,-0.803,-0.97,-1.02,-1.118,-1.22,-2.047,-2.178,-2.2,-2.267,-2.43,-2.524,-2.542,-2.578,-2.598,-2.645,-2.888,-2.999,-3.043,-3.192,-3.315,-3.339,-3.367,-3.41,-4.144,-4.417,-4.45,-4.586,-4.596,-4.788,-4.808,-4.994,-5.143,-5.146,-5.179,-5.363,-5.394,-5.502,-5.688,-5.72,-5.75,-5.784,-5.897,-6.034,-6.126,-6.216,-6.252,-6.368,-6.496,-6.522,-6.538,-6.573,-6.625,-6.696,-6.771,-6.773,-6.875,-7.032,-7.058,-7.163,-7.298,-7.307,-7.328,-7.411,-7.441,-7.458,-7.472,-7.489,-7.542,-7.694,-7.744,-7.766,-7.966,-7.989,-8.010999999999999,-8.09,-8.090999999999999,-8.138,-8.151,-8.189,-8.196999999999999,-8.247999999999999,-8.266,-8.294,-8.298999999999999,-8.303000000000001,-8.364000000000001,-8.369,-8.381,-8.382999999999999,-8.388999999999999,-8.391,-8.393000000000001,-8.393000000000001,-8.394]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Protein<\/th>\n      <th>logFC<\/th>\n      <th>AveExpr<\/th>\n      <th>t<\/th>\n      <th>P.Value<\/th>\n      <th>adj.P.Val<\/th>\n      <th>B<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"Protein","targets":0},{"name":"logFC","targets":1},{"name":"AveExpr","targets":2},{"name":"t","targets":3},{"name":"P.Value","targets":4},{"name":"adj.P.Val","targets":5},{"name":"B","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-bafdd86f7813d473181e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-bafdd86f7813d473181e">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 4: Differentially expressed proteins in KIRC vs KIRP comparison.<\/caption>","data":[["PR","PYGBAB2","OXPHOSCOMPLEXVSUBUNITB","GYGGLYCOGENIN1","PYGM","PKM2","LDHA","COMPLEXIISUBUNIT30","PYGB","LDHB","HIF1ALPHA","GYS_pS641","GYS","GATA6","PYGL","ARAF_pS299","MSH2","CHK2","MITOCHONDRIA","ANNEXINVII","X1433EPSILON","ACC1","MEK1","ETS1","PI3KP110ALPHA","CAVEOLIN1","CA9","FASN","BECLIN","P53","PEA15_pS116","NDRG1_pT346","ERALPHA","PCADHERIN","YAP_pS127","DUSP4","P27_pT198","ACC_pS79","CHK2_pT68","PCNA","CLAUDIN7","JAK2","ERCC1","P16INK4A","XRCC1","BCL2A1","EGFR_pY1068","PDL1","ADAR1","ERK2","P21","EGFR","FOXO3A","CYCLINB1","CIAP","BAP1C4","GAB2","PAXILLIN","HSP70","SMAD3","CYCLIND1","P27_pT157","TAZ","S6","PRDX1","FOXM1","LCK","IRF1","KU80","RAB11","NOTCH1","RAD50","PARPCLEAVED","VEGFR2","CD20","AMPKALPHA","PKCDELTA_pS664","YAP","X4EBP1_pT37T46","DVL3","NCADHERIN","P90RSK","ASNS","COLLAGENVI","GATA3","BETACATENIN","AKT","SMAD4","MIG6","FOXO3A_pS318S321","CD49B","PKCALPHA_pS657","GAPDH","XBP1","CRAF","X4EBP1","ANNEXIN1","PKCALPHA","JAB1","MAPK_pT202Y204","X4EBP1_pT70","EGFR_pY1173","SHC_pY317","P70S6K_pT389","INPP4B","MRE11","MYH11","SETD2","JNK2","CYCLINE1","RAD51","SMAD1","MTOR","SF2","ACETYLATUBULINLYS40","PARP1","PEA15","PREX1","P38MAPK","TRANSGLUTAMINASE","GSK3ALPHABETA_pS21S9","SMAC","CD31","SRC_pY527","CMET","X1433ZETA","IGF1R_pY1135Y1136","PRAS40_pT246","EIF4E","MYOSINIIA_pS1943","CDK1_pY15","SHP2_pY542","TFRC","BIM","STATHMIN","SCD1","RB","AKT_pS473","PDK1","AR","EEF2K","PTEN","CASPASE8","SNAIL","HER2_pY1248","GSK3ALPHABETA","TUBERIN_pT1462","BRCA2","CABL","P90RSK_pT359S363","BAD_pS112","P38_pT180Y182","RAPTOR","LKB1","PDCD4","ATM","GSK3_pS9","SRC_pY416","PKCPANBETAII_pS660","FIBRONECTIN","TIGAR","CRAF_pS338","S6_pS240S244","HEREGULIN","G6PD","BRAF_pS445","X53BP1","BAX","ERALPHA_pS118","IRS1","SRC","CHK1","PAI1","CHK1_pS345","TUBERIN","ARID1A","CDK1","P27","CMYC","EPPK1","BID","BRD4","NF2","ARAF","STAT5ALPHA","PI3KP85","BRAF","S6_pS235S236","BCL2","SYK","HER3_pY1289","AKT_pT308","COG3","ERCC5","STAT3_pY705","TSC1","BCLXL","P62LCKLIGAND","IGFBP2","ACVRL1","X4EBP1_pS65","NRAS","P70S6K1","RB_pS807S811","JNK_pT183Y185","CHK1_pS296","X1433BETA","YB1","RICTOR","CASPASE7CLEAVEDD198","CKIT","CMET_pY1235","HER2","DIRAS3","HER3","EIF4G","EEF2","MSH6","PDK1_pS241","ECADHERIN","CASPASE3","BAK","NFKBP65_pS536","DJ1","RICTOR_pT1135","CJUN_pS73","MTOR_pS2448","YB1_pS102","MEK1_pS217S221","AMPKALPHA_pT172","CD26","CYCLINE2","RAB25","RBM15"],[-0.728,1.3,1.166,1.053,1.224,2.296,2.116,1.217,1.437,1.479,1.617,2.154,1.933,0.949,1.948,-0.543,-0.654,-0.664,1.071,-0.495,-0.377,-1.218,1.062,0.803,-0.545,1.742,1.329,-0.963,-0.438,-0.769,1.092,2.124,-0.709,-0.318,1.038,-0.741,-0.307,-0.842,-0.379,-0.299,-1.145,0.37,0.308,-0.84,-0.417,0.475,1.307,-0.505,-0.293,0.761,-0.469,0.526,-0.299,-0.803,-0.305,-0.368,1.006,0.596,1.119,0.364,0.229,-0.203,0.321,-0.512,-0.368,-0.336,0.461,0.275,-0.51,0.694,0.234,0.295,-0.258,0.733,0.274,0.525,-0.217,0.237,0.653,-0.333,0.306,-0.313,-0.503,0.467,-0.396,0.583,0.349,-0.258,0.471,-0.15,-0.254,-0.497,1.403,-0.306,-0.153,0.418,-0.57,-0.458,-0.31,0.726,0.179,0.403,0.268,-0.33,-0.525,-0.155,1.518,-0.159,0.22,-0.251,-0.225,-0.183,-0.26,-0.196,-0.6860000000000001,-0.337,0.296,-0.35,-0.243,0.778,0.449,-0.463,0.208,0.551,-0.203,0.24,-0.14,0.432,0.158,0.239,-0.398,0.325,0.531,-0.239,-0.152,-0.174,0.126,0.444,-0.138,-0.309,0.265,0.232,0.129,-0.143,0.178,-0.126,-0.18,0.188,0.103,-0.135,0.191,0.414,-0.225,-0.117,0.289,-0.374,0.276,0.257,0.261,0.279,0.119,-0.06,0.313,-0.151,-0.235,-0.155,-0.189,-0.189,-0.15,-0.103,-0.142,-0.06,0.338,-0.08799999999999999,-0.159,-0.052,-0.109,-0.1,-0.167,0.483,0.115,0.158,-0.164,-0.083,0.194,-0.107,-0.128,0.174,-0.118,-0.193,0.054,0.175,-0.067,0.06900000000000001,0.108,0.082,0.067,0.174,0.121,0.059,0.062,-0.049,0.055,-0.08500000000000001,-0.042,-0.028,-0.03,-0.063,0.13,0.063,0.089,-0.024,-0.064,-0.05,0.08699999999999999,0.042,-0.066,-0.035,-0.039,-0.08599999999999999,0.024,-0.045,0.051,0.02,0.012,0.015,0.014,-0.012,-0.01,-0.025,0.01,0.005,0.004,-0.003],[-0.314,-0.391,-0.474,-0.546,-0.44,0.234,0.138,-0.443,-0.304,-0.3,-0.185,0.158,0.017,-0.605,0.033,0.353,-0.4,-0.655,-0.533,0.14,0.255,-1.051,0.206,0.04,-0.194,0.643,-0.371,-2.113,0.014,0.052,0.022,2.691,-1.414,-0.169,0.14,-0.471,-0.148,-0.756,0.067,-0.194,-0.828,-0.101,-0.107,-0.528,-0.205,0.503,1.255,0.083,-0.418,0.137,0.025,0.554,0.094,-2.021,-0.078,-0.334,0.338,0.176,0.4,0.183,0.11,0.07000000000000001,0.27,-0.58,0.213,-0.442,0.11,0.16,-0.669,0.62,0.094,-0.279,0.133,0.29,0.327,0.72,0.147,0.003,-0.27,-0.505,0.272,0.158,-0.469,0.581,-0.379,-0.26,-0.154,0.156,0.599,0.095,0.199,0.633,1.009,0.056,-0.215,-0.234,0.183,0.674,0.295,0.387,-0.073,0.41,0.181,0.08599999999999999,-0.015,0.143,1.796,0.008999999999999999,-0.102,-0.766,0.07000000000000001,-0.253,-0.171,0.045,0.095,-1.317,-0.143,-0.15,-0.058,0.571,0.023,-0.186,0.49,0.148,0.478,-0.308,0.235,0.163,-0.328,0.06,-0.422,0.277,-1.353,-0.235,0.058,0.323,0.13,0.293,0.151,0.1,-0.283,0.039,0.225,0.135,0.154,-0.176,0.271,0.165,-0.021,0.14,0.09,0.393,0.113,0.066,-0.359,-0.118,-0.005,0.2,0.197,-0.235,-0.11,-0.057,-0.147,0.357,0.373,0.016,-0.694,0.026,-0.28,-0.151,-0.005,-0.123,0.501,-0.175,-0.276,-0.121,-0.04,0.168,0.092,0.132,-0.007,-0.547,0.056,0.082,-0.55,-0.026,-0.038,-0.15,0.519,-0.203,0.169,0.131,-0.134,0.105,-0.073,0.015,-0.189,-0.004,-0.924,0.316,-0.236,0.284,-0.141,-0.782,0.089,0.029,0.098,-0.479,0.083,-0.555,0.379,0.08599999999999999,-0.226,0.343,0.431,-0.374,-1.011,-0.89,0.193,-1.63,-0.216,0.64,0.122,0.077,0.167,0.058,-0.025,0.113,0.26,-0.182,0.961,-0.268,-0.721,-0.662],[-68.158,66.78100000000001,58.442,47.433,44.582,42.161,41.117,40.361,38.775,38.367,36.476,36.256,35.252,32.343,31.693,-28.921,-28.531,-28.296,28.224,-26.495,-26.185,-25.947,25.8,24.287,-24.205,23.843,23.618,-23.533,-23.11,-23.069,22.768,22.246,-22.199,-22.121,21.597,-21.213,-20.639,-20.46,-20.447,-20.318,-19.994,19.927,19.82,-19.491,-19.094,18.895,18.234,-17.952,-17.774,17.561,-17.451,17.348,-16.377,-16.272,-16.226,-16,15.807,15.766,15.657,15.486,14.85,-14.726,14.647,-14.563,-14.524,-14.226,14.217,14.131,-14.068,14.003,13.935,13.922,-13.895,13.681,13.478,13.259,-13.07,12.346,12.124,-11.983,11.973,-11.894,-11.782,11.754,-11.589,11.585,11.577,-11.541,11.511,-11.366,-11.287,-11.179,11.088,-11.082,-11.075,10.865,-10.835,-10.739,-10.681,10.583,10.427,10.357,10.356,-10.249,-10.233,-10.197,10.108,-10.053,10.042,-10.005,-9.890000000000001,-9.856999999999999,-9.817,-9.798,-9.779,-9.715,9.571,-9.457000000000001,-9.426,9.35,9.265000000000001,-9.257,9.202999999999999,9.169,-9.140000000000001,8.986000000000001,-8.343999999999999,8.32,8.178000000000001,8.154,-8.137,8.122,7.944,-7.931,-7.871,-7.832,7.765,7.671,-7.452,-7.333,7.243,7.158,7.115,-6.998,6.946,-6.936,-6.919,6.831,6.773,-6.676,6.623,6.6,-6.575,-6.383,6.302,-5.997,5.98,5.915,5.606,5.581,5.503,-5.496,5.451,-5.395,-5.282,-5.165,-4.922,-4.891,-4.891,-4.88,-4.731,-4.708,4.559,-4.484,-4.48,-4.463,-4.362,-4.282,-4.264,4.148,4.147,4.143,-3.951,-3.858,3.84,-3.812,-3.503,3.365,-3.291,-3.162,3.088,3.06,-3.041,3.011,2.937,2.886,2.776,2.716,2.628,2.518,2.47,-2.466,2.345,-2.299,-2.113,-2.081,-2.039,-1.986,1.945,1.831,1.753,-1.725,-1.7,-1.667,1.625,1.51,-1.407,-1.378,-1.267,-1.263,1.18,-0.991,0.909,0.791,0.706,0.701,0.642,-0.489,-0.36,-0.359,0.188,0.18,0.114,-0.07199999999999999],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.002,0.002,0.002,0.002,0.003,0.003,0.004,0.006,0.007,0.008999999999999999,0.012,0.014,0.014,0.019,0.022,0.035,0.038,0.042,0.047,0.052,0.067,0.08,0.08500000000000001,0.09,0.096,0.105,0.132,0.16,0.169,0.205,0.207,0.238,0.322,0.364,0.429,0.48,0.484,0.521,0.625,0.719,0.72,0.851,0.857,0.91,0.9429999999999999],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.001,0.001,0.001,0.002,0.003,0.003,0.003,0.003,0.004,0.005,0.007,0.008,0.01,0.014,0.016,0.016,0.022,0.025,0.04,0.043,0.047,0.053,0.058,0.075,0.089,0.094,0.098,0.105,0.114,0.142,0.172,0.181,0.22,0.22,0.252,0.339,0.381,0.448,0.5,0.501,0.537,0.642,0.732,0.732,0.862,0.864,0.913,0.9429999999999999],[705.924,693.456,613.774,496.454,463.651,434.998,422.416,413.225,393.718,388.657,364.938,362.146,349.379,311.822,303.323,266.815,261.642,258.535,257.572,234.602,230.488,227.319,225.375,205.303,204.217,199.425,196.447,195.329,189.756,189.221,185.263,178.415,177.794,176.782,169.948,164.951,157.538,155.236,155.062,153.407,149.256,148.41,147.035,142.865,137.845,135.338,127.081,123.598,121.406,118.793,117.444,116.195,104.508,103.272,102.717,100.061,97.797,97.318,96.04600000000001,94.069,86.78,85.38,84.48999999999999,83.545,83.114,79.795,79.69499999999999,78.748,78.05,77.34,76.58799999999999,76.45,76.155,73.82299999999999,71.636,69.298,67.301,59.81,57.56,56.151,56.053,55.268,54.159,53.879,52.271,52.224,52.151,51.797,51.513,50.112,49.354,48.319,47.457,47.399,47.339,45.361,45.088,44.191,43.66,42.762,41.34,40.715,40.701,39.739,39.596,39.283,38.491,38.006,37.904,37.579,36.58,36.29,35.942,35.784,35.613,35.067,33.835,32.875,32.611,31.979,31.274,31.208,30.76,30.483,30.243,28.995,23.958,23.781,22.711,22.528,22.406,22.295,20.981,20.884,20.445,20.164,19.687,19.016,17.481,16.665,16.053,15.484,15.196,14.421,14.084,14.019,13.91,13.343,12.969,12.355,12.022,11.879,11.722,10.551,10.063,8.279999999999999,8.186,7.815,6.115,5.981,5.564,5.526,5.292,4.997,4.417,3.829,2.645,2.497,2.495,2.446,1.751,1.644,0.974,0.645,0.628,0.553,0.121,-0.218,-0.295,-0.771,-0.774,-0.79,-1.551,-1.907,-1.973,-2.078,-3.189,-3.655,-3.898,-4.309,-4.538,-4.623,-4.682,-4.77,-4.987,-5.135,-5.444,-5.608,-5.84,-6.12,-6.24,-6.249,-6.538,-6.645,-7.052,-7.119,-7.206,-7.312,-7.391,-7.606,-7.746,-7.794,-7.837,-7.892,-7.962,-8.141,-8.289999999999999,-8.331,-8.478,-8.483000000000001,-8.584,-8.789,-8.867000000000001,-8.968,-9.031000000000001,-9.035,-9.074,-9.161,-9.215999999999999,-9.215999999999999,-9.263,-9.265000000000001,-9.273999999999999,-9.278]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Protein<\/th>\n      <th>logFC<\/th>\n      <th>AveExpr<\/th>\n      <th>t<\/th>\n      <th>P.Value<\/th>\n      <th>adj.P.Val<\/th>\n      <th>B<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"name":"Protein","targets":0},{"name":"logFC","targets":1},{"name":"AveExpr","targets":2},{"name":"t","targets":3},{"name":"P.Value","targets":4},{"name":"adj.P.Val","targets":5},{"name":"B","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="comparative-enrichment-analyses-across-cancer-types" class="level3" data-number="3.4.2">
<h3 data-number="3.4.2" class="anchored" data-anchor-id="comparative-enrichment-analyses-across-cancer-types"><span class="header-section-number">3.4.2</span> Comparative enrichment analyses across cancer types</h3>
<p>In the next steps we further characterize the differentially abundant proteins across the three kidney cancers by performing a Gene Set Enrichment Analysis (GSEA).</p>
<section id="go-biological-process-terms" class="level4" data-number="3.4.2.1">
<h4 data-number="3.4.2.1" class="anchored" data-anchor-id="go-biological-process-terms"><span class="header-section-number">3.4.2.1</span> GO Biological Process terms</h4>
<p>Gene Ontology Biological Process are the larger processes or ‘biological programs’ accomplished by the concerted action of multiple molecular activities. Examples of broad BP terms are <em>“DNA repair”</em> or <em>“signal transduction”</em>. Examples of more specific terms are <em>“cytosine biosynthetic process”</em> or <em>“D-glucose transmembrane transport”</em>.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 11: dotplot of enriched GO Biological Process terms.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-22-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 12: Term maps reporting relationships between GO Biological Process across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-23-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 13: gene maps reporting relationships between GO Biological Process enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>The enriched GO BP terms in KIRC vs.&nbsp;KIRP and KIRC vs.&nbsp;KICH suggest that KIRC has a strong enrichment in its catabolic metabolism, with enriched terms such as <em>“carbohydrate catabolic process”</em>, <em>“glycogen catabolic process”</em> and <em>“glycogen metabolic process”</em>. Enrichment of these terms are driven by the overabundance of Glycogen Phosphorylase B, L and M (<em>PYGB</em>, <em>PYGL</em>, <em>PYGM</em>), Lactate Dehydrogenase A (<em>LDHA</em>) and Glyceraldehyde-3-Phosphate Dehydrogenase (<em>GAPDH</em>) in KIRC when compared to KIRP and KICH.</p>
<p>KICH seems instead to retain more its differentiation as highlighted by the lower abundance of Poly(ADP-Ribose) Polymerase 1 (<em>PARP1</em>) when compared to KIRC and KIRP, resulting in the enrichment of GO BP terms related to <em>“Negative regulation of telomere maintainance”</em>.</p>
<p>The complete list of 227 GO BP terms enriched across the three contrasts is reported below.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-20da075b4c71c4d0002a" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-20da075b4c71c4d0002a">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 5: enriched GO BP terms.<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["GO:0016052","GO:0005980","GO:0009251","GO:0000272","GO:0005977","GO:0044042","GO:0006112","GO:0005976","GO:0036293","GO:0006089","GO:0006090","GO:0070482","GO:0036092","GO:0032205","GO:1990090","GO:1990089","GO:0070266","GO:0010332","GO:0097300","GO:1904356","GO:0051402","GO:0046854","GO:0001666","GO:0010833","GO:0002573","GO:0030099","GO:0032205","GO:0051054","GO:0010332","GO:0006310","GO:1904356","GO:0010639","GO:0030888","GO:0045911","GO:0002562","GO:0016444","GO:0002200","GO:0010833","GO:1900182","GO:2001251","GO:0032204","GO:0042100","GO:0030316","GO:0032526","GO:0050864","GO:0000018","GO:0010212","GO:1900180","GO:0051053","GO:0000723","GO:0000724","GO:0000725","GO:0071478","GO:0032200","GO:0043123","GO:0031334","GO:0002520","GO:0043523","GO:0050670","GO:0002274","GO:0032944","GO:0033044","GO:0002703","GO:0034599","GO:0014866","GO:0030242","GO:0033572","GO:0045141","GO:0048251","GO:0051409","GO:1900543","GO:1904760","GO:0043122","GO:0070663","GO:0008611","GO:0030223","GO:0033148","GO:0045980","GO:0046504","GO:0062100","GO:0070213","GO:0071044","GO:0071481","GO:0090220","GO:0090557","GO:1904181","GO:0002699","GO:0042113","GO:0051402","GO:0000012","GO:0002327","GO:0010635","GO:0033145","GO:0034397","GO:0097384","GO:1901503","GO:1903624","GO:0018105","GO:0001666","GO:0007249","GO:0002366","GO:0051547","GO:0002263","GO:0006302","GO:0046651","GO:0018209","GO:0034504","GO:0032943","GO:0036293","GO:0033008","GO:0036446","GO:0042159","GO:0043306","GO:0045187","GO:0062197","GO:0097193","GO:0044403","GO:0043517","GO:0050802","GO:1900034","GO:1904177","GO:0071214","GO:0104004","GO:0030889","GO:0033151","GO:0090399","GO:0070482","GO:0006977","GO:0010042","GO:0045780","GO:1904044","GO:0070661","GO:0022410","GO:0042749","GO:1903376","GO:2001169","GO:0010038","GO:0008334","GO:0034244","GO:0051546","GO:0051900","GO:1903978","GO:0006662","GO:0033005","GO:0036480","GO:0042745","GO:0044030","GO:0046485","GO:0002696","GO:0023019","GO:0030431","GO:0032042","GO:0036092","GO:0002697","GO:0032785","GO:0051882","GO:0050867","GO:0006979","GO:0010839","GO:0071294","GO:1903578","GO:0043302","GO:0046697","GO:0043254","GO:0018904","GO:0060391","GO:1904353","GO:0071480","GO:1904357","GO:0009314","GO:0016540","GO:0048732","GO:0010165","GO:0031571","GO:0044819","GO:0045830","GO:0000002","GO:0010039","GO:0043304","GO:1901798","GO:0003012","GO:0008053","GO:0002443","GO:0006308","GO:0030808","GO:0033006","GO:1900371","GO:0051656","GO:0050869","GO:1903829","GO:0032102","GO:1903131","GO:0032212","GO:0071353","GO:0045785","GO:0001893","GO:0008340","GO:0010613","GO:0030224","GO:0033146","GO:0060544","GO:0002068","GO:0014742","GO:0070670","GO:0007095","GO:0046688","GO:0051385","GO:1904358","GO:1905168","GO:0007143","GO:0043516","GO:0016233","GO:0045191","GO:0016052","GO:0005980","GO:0009251","GO:0000272","GO:0005977","GO:0044042","GO:0006112","GO:0005976","GO:0006090","GO:0006089","GO:0015980","GO:0006096","GO:0014866","GO:0048251"],["carbohydrate catabolic process","glycogen catabolic process","glucan catabolic process","polysaccharide catabolic process","glycogen metabolic process","glucan metabolic process","energy reserve metabolic process","polysaccharide metabolic process","response to decreased oxygen levels","lactate metabolic process","pyruvate metabolic process","response to oxygen levels","phosphatidylinositol-3-phosphate biosynthetic process","negative regulation of telomere maintenance","cellular response to nerve growth factor stimulus","response to nerve growth factor","necroptotic process","response to gamma radiation","programmed necrotic cell death","regulation of telomere maintenance via telomere lengthening","neuron apoptotic process","phosphatidylinositol phosphate biosynthetic process","response to hypoxia","telomere maintenance via telomere lengthening","myeloid leukocyte differentiation","myeloid cell differentiation","negative regulation of telomere maintenance","positive regulation of DNA metabolic process","response to gamma radiation","DNA recombination","regulation of telomere maintenance via telomere lengthening","negative regulation of organelle organization","regulation of B cell proliferation","positive regulation of DNA recombination","somatic diversification of immune receptors via germline recombination within a single locus","somatic cell DNA recombination","somatic diversification of immune receptors","telomere maintenance via telomere lengthening","positive regulation of protein localization to nucleus","negative regulation of chromosome organization","regulation of telomere maintenance","B cell proliferation","osteoclast differentiation","response to retinoic acid","regulation of B cell activation","regulation of DNA recombination","response to ionizing radiation","regulation of protein localization to nucleus","negative regulation of DNA metabolic process","telomere maintenance","double-strand break repair via homologous recombination","recombinational repair","cellular response to radiation","telomere organization","positive regulation of canonical NF-kappaB signal transduction","positive regulation of protein-containing complex assembly","immune system development","regulation of neuron apoptotic process","regulation of lymphocyte proliferation","myeloid leukocyte activation","regulation of mononuclear cell proliferation","regulation of chromosome organization","regulation of leukocyte mediated immunity","cellular response to oxidative stress","skeletal myofibril assembly","autophagy of peroxisome","transferrin transport","meiotic telomere clustering","elastic fiber assembly","response to nitrosative stress","negative regulation of purine nucleotide metabolic process","regulation of myofibroblast differentiation","regulation of canonical NF-kappaB signal transduction","regulation of leukocyte proliferation","ether lipid biosynthetic process","neutrophil differentiation","positive regulation of intracellular estrogen receptor signaling pathway","negative regulation of nucleotide metabolic process","glycerol ether biosynthetic process","positive regulation of programmed necrotic cell death","protein auto-ADP-ribosylation","histone mRNA catabolic process","cellular response to X-ray","chromosome localization to nuclear envelope involved in homologous chromosome segregation","establishment of endothelial intestinal barrier","positive regulation of membrane depolarization","positive regulation of immune effector process","B cell activation","neuron apoptotic process","single strand break repair","immature B cell differentiation","regulation of mitochondrial fusion","positive regulation of intracellular steroid hormone receptor signaling pathway","telomere localization","cellular lipid biosynthetic process","ether biosynthetic process","regulation of DNA catabolic process","peptidyl-serine phosphorylation","response to hypoxia","canonical NF-kappaB signal transduction","leukocyte activation involved in immune response","regulation of keratinocyte migration","cell activation involved in immune response","double-strand break repair","lymphocyte proliferation","peptidyl-serine modification","protein localization to nucleus","mononuclear cell proliferation","response to decreased oxygen levels","positive regulation of mast cell activation involved in immune response","myofibroblast differentiation","lipoprotein catabolic process","positive regulation of mast cell degranulation","regulation of circadian sleep/wake cycle, sleep","cellular response to chemical stress","intrinsic apoptotic signaling pathway","biological process involved in symbiotic interaction","positive regulation of DNA damage response, signal transduction by p53 class mediator","circadian sleep/wake cycle, sleep","regulation of cellular response to heat","regulation of adipose tissue development","cellular response to abiotic stimulus","cellular response to environmental stimulus","negative regulation of B cell proliferation","V(D)J recombination","replicative senescence","response to oxygen levels","DNA damage response, signal transduction by p53 class mediator resulting in cell cycle arrest","response to manganese ion","positive regulation of bone resorption","response to aldosterone","leukocyte proliferation","circadian sleep/wake cycle process","regulation of circadian sleep/wake cycle","regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway","regulation of ATP biosynthetic process","response to metal ion","histone mRNA metabolic process","negative regulation of transcription elongation by RNA polymerase II","keratinocyte migration","regulation of mitochondrial depolarization","regulation of microglial cell activation","glycerol ether metabolic process","positive regulation of mast cell activation","neuron intrinsic apoptotic signaling pathway in response to oxidative stress","circadian sleep/wake cycle","regulation of DNA methylation","ether lipid metabolic process","positive regulation of leukocyte activation","signal transduction involved in regulation of gene expression","sleep","mitochondrial DNA metabolic process","phosphatidylinositol-3-phosphate biosynthetic process","regulation of immune effector process","negative regulation of DNA-templated transcription, elongation","mitochondrial depolarization","positive regulation of cell activation","response to oxidative stress","negative regulation of keratinocyte proliferation","cellular response to zinc ion","regulation of ATP metabolic process","positive regulation of leukocyte degranulation","decidualization","regulation of protein-containing complex assembly","ether metabolic process","positive regulation of SMAD protein signal transduction","regulation of telomere capping","cellular response to gamma radiation","negative regulation of telomere maintenance via telomere lengthening","response to radiation","protein autoprocessing","gland development","response to X-ray","mitotic G1 DNA damage checkpoint signaling","mitotic G1/S transition checkpoint signaling","positive regulation of isotype switching","mitochondrial genome maintenance","response to iron ion","regulation of mast cell degranulation","positive regulation of signal transduction by p53 class mediator","muscle system process","mitochondrial fusion","leukocyte mediated immunity","DNA catabolic process","regulation of nucleotide biosynthetic process","regulation of mast cell activation involved in immune response","regulation of purine nucleotide biosynthetic process","establishment of organelle localization","negative regulation of B cell activation","positive regulation of protein localization","negative regulation of response to external stimulus","mononuclear cell differentiation","positive regulation of telomere maintenance via telomerase","cellular response to interleukin-4","positive regulation of cell adhesion","maternal placenta development","determination of adult lifespan","positive regulation of cardiac muscle hypertrophy","monocyte differentiation","regulation of intracellular estrogen receptor signaling pathway","regulation of necroptotic process","glandular epithelial cell development","positive regulation of muscle hypertrophy","response to interleukin-4","mitotic G2 DNA damage checkpoint signaling","response to copper ion","response to mineralocorticoid","positive regulation of telomere maintenance via telomere lengthening","positive regulation of double-strand break repair via homologous recombination","female meiotic nuclear division","regulation of DNA damage response, signal transduction by p53 class mediator","telomere capping","regulation of isotype switching","carbohydrate catabolic process","glycogen catabolic process","glucan catabolic process","polysaccharide catabolic process","glycogen metabolic process","glucan metabolic process","energy reserve metabolic process","polysaccharide metabolic process","pyruvate metabolic process","lactate metabolic process","energy derivation by oxidation of organic compounds","glycolytic process","skeletal myofibril assembly","elastic fiber assembly"],["5/14","3/14","3/14","3/14","3/14","3/14","3/14","3/14","4/14","2/14","3/14","4/14","2/14","2/14","2/14","2/14","2/14","2/14","2/14","2/14","3/14","2/14","3/14","2/14","4/7","4/7","2/7","3/7","2/7","3/7","2/7","3/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","2/7","2/7","1/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","2/7","2/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","2/7","2/7","2/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","5/9","3/9","3/9","3/9","3/9","3/9","3/9","3/9","3/9","2/9","3/9","2/9","1/9","1/9"],["163/18870","18/18870","19/18870","21/18870","76/18870","78/18870","89/18870","102/18870","315/18870","18/18870","118/18870","343/18870","22/18870","35/18870","46/18870","48/18870","49/18870","51/18870","60/18870","61/18870","278/18870","66/18870","298/18870","81/18870","235/18870","430/18870","35/18870","298/18870","51/18870","340/18870","61/18870","363/18870","65/18870","71/18870","72/18870","72/18870","78/18870","81/18870","93/18870","94/18870","104/18870","104/18870","106/18870","106/18870","130/18870","136/18870","138/18870","143/18870","146/18870","156/18870","175/18870","179/18870","181/18870","187/18870","196/18870","198/18870","199/18870","232/18870","238/18870","241/18870","242/18870","247/18870","249/18870","255/18870","10/18870","10/18870","10/18870","10/18870","10/18870","10/18870","10/18870","10/18870","265/18870","267/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","270/18870","278/18870","278/18870","12/18870","12/18870","12/18870","12/18870","12/18870","12/18870","12/18870","13/18870","295/18870","298/18870","300/18870","301/18870","14/18870","305/18870","307/18870","307/18870","308/18870","310/18870","314/18870","315/18870","15/18870","15/18870","15/18870","15/18870","15/18870","317/18870","319/18870","322/18870","16/18870","16/18870","16/18870","16/18870","334/18870","334/18870","17/18870","17/18870","17/18870","343/18870","18/18870","18/18870","18/18870","18/18870","350/18870","19/18870","19/18870","19/18870","19/18870","359/18870","20/18870","20/18870","20/18870","20/18870","20/18870","21/18870","21/18870","21/18870","21/18870","21/18870","21/18870","380/18870","22/18870","22/18870","22/18870","22/18870","389/18870","23/18870","23/18870","396/18870","400/18870","24/18870","24/18870","24/18870","25/18870","25/18870","419/18870","26/18870","26/18870","26/18870","27/18870","27/18870","428/18870","28/18870","443/18870","29/18870","29/18870","29/18870","29/18870","30/18870","30/18870","30/18870","30/18870","452/18870","31/18870","466/18870","32/18870","32/18870","32/18870","32/18870","469/18870","33/18870","476/18870","481/18870","481/18870","34/18870","34/18870","482/18870","35/18870","35/18870","35/18870","35/18870","35/18870","35/18870","36/18870","36/18870","36/18870","37/18870","37/18870","37/18870","37/18870","37/18870","38/18870","38/18870","39/18870","39/18870","163/18870","18/18870","19/18870","21/18870","76/18870","78/18870","89/18870","102/18870","118/18870","18/18870","337/18870","89/18870","10/18870","10/18870"],[8.499999999999999e-08,2.64e-07,3.13e-07,4.29e-07,2.21e-05,2.39e-05,3.56e-05,5.35e-05,6.68e-05,7.77e-05,8.25e-05,9.3e-05,0.000117,0.0003,0.000519,0.000565,0.000589,0.000638,0.000883,0.0009120000000000001,0.00102,0.00107,0.00125,0.0016,7.97e-07,8.81e-06,6.98e-05,0.00013,0.000149,0.000192,0.000214,0.000233,0.000243,0.00029,0.000298,0.000298,0.00035,0.000377,0.000497,0.000507,0.00062,0.00062,0.000644,0.000644,0.000967,0.00106,0.00109,0.00117,0.00122,0.00139,0.00174,0.00182,0.00186,0.00199,0.00218,0.00222,0.00224,0.00303,0.00319,0.00327,0.0033,0.00343,0.00349,0.00365,0.0037,0.0037,0.0037,0.0037,0.0037,0.0037,0.0037,0.0037,0.00394,0.004,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00409,0.00433,0.00433,0.00444,0.00444,0.00444,0.00444,0.00444,0.00444,0.00444,0.00481,0.00486,0.00495,0.00502,0.00505,0.00518,0.00518,0.00525,0.00525,0.00528,0.00535,0.00548,0.00552,0.00555,0.00555,0.00555,0.00555,0.00555,0.00559,0.00566,0.00576,0.00592,0.00592,0.00592,0.00592,0.00619,0.00619,0.00629,0.00629,0.00629,0.00651,0.00666,0.00666,0.00666,0.00666,0.00677,0.00703,0.00703,0.00703,0.00703,0.00712,0.0074,0.0074,0.0074,0.0074,0.0074,0.00777,0.00777,0.00777,0.00777,0.00777,0.00777,0.007939999999999999,0.00813,0.00813,0.00813,0.00813,0.00831,0.008500000000000001,0.008500000000000001,0.0086,0.00877,0.008869999999999999,0.008869999999999999,0.008869999999999999,0.00924,0.00924,0.00959,0.00961,0.00961,0.00961,0.00997,0.00997,0.009990000000000001,0.0103,0.0107,0.0107,0.0107,0.0107,0.0107,0.0111,0.0111,0.0111,0.0111,0.0111,0.0114,0.0118,0.0118,0.0118,0.0118,0.0118,0.0119,0.0122,0.0123,0.0125,0.0125,0.0125,0.0125,0.0126,0.0129,0.0129,0.0129,0.0129,0.0129,0.0129,0.0133,0.0133,0.0133,0.0136,0.0136,0.0136,0.0136,0.0136,0.014,0.014,0.0144,0.0144,5.54e-09,6.1e-08,7.24e-08,9.94e-08,5.18e-06,5.61e-06,8.35e-06,1.26e-05,1.95e-05,3.08e-05,0.000438,0.000775,0.00476,0.00476],[6.26e-05,7.669999999999999e-05,7.669999999999999e-05,7.889999999999999e-05,0.00294,0.00294,0.00374,0.00492,0.00546,0.00552,0.00552,0.0057,0.00663,0.0158,0.0255,0.0255,0.0255,0.0261,0.0336,0.0336,0.0357,0.0357,0.0399,0.0491,0.00052,0.00287,0.0152,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0175,0.0176,0.0207,0.0207,0.021,0.021,0.021,0.021,0.03,0.0309,0.0309,0.0317,0.0317,0.0348,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0402,0.0402,0.0402,0.0402,0.0402,0.0406,0.0406,0.0406,0.0406,0.0406,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0411,0.0411,0.0411,0.0411,0.0411,0.0417,0.042,0.042,0.0422,0.0422,0.0422,0.0422,0.0422,0.0433,0.0433,0.0438,0.0438,0.0438,0.0438,0.0446,0.0446,0.0446,0.0459,0.0459,0.0459,0.0459,0.0459,0.0459,0.0461,0.0461,0.0461,0.0461,0.0461,0.0472,0.0472,0.0472,0.0472,0.0472,0.0472,0.0474,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0481,0.0481,0.0481,0.0481,0.0481,0.0481,0.0481,0.0481,0.0489,0.0489,0.0496,0.0496,7.59e-07,3.31e-06,3.31e-06,3.4e-06,0.000128,0.000128,0.000163,0.000215,0.000297,0.000422,0.00545,0.00885,0.0466,0.0466],[3.57e-05,4.38e-05,4.38e-05,4.5e-05,0.00168,0.00168,0.00213,0.00281,0.00312,0.00315,0.00315,0.00325,0.00378,0.008999999999999999,0.0145,0.0146,0.0146,0.0149,0.0192,0.0192,0.0204,0.0204,0.0228,0.028,0.000188,0.00104,0.00548,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00634,0.00635,0.00748,0.00748,0.0076,0.0076,0.0076,0.0076,0.0109,0.0112,0.0112,0.0115,0.0115,0.0126,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0145,0.0145,0.0145,0.0145,0.0145,0.0147,0.0147,0.0147,0.0147,0.0147,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0149,0.0149,0.0149,0.0149,0.0149,0.0151,0.0152,0.0152,0.0153,0.0153,0.0153,0.0153,0.0153,0.0157,0.0157,0.0158,0.0158,0.0158,0.0158,0.0161,0.0161,0.0161,0.0166,0.0166,0.0166,0.0166,0.0166,0.0166,0.0167,0.0167,0.0167,0.0167,0.0167,0.0171,0.0171,0.0171,0.0171,0.0171,0.0171,0.0171,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0174,0.0174,0.0174,0.0174,0.0174,0.0174,0.0174,0.0174,0.0177,0.0177,0.0179,0.0179,4.61e-07,2.01e-06,2.01e-06,2.07e-06,7.77e-05,7.77e-05,9.92e-05,0.000131,0.00018,0.000256,0.00331,0.00537,0.0283,0.0283],["GAPDH;LDHA;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","ATM;CA9;EEF2K;GATA6","LDHA;LDHB","GAPDH;LDHA;LDHB","ATM;CA9;EEF2K;GATA6","ATM;INPP4B","ATM;PARP1","EEF2K;PARP1","EEF2K;PARP1","PARP1;PYGL","ATM;PARP1","PARP1;PYGL","ATM;PARP1","ATM;GAPDH;PARP1","ATM;INPP4B","ATM;CA9;GATA6","ATM;PARP1","FASN;GAB2;PARP1;TFRC","FASN;GAB2;PARP1;TFRC","ATM;PARP1","ATM;PARP1;TFRC","ATM;PARP1","ATM;PARP1;TFRC","ATM;PARP1","ATM;PARP1;TFRC","ATM;TFRC","PARP1;TFRC","ATM;TFRC","ATM;TFRC","ATM;TFRC","ATM;PARP1","PARP1;TFRC","ATM;PARP1","ATM;PARP1","ATM;TFRC","GAB2;TFRC","ATM;TFRC","ATM;TFRC","PARP1;TFRC","ATM;PARP1","PARP1;TFRC","ATM;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","PARP1;TFRC","ATM;TFRC","ATM;TFRC","ATM;PARP1","ATM;TFRC","ATM;GAB2","ATM;TFRC","ATM;PARP1","GAB2;TFRC","ATM;PARP1","MYH11","ATM","TFRC","ATM","MYH11","ATM","PARP1","PARP1","PARP1;TFRC","ATM;TFRC","FASN","FASN","PARP1","PARP1","FASN","PARP1","PARP1","ATM","ATM","ATM","FASN","PARP1","GAB2;TFRC","ATM;TFRC","ATM;PARP1","PARP1","ATM","TFRC","PARP1","ATM","FASN","FASN","ATM","ATM;TFRC","ATM;TFRC","PARP1;TFRC","GAB2;TFRC","EPPK1","GAB2;TFRC","ATM;PARP1","ATM;TFRC","ATM;TFRC","PARP1;TFRC","ATM;TFRC","ATM;TFRC","GAB2","PARP1","ATM","GAB2","PARP1","ATM;PARP1","ATM;PARP1","FASN;TFRC","ATM","PARP1","ATM","PARP1","ATM;PARP1","ATM;PARP1","ATM","ATM","ATM","ATM;TFRC","ATM","TFRC","TFRC","PARP1","ATM;TFRC","PARP1","PARP1","PARP1","PARP1","PARP1;TFRC","ATM","PARP1","EPPK1","PARP1","ATM","FASN","GAB2","PARP1","PARP1","PARP1","FASN","GAB2;TFRC","PARP1","PARP1","PARP1","ATM","GAB2;TFRC","PARP1","PARP1","GAB2;TFRC","ATM;PARP1","EPPK1","PARP1","PARP1","GAB2","PARP1","ATM;TFRC","FASN","PARP1","ATM","ATM","PARP1","ATM;PARP1","PARP1","ATM;FASN","ATM","ATM","ATM","TFRC","PARP1","TFRC","GAB2","ATM","MYH11;PARP1","TFRC","GAB2;TFRC","ATM","PARP1","GAB2","PARP1","ATM;GAB2","ATM","PARP1;TFRC","EPPK1;PARP1","ATM;FASN","ATM","FASN","ATM;TFRC","PARP1","ATM","PARP1","FASN","PARP1","PARP1","FASN","PARP1","FASN","ATM","TFRC","PARP1","ATM","PARP1","ATM","ATM","ATM","TFRC","GAPDH;LDHA;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","LDHA;LDHB","PYGB;PYGL;PYGM","GAPDH;LDHA","MYH11","MYH11"],[5,3,3,3,3,3,3,3,4,2,3,4,2,2,2,2,2,2,2,2,3,2,3,2,4,4,2,3,2,3,2,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,2,2,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,1,1,1,1,1,1,1,1,2,2,2,2,1,2,2,2,2,2,2,2,1,1,1,1,1,2,2,2,1,1,1,1,2,2,1,1,1,2,1,1,1,1,2,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,2,1,1,2,2,1,1,1,1,1,2,1,1,1,1,1,2,1,2,1,1,1,1,1,1,1,1,2,1,2,1,1,1,1,2,1,2,2,2,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,5,3,3,3,3,3,3,3,3,2,3,2,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":5,"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="go-cellular-compartments-terms" class="level4" data-number="3.4.2.2">
<h4 data-number="3.4.2.2" class="anchored" data-anchor-id="go-cellular-compartments-terms"><span class="header-section-number">3.4.2.2</span> GO Cellular Compartments terms</h4>
<p>Gene Ontology Cellular Compartment serves to capture the cellular location where a molecular function takes place.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-3" role="tab" aria-controls="tabset-3-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-25-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 14: dotplot of enriched GO Cellular Compartments terms.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-26-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 15: Term maps reporting relationships between GO Cellular Compartments across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-27-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 16: gene maps reporting relationships between GO Cellular Compartments enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>GO CC enrichment analyses do not provides any further insights on the characterization of kidney carcinomas based on their proteomics profiles.</p>
<p>The complete list of 23 GO CC terms enriched across the three contrasts is reported below.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-feda31ef9a0134320d5e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-feda31ef9a0134320d5e">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 6: enriched GO CC terms.<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23"],["KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["GO:0042470","GO:0048770","GO:0000781","GO:0016323","GO:0009925","GO:0045178","GO:0005859","GO:0016327","GO:1990391","GO:0098687","GO:0032982","GO:0016460","GO:0043596","GO:1990204","GO:0005859","GO:0045121","GO:0098857","GO:0034774","GO:0060205","GO:0031983","GO:0032982","GO:0016460","GO:0031528"],["melanosome","pigment granule","chromosome, telomeric region","basolateral plasma membrane","basal plasma membrane","basal part of cell","muscle myosin complex","apicolateral plasma membrane","DNA repair complex","chromosomal region","myosin filament","myosin II complex","nuclear replication fork","oxidoreductase complex","muscle myosin complex","membrane raft","membrane microdomain","secretory granule lumen","cytoplasmic vesicle lumen","vesicle lumen","myosin filament","myosin II complex","microvillus membrane"],["3/7","3/7","2/7","2/7","2/7","2/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","2/9","1/9","2/9","2/9","2/9","2/9","2/9","1/9","1/9","1/9"],["112/19886","112/19886","171/19886","249/19886","282/19886","301/19886","15/19886","22/19886","22/19886","399/19886","24/19886","26/19886","34/19886","128/19886","15/19886","286/19886","287/19886","322/19886","325/19886","326/19886","24/19886","26/19886","33/19886"],[5.99e-06,5.99e-06,0.0015,0.00315,0.00401,0.00456,0.00527,0.00772,0.00772,0.007889999999999999,0.00842,0.00912,0.0119,0.00144,0.00677,0.00694,0.00699,0.00873,0.00889,0.00894,0.0108,0.0117,0.0148],[0.000147,0.000147,0.0245,0.0369,0.0369,0.0369,0.0369,0.0372,0.0372,0.0372,0.0372,0.0372,0.0449,0.0383,0.0383,0.0383,0.0383,0.0383,0.0383,0.0383,0.039,0.039,0.0445],[5.99e-05,5.99e-05,0.01,0.0151,0.0151,0.0151,0.0151,0.0152,0.0152,0.0152,0.0152,0.0152,0.0183,0.0188,0.0188,0.0188,0.0188,0.0188,0.0188,0.0188,0.0192,0.0192,0.0219],["FASN;MYH11;TFRC","FASN;MYH11;TFRC","ATM;PARP1","EPPK1;TFRC","EPPK1;TFRC","EPPK1;TFRC","MYH11","EPPK1","ATM","ATM;PARP1","MYH11","MYH11","PARP1","LDHA;LDHB","MYH11","GAB2;LDHB","GAB2;LDHB","PYGB;PYGL","PYGB;PYGL","PYGB;PYGL","MYH11","MYH11","CA9"],[3,3,2,2,2,2,1,1,1,2,1,1,1,2,1,2,2,2,2,2,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":5,"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="go-molecular-function-terms" class="level4" data-number="3.4.2.3">
<h4 data-number="3.4.2.3" class="anchored" data-anchor-id="go-molecular-function-terms"><span class="header-section-number">3.4.2.3</span> GO Molecular Function terms</h4>
<p>Gene Ontology Molecular Function serves to represent molecular-level activities performed by gene products.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-4-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-1" role="tab" aria-controls="tabset-4-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-2" role="tab" aria-controls="tabset-4-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-3" role="tab" aria-controls="tabset-4-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-4-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-4-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-29-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 17: dotplot of enriched GO Molecular Functions terms.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-30-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 18: Term maps reporting relationships between GO Molecular Functions across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-31-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 19: gene maps reporting relationships between GO Molecular Functions enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>Enrichment of GO MF terms confirm the catabolic signature of KIRC when compared to KIRP and KICH, with enriched terms such as <em>“glycosiltransferase activity”</em> and <em>“pyridoxal phosphate binding”</em> driven by the over abundance of <em>PYGB</em>, <em>PYGL</em> and <em>PYGM</em>.</p>
<p>The complete list of 35 GO MF terms enriched across the three contrasts is reported below.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-782abf12ed2064fa07ad" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-782abf12ed2064fa07ad">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 7: enriched GO MF terms.<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["GO:0051287","GO:0030170","GO:0070279","GO:0016757","GO:0019842","GO:0016758","GO:0016616","GO:0016614","GO:0016303","GO:0016884","GO:0032052","GO:0005536","GO:0019215","GO:0046030","GO:0004089","GO:0016208","GO:0030170","GO:0070279","GO:0019842","GO:0016758","GO:0016757","GO:0051287","GO:0016616","GO:0016614","GO:0032052","GO:0005536","GO:0004089","GO:0016208","GO:0005068","GO:0016769","GO:0043325","GO:0097718","GO:0000146","GO:0005547","GO:0016620"],["NAD binding","pyridoxal phosphate binding","vitamin B6 binding","glycosyltransferase activity","vitamin binding","hexosyltransferase activity","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","oxidoreductase activity, acting on CH-OH group of donors","1-phosphatidylinositol-3-kinase activity","carbon-nitrogen ligase activity, with glutamine as amido-N-donor","bile acid binding","glucose binding","intermediate filament binding","inositol trisphosphate phosphatase activity","carbonate dehydratase activity","AMP binding","pyridoxal phosphate binding","vitamin B6 binding","vitamin binding","hexosyltransferase activity","glycosyltransferase activity","NAD binding","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","oxidoreductase activity, acting on CH-OH group of donors","bile acid binding","glucose binding","carbonate dehydratase activity","AMP binding","transmembrane receptor protein tyrosine kinase adaptor activity","transferase activity, transferring nitrogenous groups","phosphatidylinositol-3,4-bisphosphate binding","disordered domain specific binding","microfilament motor activity","phosphatidylinositol-3,4,5-trisphosphate binding","oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor"],["3/14","3/14","3/14","4/14","3/14","3/14","2/14","2/14","1/14","1/14","1/14","1/14","1/14","1/14","1/14","1/14","3/9","3/9","3/9","3/9","3/9","2/9","2/9","2/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9"],["53/18496","56/18496","57/18496","268/18496","150/18496","196/18496","125/18496","135/18496","10/18496","10/18496","10/18496","11/18496","11/18496","12/18496","15/18496","15/18496","56/18496","57/18496","150/18496","196/18496","268/18496","53/18496","125/18496","135/18496","10/18496","11/18496","15/18496","15/18496","17/18496","26/18496","29/18496","32/18496","37/18496","39/18496","39/18496"],[7.91e-06,9.35e-06,9.86e-06,3.85e-05,0.000178,0.000391,0.00391,0.00454,0.00755,0.00755,0.00755,0.0083,0.0083,0.009050000000000001,0.0113,0.0113,2.18e-06,2.3e-06,4.24e-05,9.390000000000001e-05,0.000237,0.000286,0.00158,0.00184,0.00486,0.00534,0.00728,0.00728,0.008240000000000001,0.0126,0.014,0.0155,0.0179,0.0188,0.0188],[0.00023,0.00023,0.00023,0.000673,0.0025,0.00457,0.0391,0.0398,0.0447,0.0447,0.0447,0.0447,0.0447,0.0452,0.0494,0.0494,5.75e-05,5.75e-05,0.000706,0.00117,0.00237,0.00239,0.0113,0.0115,0.0267,0.0267,0.0303,0.0303,0.0317,0.0449,0.0468,0.0483,0.0495,0.0495,0.0495],[0.000111,0.000111,0.000111,0.000324,0.0012,0.0022,0.0188,0.0191,0.0215,0.0215,0.0215,0.0215,0.0215,0.0218,0.0238,0.0238,2.54e-05,2.54e-05,0.000312,0.000519,0.00105,0.00105,0.00499,0.00509,0.0118,0.0118,0.0134,0.0134,0.014,0.0199,0.0207,0.0214,0.0219,0.0219,0.0219],["GAPDH;LDHB;PARP1","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PARP1;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","ATM","ASNS","PYGL","PYGL","EPPK1","INPP4B","CA9","PYGL","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;LDHB","LDHA;LDHB","LDHA;LDHB","PYGL","PYGL","CA9","PYGL","GAB2","GAPDH","GAB2","GAPDH","MYH11","GAB2","GAPDH"],[3,3,3,4,3,3,2,2,1,1,1,1,1,1,1,1,3,3,3,3,3,2,2,2,1,1,1,1,1,1,1,1,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":5,"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="kegg-pathways" class="level4" data-number="3.4.2.4">
<h4 data-number="3.4.2.4" class="anchored" data-anchor-id="kegg-pathways"><span class="header-section-number">3.4.2.4</span> KEGG pathways</h4>
<p>KEGG pathways are manually curated and capture our knowledge of the molecular interaction, reaction and relation networks.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-5-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-1" role="tab" aria-controls="tabset-5-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-2" role="tab" aria-controls="tabset-5-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-3" role="tab" aria-controls="tabset-5-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-5-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-5-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-33-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 20: dotplot of enriched KEGG pathways.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-5-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-34-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 21: Term maps reporting relationships between KEGG pathways enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-5-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-35-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 22: gene maps reporting relationships between KEGG pathways enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>Also the enriched KEGG pathways revolved around the metabolic disregulation of KIRC (e.g.: <em>“Glycolysis / Gluconeogenesis”</em> and <em>“Starch and sucrose metabolism”</em>) when compared to KIRP and KICH, linked as well to <em>“Necroptosis”</em> and <em>“HIF-1 signaling pathway”</em>.</p>
<p>The complete list of 30 KEGG pathways across the three contrasts is reported below.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-d3752a3b70ef733f7a0a" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d3752a3b70ef733f7a0a">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 8: enriched KEGG pathways.<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["Organismal Systems","Metabolism","Cellular Processes","Metabolism","Human Diseases","Environmental Information Processing","Organismal Systems","Metabolism","Metabolism","Metabolism","Human Diseases","Metabolism","Environmental Information Processing","Cellular Processes",null,"Environmental Information Processing","Organismal Systems","Metabolism","Metabolism","Human Diseases","Environmental Information Processing","Organismal Systems","Metabolism","Cellular Processes","Metabolism","Metabolism","Human Diseases","Metabolism",null,"Human Diseases"],["Endocrine system","Carbohydrate metabolism","Cell growth and death","Carbohydrate metabolism","Endocrine and metabolic disease","Signal transduction","Endocrine system","Carbohydrate metabolism","Carbohydrate metabolism","Amino acid metabolism","Cancer: overview","Global and overview maps","Signal transduction","Cell growth and death",null,"Signal transduction","Endocrine system","Carbohydrate metabolism","Carbohydrate metabolism","Endocrine and metabolic disease","Signal transduction","Endocrine system","Carbohydrate metabolism","Cell growth and death","Carbohydrate metabolism","Amino acid metabolism","Cancer: overview","Energy metabolism",null,"Infectious disease: bacterial"],["hsa04922","hsa00500","hsa04217","hsa00010","hsa04931","hsa04066","hsa04910","hsa00640","hsa00620","hsa00270","hsa05230","hsa01230","hsa04064","hsa04210","hsa03273","hsa04064","hsa04922","hsa00500","hsa00010","hsa04931","hsa04066","hsa04910","hsa00640","hsa04217","hsa00620","hsa00270","hsa05230","hsa00910","hsa03273","hsa05130"],["Glucagon signaling pathway","Starch and sucrose metabolism","Necroptosis","Glycolysis / Gluconeogenesis","Insulin resistance","HIF-1 signaling pathway","Insulin signaling pathway","Propanoate metabolism","Pyruvate metabolism","Cysteine and methionine metabolism","Central carbon metabolism in cancer","Biosynthesis of amino acids","NF-kappa B signaling pathway","Apoptosis","Virion - Lassa virus and SFTS virus","NF-kappa B signaling pathway","Glucagon signaling pathway","Starch and sucrose metabolism","Glycolysis / Gluconeogenesis","Insulin resistance","HIF-1 signaling pathway","Insulin signaling pathway","Propanoate metabolism","Necroptosis","Pyruvate metabolism","Cysteine and methionine metabolism","Central carbon metabolism in cancer","Nitrogen metabolism","Virion - Lassa virus and SFTS virus","Pathogenic Escherichia coli infection"],["5/12","3/12","4/12","3/12","3/12","3/12","3/12","2/12","2/12","2/12","2/12","2/12","2/12","2/12","2/6","2/6","5/9","3/9","3/9","3/9","3/9","3/9","2/9","3/9","2/9","2/9","2/9","1/9","1/9","2/9"],["107/8543","40/8543","159/8543","67/8543","109/8543","110/8543","138/8543","32/8543","47/8543","52/8543","71/8543","75/8543","105/8543","137/8543","17/8543","105/8543","107/8543","40/8543","67/8543","109/8543","110/8543","138/8543","32/8543","159/8543","47/8543","52/8543","71/8543","17/8543","17/8543","203/8543"],[2.07e-07,2.03e-05,5.09e-05,9.64e-05,0.000409,0.00042,0.000815,0.000876,0.00189,0.00231,0.00426,0.00474,0.00911,0.0152,5.56e-05,0.00217,3.4e-08,7.83e-06,3.75e-05,0.000161,0.000165,0.000323,0.000481,0.000489,0.00104,0.00127,0.00236,0.0178,0.0178,0.0181],[7.880000000000001e-06,0.000386,0.000645,0.000916,0.00266,0.00266,0.00416,0.00416,0.00797,0.00877,0.0147,0.015,0.0266,0.0412,0.00234,0.0456,1.05e-06,0.000121,0.000387,0.00102,0.00102,0.00167,0.0019,0.0019,0.00358,0.00395,0.00665,0.0401,0.0401,0.0401],[4.8e-06,0.000235,0.000393,0.000558,0.00162,0.00162,0.00254,0.00254,0.00486,0.00534,0.008970000000000001,0.00915,0.0162,0.0251,0.00193,0.0377,6.08e-07,7.01e-05,0.000223,0.00059,0.00059,0.000962,0.00109,0.00109,0.00207,0.00228,0.00384,0.0232,0.0232,0.0232],["LDHA;LDHB;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PARP1;PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","LDHA;LDHB","LDHA;LDHB","ASNS;GAPDH","ATM;PARP1","ATM;PARP1","MYH11;TFRC","ATM;PARP1","LDHA;LDHB;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","LDHA;LDHB","PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","LDHA;LDHB","CA9","MYH11","GAPDH;MYH11"],[5,3,4,3,3,3,3,2,2,2,2,2,2,2,2,2,5,3,3,3,3,3,2,3,2,2,2,1,1,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>category<\/th>\n      <th>subcategory<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[8,9,10,12]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"category","targets":2},{"name":"subcategory","targets":3},{"name":"ID","targets":4},{"name":"Description","targets":5},{"name":"GeneRatio","targets":6},{"name":"BgRatio","targets":7},{"name":"pvalue","targets":8},{"name":"p.adjust","targets":9},{"name":"qvalue","targets":10},{"name":"geneID","targets":11},{"name":"Count","targets":12}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":5,"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="reactome-pathways" class="level4" data-number="3.4.2.5">
<h4 data-number="3.4.2.5" class="anchored" data-anchor-id="reactome-pathways"><span class="header-section-number">3.4.2.5</span> Reactome pathways</h4>
<p>Similarly to KEGG, Reactome pathways are manually curated biological pathways.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-6-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-1" role="tab" aria-controls="tabset-6-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-6-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-6-2" role="tab" aria-controls="tabset-6-2" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-6-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-6-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-37-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 23: dotplot of enriched Reactome pathways.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-6-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-6-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-38-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 24: gene maps reporting relationships between Reactome pathways enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>Again, enriched Reactome pathways characterized KIRC <em>“Glycogen metabolism”</em>, <em>“Metabolosim of charbohydrates”</em> and <em>“Pyruvate metabolism”</em> when compared to KIRP and KICH kidney carcinomas.</p>
<p>The complete list of 10 Reactome pathways enriched across the three contrasts is reported below.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-9042846810e6fee9eb46" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-9042846810e6fee9eb46">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 9: enriched Reactome pathways.<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["R-HSA-70221","R-HSA-8982491","R-HSA-71387","R-HSA-70268","R-HSA-71406","R-HSA-70221","R-HSA-8982491","R-HSA-71387","R-HSA-70268","R-HSA-71406"],["Glycogen breakdown (glycogenolysis)","Glycogen metabolism","Metabolism of carbohydrates","Pyruvate metabolism","Pyruvate metabolism and Citric Acid (TCA) cycle","Glycogen breakdown (glycogenolysis)","Glycogen metabolism","Metabolism of carbohydrates","Pyruvate metabolism","Pyruvate metabolism and Citric Acid (TCA) cycle"],["3/13","3/13","4/13","2/13","2/13","3/9","3/9","4/9","2/9","2/9"],["16/11009","27/11009","299/11009","31/11009","55/11009","16/11009","27/11009","299/11009","31/11009","55/11009"],[7.14e-07,3.7e-06,0.000314,0.000587,0.00185,2.1e-07,1.09e-06,6.03e-05,0.000273,0.0008630000000000001],[7.499999999999999e-05,0.000194,0.011,0.0154,0.0388,1.24e-05,3.23e-05,0.00119,0.00403,0.0102],[4.21e-05,0.000109,0.00617,0.00865,0.0218,4.43e-06,1.15e-05,0.000423,0.00144,0.00363],["PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB"],[3,3,4,2,2,3,3,4,2,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":5,"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="wikipathways" class="level4" data-number="3.4.2.6">
<h4 data-number="3.4.2.6" class="anchored" data-anchor-id="wikipathways"><span class="header-section-number">3.4.2.6</span> WikiPathways</h4>
<p>Lastly, also WikiPathways provides another angle on a set of manually curated biological pathways.</p>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-7-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-7-1" role="tab" aria-controls="tabset-7-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-7-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-7-2" role="tab" aria-controls="tabset-7-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-7-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-7-3" role="tab" aria-controls="tabset-7-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-7-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-7-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-40-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption>Figure 25: dotplot of enriched Wikipathways pathways.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-7-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-7-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-41-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 26: Term maps reporting relationships between WikiPathways enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-7-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-7-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-42-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 27: gene maps reporting relationships between WikiPathways enriched across the different contrasts.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<p>The enriched WikiPathways confirmed the <em>“Aerobiuc glycolysis - augmented”</em> in KIRC, when compared to KIRP and KIRCH, as well as a proper <em>“Clear cell renal cell carcinoma pathwyas”</em>, driven by the abundance of <em>GAPDH</em>, <em>LDHA</em> and <em>LDHB</em>.</p>
<p>KICH retention of differentiation was supported by the abundance of the ATM Serine/Threonine Kinase <em>ATM</em> and <em>PARP1</em>, which resulted in the enrichment of terms such as <em>“DNA IR-double strand breaks and cellular response via ATM”</em>.</p>
<p>The complete list of 53 WikiPathways pathways enriched across the three contrasts is reported below.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-855ade5148fb0b9e77d2" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-855ade5148fb0b9e77d2">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 10: enriched WikiPathways.<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["WP500","WP534","WP5049","WP4236","WP4628","WP5314","WP4018","WP1946","WP5355","WP5215","WP4290","WP3959","WP4656","WP4016","WP3925","WP3959","WP4656","WP4016","WP4874","WP4946","WP4673","WP4262","WP2874","WP3630","WP5118","WP2007","WP2513","WP186","WP4724","WP3644","WP5172","WP2011","WP1742","WP3640","WP3657","WP5066","WP357","WP3645","WP5103","WP5046","WP500","WP534","WP5049","WP4018","WP4236","WP4628","WP5314","WP5355","WP1946","WP4290","WP3888","WP5166","WP5173"],["Glycogen synthesis and degradation","Glycolysis and gluconeogenesis","Glycolysis in senescence","Krebs cycle disorders","Aerobic glycolysis - augmented","Lactate shuttle in glial cells","Clear cell renal cell carcinoma pathways","Cori cycle","Metabolic epileptic disorders","Targeted agents in triple negative breast cancer","Metabolic reprogramming in colon cancer","DNA IR-double strand breaks and cellular response via ATM","Joubert syndrome","DNA IR-damage and cellular response via ATR","Amino acid metabolism","DNA IR-double strand breaks and cellular response via ATM","Joubert syndrome","DNA IR-damage and cellular response via ATR","CAMKK2 pathway","DNA repair pathways, full network","Male infertility","Breast cancer pathway","Liver X receptor pathway","NAD metabolism, sirtuins and aging","SMC1/SMC3 role in DNA damage - Cornelia de Lange Syndrome","Iron metabolism in placenta","Nanoparticle triggered regulated necrosis","Homologous recombination","Omega-9 fatty acid synthesis","NAD+ metabolism","Iron metabolism disorders","SREBF and miR33 in cholesterol and lipid homeostasis","TP53 network","Imatinib and chronic myeloid leukemia","Hematopoietic stem cell gene regulation by GABP alpha/beta complex","FOXA2 pathway","Fatty acid biosynthesis","NAD+ biosynthetic pathways","Progeria-associated lipodystrophy","NAD metabolism in oncogene-induced senescence and mitochondrial dysfunction-associated senescence","Glycogen synthesis and degradation","Glycolysis and gluconeogenesis","Glycolysis in senescence","Clear cell renal cell carcinoma pathways","Krebs cycle disorders","Aerobic glycolysis - augmented","Lactate shuttle in glial cells","Metabolic epileptic disorders","Cori cycle","Metabolic reprogramming in colon cancer","VEGFA-VEGFR2 signaling","Glyoxylate metabolism","Disorders of galactose metabolism "],["3/13","3/13","2/13","2/13","2/13","2/13","3/13","2/13","3/13","2/13","2/13","2/13","2/13","2/13","2/13","2/6","2/6","2/6","2/6","2/6","2/6","2/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","1/6","3/9","3/9","2/9","3/9","2/9","2/9","2/9","3/9","2/9","2/9","3/9","1/9","1/9"],["40/9133","45/9133","11/9133","13/9133","13/9133","13/9133","86/9133","17/9133","91/9133","38/9133","42/9133","55/9133","76/9133","81/9133","91/9133","55/9133","76/9133","81/9133","110/9133","120/9133","145/9133","155/9133","10/9133","11/9133","11/9133","12/9133","12/9133","13/9133","14/9133","16/9133","17/9133","18/9133","19/9133","20/9133","21/9133","21/9133","22/9133","22/9133","22/9133","23/9133","40/9133","45/9133","11/9133","86/9133","13/9133","13/9133","13/9133","91/9133","17/9133","42/9133","432/9133","11/9133","13/9133"],[2.16e-05,3.09e-05,0.000102,0.000145,0.000145,0.000145,0.000215,0.000251,0.000255,0.00128,0.00156,0.00266,0.00502,0.00569,0.00713,0.000526,0.001,0.00114,0.00209,0.00248,0.0036,0.0041,0.00655,0.00721,0.00721,0.007860000000000001,0.007860000000000001,0.00851,0.00916,0.0105,0.0111,0.0118,0.0124,0.0131,0.0137,0.0137,0.0144,0.0144,0.0144,0.015,6.42e-06,9.2e-06,4.73e-05,6.499999999999999e-05,6.7e-05,6.7e-05,6.7e-05,7.7e-05,0.000117,0.000728,0.00713,0.0108,0.0127],[0.00133,0.00133,0.00207,0.00207,0.00207,0.00207,0.00243,0.00243,0.00243,0.011,0.0122,0.0191,0.0332,0.0349,0.0409,0.03,0.03,0.03,0.0392,0.0392,0.0463,0.0463,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0473,0.0475,0.000225,0.000225,0.000469,0.000469,0.000469,0.000469,0.000469,0.000471,0.000634,0.00357,0.0318,0.0441,0.048],[0.0006669999999999999,0.0006669999999999999,0.00104,0.00104,0.00104,0.00104,0.00122,0.00122,0.00122,0.00551,0.00612,0.00957,0.0167,0.0175,0.0205,0.00839,0.00839,0.00839,0.011,0.011,0.013,0.013,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0132,0.0133,0.000102,0.000102,0.000211,0.000211,0.000211,0.000211,0.000211,0.000213,0.000286,0.00161,0.0143,0.0199,0.0217],["PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","GAPDH;LDHA","LDHA;LDHB","GAPDH;LDHA","LDHA;LDHB","GAPDH;LDHA;LDHB","GAPDH;LDHA","GAPDH;LDHA;LDHB","INPP4B;PARP1","GAPDH;LDHA","ATM;PARP1","ATM;PARP1","ATM;PARP1","ASNS;LDHA","ATM;PARP1","ATM;PARP1","ATM;PARP1","FASN;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","FASN","PARP1","ATM","TFRC","PARP1","ATM","FASN","PARP1","TFRC","FASN","ATM","GAB2","ATM","FASN","FASN","PARP1","PARP1","PARP1","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","GAPDH;LDHA","GAPDH;LDHA;LDHB","LDHA;LDHB","GAPDH;LDHA","LDHA;LDHB","GAPDH;LDHA;LDHB","GAPDH;LDHA","GAPDH;LDHA","GAPDH;LDHA;MYH11","LDHA","PYGL"],[3,3,2,2,2,2,3,2,3,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,3,3,2,3,2,2,2,3,2,2,3,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":5,"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
</section>
<section id="sec-lesson-learnt-proteomics" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="sec-lesson-learnt-proteomics"><span class="header-section-number">3.5</span> Lessons Learnts</h2>
<p>Based on proteomics data, we have learnt:</p>
<ul>
<li>Exploratory analysis suggests:
<ul>
<li>KICH clusters with KIRP based on proteomics data.</li>
<li>There is a good correlation between the differences in protein abundances across the kidney carcinomas that correlate with a poorer prognosis, and this signal does not correlate with the kidney carcinoma type (<em>cancer type</em>), nor with the gene expression (<em>subtype mRNA</em> and <em>subtype miRNA</em>).</li>
</ul></li>
<li>Differential Gene Expression analysis:
<ul>
<li>KIRC has a strong signature of 19 proteins.</li>
<li>KIRC has upregulated glycolysis, glycogen and hypoxia response pathways.</li>
</ul></li>
</ul>
</section>
<section id="session-information" class="level2" data-number="3.6">
<h2 data-number="3.6" class="anchored" data-anchor-id="session-information"><span class="header-section-number">3.6</span> Session Information</h2>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: openSUSE Tumbleweed

Matrix products: default
BLAS/LAPACK: /home/andrea/miniforge3/envs/moai/lib/libmkl_rt.so.2;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=it_IT.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Brussels
tzcode source: system (glibc)

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] UpSetR_1.4.0           umap_0.2.10.0          stringr_1.5.1         
 [4] scales_1.3.0           RColorBrewer_1.1-3     PCAtools_2.14.0       
 [7] matrixStats_1.5.0      org.Hs.eg.db_3.18.0    gridExtra_2.3         
[10] forcats_1.0.0          EnhancedVolcano_1.20.0 ggrepel_0.9.6         
[13] ggplot2_3.5.1          edgeR_4.0.16           limma_3.58.1          
[16] DT_0.33                dplyr_1.1.4            DOSE_3.28.2           
[19] data.table_1.16.4      cowplot_1.1.3          clusterProfiler_4.10.1
[22] BiocSingular_1.18.0    BiocParallel_1.36.0    AnnotationDbi_1.64.1  
[25] IRanges_2.36.0         S4Vectors_0.40.2       Biobase_2.62.0        
[28] BiocGenerics_0.48.1   

loaded via a namespace (and not attached):
  [1] jsonlite_1.8.9            magrittr_2.0.3           
  [3] farver_2.1.2              rmarkdown_2.29           
  [5] fs_1.6.5                  zlibbioc_1.48.2          
  [7] vctrs_0.6.5               DelayedMatrixStats_1.24.0
  [9] memoise_2.0.1             RCurl_1.98-1.16          
 [11] askpass_1.2.1             ggtree_3.10.1            
 [13] htmltools_0.5.8.1         S4Arrays_1.2.1           
 [15] SparseArray_1.2.4         gridGraphics_0.5-1       
 [17] sass_0.4.9                bslib_0.8.0              
 [19] htmlwidgets_1.6.4         plyr_1.8.9               
 [21] cachem_1.1.0              igraph_2.1.4             
 [23] lifecycle_1.0.4           pkgconfig_2.0.3          
 [25] rsvd_1.0.5                Matrix_1.6-5             
 [27] R6_2.5.1                  fastmap_1.2.0            
 [29] gson_0.1.0                GenomeInfoDbData_1.2.11  
 [31] MatrixGenerics_1.14.0     digest_0.6.37            
 [33] aplot_0.2.4               enrichplot_1.22.0        
 [35] ggnewscale_0.5.0          colorspace_2.1-1         
 [37] patchwork_1.3.0           RSpectra_0.16-2          
 [39] dqrng_0.4.1               irlba_2.3.5.1            
 [41] crosstalk_1.2.1           RSQLite_2.3.9            
 [43] beachmat_2.18.1           labeling_0.4.3           
 [45] httr_1.4.7                polyclip_1.10-7          
 [47] abind_1.4-8               compiler_4.3.2           
 [49] bit64_4.6.0-1             withr_3.0.2              
 [51] graphite_1.48.0           viridis_0.6.5            
 [53] DBI_1.2.3                 ggforce_0.4.2            
 [55] MASS_7.3-60.0.1           openssl_2.3.1            
 [57] rappdirs_0.3.3            DelayedArray_0.28.0      
 [59] HDO.db_0.99.1             tools_4.3.2              
 [61] ape_5.8-1                 scatterpie_0.2.4         
 [63] glue_1.8.0                nlme_3.1-167             
 [65] GOSemSim_2.28.1           shadowtext_0.1.4         
 [67] reshape2_1.4.4            fgsea_1.28.0             
 [69] generics_0.1.3            gtable_0.3.6             
 [71] tidyr_1.3.1               tidygraph_1.3.1          
 [73] ScaledMatrix_1.10.0       XVector_0.42.0           
 [75] pillar_1.10.1             yulab.utils_0.2.0        
 [77] splines_4.3.2             tweenr_2.0.3             
 [79] treeio_1.26.0             lattice_0.22-6           
 [81] bit_4.5.0.1               tidyselect_1.2.1         
 [83] GO.db_3.18.0              locfit_1.5-9.10          
 [85] Biostrings_2.70.3         reactome.db_1.86.2       
 [87] knitr_1.49                xfun_0.50                
 [89] graphlayouts_1.2.2        statmod_1.5.0            
 [91] stringi_1.8.4             lazyeval_0.2.2           
 [93] ggfun_0.1.8               yaml_2.3.10              
 [95] ReactomePA_1.46.0         evaluate_1.0.3           
 [97] codetools_0.2-20          ggraph_2.2.1             
 [99] tibble_3.2.1              qvalue_2.34.0            
[101] graph_1.80.0              BiocManager_1.30.25      
[103] ggplotify_0.1.2           cli_3.6.3                
[105] reticulate_1.40.0         jquerylib_0.1.4          
[107] munsell_0.5.1             Rcpp_1.0.14              
[109] GenomeInfoDb_1.38.8       png_0.1-8                
[111] parallel_4.3.2            blob_1.2.4               
[113] sparseMatrixStats_1.14.0  bitops_1.0-9             
[115] viridisLite_0.4.2         tidytree_0.4.6           
[117] purrr_1.0.2               crayon_1.5.3             
[119] rlang_1.1.5               fastmatch_1.1-6          
[121] KEGGREST_1.42.0          </code></pre>
</div>
</div>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./transcriptomics-analysis.html" class="pagination-link" aria-label="Transcriptomics Analysis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./micro-RNAs-analysis.html" class="pagination-link" aria-label="micro-RNAs Analysis">
        <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>