<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>3&nbsp; Proteomics Analysis – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./micro-RNAs-analysis.html" rel="next">
<link href="./transcriptomics-analysis.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-fixedcolumns-1.13.6/css/fixedColumns.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedcolumns-1.13.6/js/dataTables.fixedColumns.min.js"></script>
<link href="site_libs/dt-ext-fixedheader-1.13.6/css/fixedHeader.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedheader-1.13.6/js/dataTables.fixedHeader.min.js"></script>
<script src="site_libs/dt-plugin-ellipsis-1.13.6/source.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./proteomics-analysis.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./surivival-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">surivival-analysis.html</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell Cancer molecular subtypes</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cancer-signatures.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">cancer-signatures.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./autoencoder.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">autoencoder.html</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./cellular-deconvolution.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Cellular Deconvolution from Bulk RNA-seq</span></span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active" data-toc-expanded="99">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">3.1</span> On this page</a></li>
  <li><a href="#proteomics-data-overview-and-qc" id="toc-proteomics-data-overview-and-qc" class="nav-link" data-scroll-target="#proteomics-data-overview-and-qc"><span class="header-section-number">3.2</span> Proteomics data overview and QC</a>
  <ul>
  <li><a href="#spectral-counts-overview" id="toc-spectral-counts-overview" class="nav-link" data-scroll-target="#spectral-counts-overview"><span class="header-section-number">3.2.1</span> Spectral counts overview</a></li>
  <li><a href="#data-imputation" id="toc-data-imputation" class="nav-link" data-scroll-target="#data-imputation"><span class="header-section-number">3.2.2</span> Data imputation</a></li>
  </ul></li>
  <li><a href="#dimesionality-reduction-and-dataset-exploration" id="toc-dimesionality-reduction-and-dataset-exploration" class="nav-link" data-scroll-target="#dimesionality-reduction-and-dataset-exploration"><span class="header-section-number">3.3</span> Dimesionality Reduction and Dataset Exploration</a>
  <ul>
  <li><a href="#umap-on-filtered-transcriptomics-data" id="toc-umap-on-filtered-transcriptomics-data" class="nav-link" data-scroll-target="#umap-on-filtered-transcriptomics-data"><span class="header-section-number">3.3.1</span> UMAP on filtered transcriptomics data</a></li>
  <li><a href="#principal-component-analysis-pca" id="toc-principal-component-analysis-pca" class="nav-link" data-scroll-target="#principal-component-analysis-pca"><span class="header-section-number">3.3.2</span> Principal Component Analysis (PCA)</a></li>
  </ul></li>
  <li><a href="#differential-protein-abundance-analysis" id="toc-differential-protein-abundance-analysis" class="nav-link" data-scroll-target="#differential-protein-abundance-analysis"><span class="header-section-number">3.4</span> Differential protein abundance analysis</a>
  <ul>
  <li><a href="#identification-of-differentially-expressed-proteins" id="toc-identification-of-differentially-expressed-proteins" class="nav-link" data-scroll-target="#identification-of-differentially-expressed-proteins"><span class="header-section-number">3.4.1</span> Identification of differentially expressed proteins</a></li>
  <li><a href="#comparative-enrichment-analyses-across-cancer-types" id="toc-comparative-enrichment-analyses-across-cancer-types" class="nav-link" data-scroll-target="#comparative-enrichment-analyses-across-cancer-types"><span class="header-section-number">3.4.2</span> Comparative enrichment analyses across cancer types</a>
  <ul>
  <li><a href="#go-biological-process-terms" id="toc-go-biological-process-terms" class="nav-link" data-scroll-target="#go-biological-process-terms"><span class="header-section-number">3.4.2.1</span> GO Biological Process terms</a></li>
  <li><a href="#go-cellular-compartments-terms" id="toc-go-cellular-compartments-terms" class="nav-link" data-scroll-target="#go-cellular-compartments-terms"><span class="header-section-number">3.4.2.2</span> GO Cellular Compartments terms</a></li>
  <li><a href="#go-molecular-function-terms" id="toc-go-molecular-function-terms" class="nav-link" data-scroll-target="#go-molecular-function-terms"><span class="header-section-number">3.4.2.3</span> GO Molecular Function terms</a></li>
  <li><a href="#kegg-pathways" id="toc-kegg-pathways" class="nav-link" data-scroll-target="#kegg-pathways"><span class="header-section-number">3.4.2.4</span> KEGG pathways</a></li>
  <li><a href="#reactome-pathways" id="toc-reactome-pathways" class="nav-link" data-scroll-target="#reactome-pathways"><span class="header-section-number">3.4.2.5</span> Reactome pathways</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-proteomics" id="toc-sec-lesson-learnt-proteomics" class="nav-link" data-scroll-target="#sec-lesson-learnt-proteomics"><span class="header-section-number">3.5</span> Lessons Learnts</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./proteomics-analysis.html"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span id="sec-proteomics-analysis" class="quarto-section-identifier"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="3.1">
<h2 data-number="3.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">3.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at Lesson Learnt: <a href="#sec-lesson-learnt-proteomics" class="quarto-xref"><span>Section 3.5</span></a>.</p>
<ul>
<li>Here we investigate the proteomics data across the three Kidney Cancers.</li>
<li>First, we perform a QC on protein spectra abundance, their distribution across samples and we impute eventual missing values.</li>
<li>Then, we do some exploratory analyses on samples, gene expression and clinical covariates.</li>
<li>We then run a formal Differential Protein Abundance analysis to identify proteins that have different expression levels across the three Kidney cancer types.</li>
<li>We perform Gene Set Enrichment Analyses on the differentially abundant proteins to investigate biological and molecular themes that discriminates between the three Kidney cancer types.</li>
</ul>
</section>
<section id="proteomics-data-overview-and-qc" class="level2" data-number="3.2">
<h2 data-number="3.2" class="anchored" data-anchor-id="proteomics-data-overview-and-qc"><span class="header-section-number">3.2</span> Proteomics data overview and QC</h2>
<section id="spectral-counts-overview" class="level3" data-number="3.2.1">
<h3 data-number="3.2.1" class="anchored" data-anchor-id="spectral-counts-overview"><span class="header-section-number">3.2.1</span> Spectral counts overview</h3>
<p>For TCGA Kidney cancer proteomics data, we will perform analyses similar to the ones performed for the transcriptomics one (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>). We first start with an overview of the datasets and QC.</p>
<p>Let’s check the spectral counts distributions across samples, and the protein completeness across samples.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 1: Proteomics spectral counts missing values</figcaption>
</figure>
</div>
</div>
</div>
<p>Looking at the distributions, we can conclude that the spectral intensities were already normalized and scaled, since the spectral values are centered around 0 and have a normal distribution.</p>
<p>We have 258 proteins in total, across 709 samples (79.9% of samples had proteomics data).</p>
<ul>
<li>24 proteins were missing in all samples (100% of patients with proteomics data available)</li>
<li>1 protein was missing in 440 samples (62.1% of patients with proteomics data available)</li>
<li>16 proteins were missing in 269 samples (37.9% of patients with proteomics data available)</li>
<li>6 protein were missing in only 1 sample 211 proteins were detected for all patients with proteomics data available)</li>
</ul>
<p>We can obviously drop the 24 proteins for which we have not spectral counts for none of the samples. For the remaining proteins for which we do not have full occupancy, we will apply a standard imputation strategy.</p>
</section>
<section id="data-imputation" class="level3" data-number="3.2.2">
<h3 data-number="3.2.2" class="anchored" data-anchor-id="data-imputation"><span class="header-section-number">3.2.2</span> Data imputation</h3>
<p>Missing values imputation for label-free proteomics is a large and complex <a href="https://pubs.acs.org/doi/10.1021/pr501138h">field of study</a>.</p>
<p>In our Kidney cancer proteomics samples the number of proteins detected is small and the distribution of missing values is oddly discrete. This suggests that some kind of filtering was already applied during the preprocessing steps, together with scaling and normalization of the proteins spectral counts. The absence can therefore as well be ascribed to biological reasons as well, and not only to technical problems with its detection.</p>
<p>To preserve this putative biological signal, I replace in each sample the missing values with the minimum spectral count available in that sample.</p>
<p>Let’s check how the spectral distributions changed.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 2: Proteomics spectral counts after imputation</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="dimesionality-reduction-and-dataset-exploration" class="level2" data-number="3.3">
<h2 data-number="3.3" class="anchored" data-anchor-id="dimesionality-reduction-and-dataset-exploration"><span class="header-section-number">3.3</span> Dimesionality Reduction and Dataset Exploration</h2>
<section id="umap-on-filtered-transcriptomics-data" class="level3" data-number="3.3.1">
<h3 data-number="3.3.1" class="anchored" data-anchor-id="umap-on-filtered-transcriptomics-data"><span class="header-section-number">3.3.1</span> UMAP on filtered transcriptomics data</h3>
<p>Let’s check the samples clustering on a UMAP. We have almost a perfect clustering of the samples in their corresponding cancer subtypes, with just 2 KICH samples clusting with KIRC samples.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 3: Kidney proteomics UMAP</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="principal-component-analysis-pca" class="level3" data-number="3.3.2">
<h3 data-number="3.3.2" class="anchored" data-anchor-id="principal-component-analysis-pca"><span class="header-section-number">3.3.2</span> Principal Component Analysis (PCA)</h3>
<p>As we did for the transcriptomics data (see <a href="transcriptomics-analysis.html" class="quarto-xref"><span>Chapter 2</span></a>)., the next step in the Dataset exploration is to perform the Principal Component Analysis.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 4: PCA, exploration</figcaption>
</figure>
</div>
</div>
</div>
<p>The first 21 Principal Components capture more than 80% of the variance in the Kidney cancers transcriptomics dataset, with the first two components (PC1 and PC2) capturing a bit more than 40% of the variance.</p>
<p>When we project the samples in the PC1 and PC2, we can see that the PC1 separates KIRC from KICH and KIRP, which instead cluster together. The second component PC2, instead, seems to separate KICH and KIRP samples. The separation between KICH and KIRP samples along component PC2 is clearer in the proteomics data than in the transcriptomics data.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 5: PCA, biplot PC1 x PC2</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 6: PCA, pairs plots</figcaption>
</figure>
</div>
</div>
</div>
<p>Let’s check the loadings (i.e.: 21 Principal Components capture more than 80% of the variance in the) for the top 4 Principal Components. These indicate which genes are the more responsible to explain the position of the samples along the components, and the direction of this separation.</p>
<p>[[[[ADAPT TO PROTEOMICS RESULTS]]]]</p>
<p>Looking at the top 1% most variable genes (~ 140 genes), the following 5 genes are the top loadings for the first 4 Principal Components:</p>
<ol type="1">
<li><a href="https://doi.org/10.1080/21645515.2024.2350101">SPP1</a></li>
<li><a href="https://onlinelibrary.wiley.com/doi/10.1155/2019/7259691">VIM</a></li>
<li><a href="https://www.mdpi.com/2072-6694/14/6/1495">FTL</a></li>
<li><a href="https://pathsocjournals.onlinelibrary.wiley.com/doi/full/10.1002/cjp2.232">RGS5</a></li>
<li><a href="https://www.nature.com/articles/s41598-020-71997-6">LOC96610</a></li>
</ol>
<p>[[[[ADAPT TO PROTEOMICS RESULTS]]]]</p>
<p>Let’s now check the expression of the five top genes identified with the PCA across the cancer types:</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 6: PCA, loadings</figcaption>
</figure>
</div>
</div>
</div>
<p>Claudin-7 is a known diagnostic himmunohistochemical marker for <a href="https://www.sciencedirect.com/science/article/pii/S0893395222023766">classifing chromophobe renal carcinoma (KICH)</a>, and indeed seems overexpressed in KICH when compared to KIRC and KIRP.</p>
<p>Let’s check the Pearson correlation with other clinical covariates.</p>
<p>[[[[ADAPT TO PROTEOMICS RESULTS]]]]</p>
<p>Certain histological and molecular subtyping correlates perfectly with PC1 and PC2 (which discriminates KICH) immune infiltrating cells also correlates with PC1 and PC2, and may help to further characterize the subtypes and stratify patients</p>
<p>As we have learnt before (<a href="dataset-exploration.html#sec-dataset-exploration" class="quarto-xref"><span>Section 1.3.7</span></a>), KIRC patients seems to had a worse outcome than KIRP and to have tumors in more advanced stages. PC1 (computed from transcriptomics data) clearly separates KIRC and KIRP samples, and it correlates with tumor histological grade and stage, as well as clinical outcome. This show that we have good correlation between clinical observations and gene espression in Kidney cancers.</p>
<p>sadly, no correlation between the top 5 components and the outcome of therapeutic care.</p>
<p>[[[[ADAPT TO PROTEOMICS RESULTS]]]]</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 6: PCA, correlations between clinical covariates and Principal Components</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="differential-protein-abundance-analysis" class="level2" data-number="3.4">
<h2 data-number="3.4" class="anchored" data-anchor-id="differential-protein-abundance-analysis"><span class="header-section-number">3.4</span> Differential protein abundance analysis</h2>
<p>In addition to cancer type, we saw that age, ethnicity (and race) and age had somewhat a correlation with the cancer types.</p>
<p>We may want to include this covariates in the differential gene expression analysis in order to include their contribution into the model.</p>
<p>print(summary(dt)) KIRC_vs_KICH KIRP_vs_KICH KIRC_vs_KIRP Down 92 89 105 NotSig 37 48 27 Up 105 97 102</p>
<p>[[[[[[[[[[[[[ARE THE GENES/PROTEINS IN THE PCA dge??????]]]]]]]]]]</p>
<section id="identification-of-differentially-expressed-proteins" class="level3" data-number="3.4.1">
<h3 data-number="3.4.1" class="anchored" data-anchor-id="identification-of-differentially-expressed-proteins"><span class="header-section-number">3.4.1</span> Identification of differentially expressed proteins</h3>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 6: Volcano plots of each contrats reporting the proteins differentially expressed</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 7: UpSet reporting the proteins differentially expressed upregulated and downregulated in common across all contrasts</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="comparative-enrichment-analyses-across-cancer-types" class="level3" data-number="3.4.2">
<h3 data-number="3.4.2" class="anchored" data-anchor-id="comparative-enrichment-analyses-across-cancer-types"><span class="header-section-number">3.4.2</span> Comparative enrichment analyses across cancer types</h3>
<p>ORA vs GSEA:</p>
<ul>
<li>Over-Representation Analysis (ORA): It assesses whether the number of genes from a specific pathway in our list of DEGs is greater than expected by chance –&gt; To identify pathways that are significantly over-represented in a given list of DEGs, providing insights into the most affected biological processes by our treatments</li>
<li>Gene Set Enrichment Analysis (GSEA): It assesses whether the members of a pathway are randomly distributed throughout the ranked list or primarily found at the top or bottom, indicating upregulation or downregulation –&gt; To identify whether specific pathways are upregulated or downregulated across the entire list of genes, providing insight into the global effects of the treatments</li>
</ul>
<p>prep gene lists</p>
<section id="go-biological-process-terms" class="level4" data-number="3.4.2.1">
<h4 data-number="3.4.2.1" class="anchored" data-anchor-id="go-biological-process-terms"><span class="header-section-number">3.4.2.1</span> GO Biological Process terms</h4>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-1-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-1" role="tab" aria-controls="tabset-1-1" aria-selected="true" aria-current="page">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-2" role="tab" aria-controls="tabset-1-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-1-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-1-3" role="tab" aria-controls="tabset-1-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-1-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-1-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-17-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 8: dotplot of enriched GO Biological Process terms</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-18-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 9: Term maps reporting relationships between GO Biological Process across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-1-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-1-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-19-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 10: gene maps reporting relationships between GO Biological Process enriched across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-eec4baf99f895b8f1eab" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-eec4baf99f895b8f1eab">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Enrich<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["GO:0016052","GO:0005980","GO:0009251","GO:0000272","GO:0005977","GO:0044042","GO:0006112","GO:0005976","GO:0036293","GO:0006089","GO:0006090","GO:0070482","GO:0036092","GO:0032205","GO:1990090","GO:1990089","GO:0070266","GO:0010332","GO:0097300","GO:1904356","GO:0051402","GO:0046854","GO:0001666","GO:0010833","GO:0002573","GO:0030099","GO:0032205","GO:0051054","GO:0010332","GO:0006310","GO:1904356","GO:0010639","GO:0030888","GO:0045911","GO:0002562","GO:0016444","GO:0002200","GO:0010833","GO:1900182","GO:2001251","GO:0032204","GO:0042100","GO:0030316","GO:0032526","GO:0050864","GO:0000018","GO:0010212","GO:1900180","GO:0051053","GO:0000723","GO:0000724","GO:0000725","GO:0071478","GO:0032200","GO:0043123","GO:0031334","GO:0002520","GO:0043523","GO:0050670","GO:0002274","GO:0032944","GO:0033044","GO:0002703","GO:0034599","GO:0014866","GO:0030242","GO:0033572","GO:0045141","GO:0048251","GO:0051409","GO:1900543","GO:1904760","GO:0043122","GO:0070663","GO:0008611","GO:0030223","GO:0033148","GO:0045980","GO:0046504","GO:0062100","GO:0070213","GO:0071044","GO:0071481","GO:0090220","GO:0090557","GO:1904181","GO:0002699","GO:0042113","GO:0051402","GO:0000012","GO:0002327","GO:0010635","GO:0033145","GO:0034397","GO:0097384","GO:1901503","GO:1903624","GO:0018105","GO:0001666","GO:0007249","GO:0002366","GO:0051547","GO:0002263","GO:0006302","GO:0046651","GO:0018209","GO:0034504","GO:0032943","GO:0036293","GO:0033008","GO:0036446","GO:0042159","GO:0043306","GO:0045187","GO:0062197","GO:0097193","GO:0044403","GO:0043517","GO:0050802","GO:1900034","GO:1904177","GO:0071214","GO:0104004","GO:0030889","GO:0033151","GO:0090399","GO:0070482","GO:0006977","GO:0010042","GO:0045780","GO:1904044","GO:0070661","GO:0022410","GO:0042749","GO:1903376","GO:2001169","GO:0010038","GO:0008334","GO:0034244","GO:0051546","GO:0051900","GO:1903978","GO:0006662","GO:0033005","GO:0036480","GO:0042745","GO:0044030","GO:0046485","GO:0002696","GO:0023019","GO:0030431","GO:0032042","GO:0036092","GO:0002697","GO:0032785","GO:0051882","GO:0050867","GO:0006979","GO:0010839","GO:0071294","GO:1903578","GO:0043302","GO:0046697","GO:0043254","GO:0018904","GO:0060391","GO:1904353","GO:0071480","GO:1904357","GO:0009314","GO:0016540","GO:0048732","GO:0010165","GO:0031571","GO:0044819","GO:0045830","GO:0000002","GO:0010039","GO:0043304","GO:1901798","GO:0003012","GO:0008053","GO:0002443","GO:0006308","GO:0030808","GO:0033006","GO:1900371","GO:0051656","GO:0050869","GO:1903829","GO:0032102","GO:1903131","GO:0032212","GO:0071353","GO:0045785","GO:0001893","GO:0008340","GO:0010613","GO:0030224","GO:0033146","GO:0060544","GO:0002068","GO:0014742","GO:0070670","GO:0007095","GO:0046688","GO:0051385","GO:1904358","GO:1905168","GO:0007143","GO:0043516","GO:0016233","GO:0045191","GO:0016052","GO:0005980","GO:0009251","GO:0000272","GO:0005977","GO:0044042","GO:0006112","GO:0005976","GO:0006090","GO:0006089","GO:0015980","GO:0006096","GO:0014866","GO:0048251"],["carbohydrate catabolic process","glycogen catabolic process","glucan catabolic process","polysaccharide catabolic process","glycogen metabolic process","glucan metabolic process","energy reserve metabolic process","polysaccharide metabolic process","response to decreased oxygen levels","lactate metabolic process","pyruvate metabolic process","response to oxygen levels","phosphatidylinositol-3-phosphate biosynthetic process","negative regulation of telomere maintenance","cellular response to nerve growth factor stimulus","response to nerve growth factor","necroptotic process","response to gamma radiation","programmed necrotic cell death","regulation of telomere maintenance via telomere lengthening","neuron apoptotic process","phosphatidylinositol phosphate biosynthetic process","response to hypoxia","telomere maintenance via telomere lengthening","myeloid leukocyte differentiation","myeloid cell differentiation","negative regulation of telomere maintenance","positive regulation of DNA metabolic process","response to gamma radiation","DNA recombination","regulation of telomere maintenance via telomere lengthening","negative regulation of organelle organization","regulation of B cell proliferation","positive regulation of DNA recombination","somatic diversification of immune receptors via germline recombination within a single locus","somatic cell DNA recombination","somatic diversification of immune receptors","telomere maintenance via telomere lengthening","positive regulation of protein localization to nucleus","negative regulation of chromosome organization","regulation of telomere maintenance","B cell proliferation","osteoclast differentiation","response to retinoic acid","regulation of B cell activation","regulation of DNA recombination","response to ionizing radiation","regulation of protein localization to nucleus","negative regulation of DNA metabolic process","telomere maintenance","double-strand break repair via homologous recombination","recombinational repair","cellular response to radiation","telomere organization","positive regulation of canonical NF-kappaB signal transduction","positive regulation of protein-containing complex assembly","immune system development","regulation of neuron apoptotic process","regulation of lymphocyte proliferation","myeloid leukocyte activation","regulation of mononuclear cell proliferation","regulation of chromosome organization","regulation of leukocyte mediated immunity","cellular response to oxidative stress","skeletal myofibril assembly","autophagy of peroxisome","transferrin transport","meiotic telomere clustering","elastic fiber assembly","response to nitrosative stress","negative regulation of purine nucleotide metabolic process","regulation of myofibroblast differentiation","regulation of canonical NF-kappaB signal transduction","regulation of leukocyte proliferation","ether lipid biosynthetic process","neutrophil differentiation","positive regulation of intracellular estrogen receptor signaling pathway","negative regulation of nucleotide metabolic process","glycerol ether biosynthetic process","positive regulation of programmed necrotic cell death","protein auto-ADP-ribosylation","histone mRNA catabolic process","cellular response to X-ray","chromosome localization to nuclear envelope involved in homologous chromosome segregation","establishment of endothelial intestinal barrier","positive regulation of membrane depolarization","positive regulation of immune effector process","B cell activation","neuron apoptotic process","single strand break repair","immature B cell differentiation","regulation of mitochondrial fusion","positive regulation of intracellular steroid hormone receptor signaling pathway","telomere localization","cellular lipid biosynthetic process","ether biosynthetic process","regulation of DNA catabolic process","peptidyl-serine phosphorylation","response to hypoxia","canonical NF-kappaB signal transduction","leukocyte activation involved in immune response","regulation of keratinocyte migration","cell activation involved in immune response","double-strand break repair","lymphocyte proliferation","peptidyl-serine modification","protein localization to nucleus","mononuclear cell proliferation","response to decreased oxygen levels","positive regulation of mast cell activation involved in immune response","myofibroblast differentiation","lipoprotein catabolic process","positive regulation of mast cell degranulation","regulation of circadian sleep/wake cycle, sleep","cellular response to chemical stress","intrinsic apoptotic signaling pathway","biological process involved in symbiotic interaction","positive regulation of DNA damage response, signal transduction by p53 class mediator","circadian sleep/wake cycle, sleep","regulation of cellular response to heat","regulation of adipose tissue development","cellular response to abiotic stimulus","cellular response to environmental stimulus","negative regulation of B cell proliferation","V(D)J recombination","replicative senescence","response to oxygen levels","DNA damage response, signal transduction by p53 class mediator resulting in cell cycle arrest","response to manganese ion","positive regulation of bone resorption","response to aldosterone","leukocyte proliferation","circadian sleep/wake cycle process","regulation of circadian sleep/wake cycle","regulation of oxidative stress-induced neuron intrinsic apoptotic signaling pathway","regulation of ATP biosynthetic process","response to metal ion","histone mRNA metabolic process","negative regulation of transcription elongation by RNA polymerase II","keratinocyte migration","regulation of mitochondrial depolarization","regulation of microglial cell activation","glycerol ether metabolic process","positive regulation of mast cell activation","neuron intrinsic apoptotic signaling pathway in response to oxidative stress","circadian sleep/wake cycle","regulation of DNA methylation","ether lipid metabolic process","positive regulation of leukocyte activation","signal transduction involved in regulation of gene expression","sleep","mitochondrial DNA metabolic process","phosphatidylinositol-3-phosphate biosynthetic process","regulation of immune effector process","negative regulation of DNA-templated transcription, elongation","mitochondrial depolarization","positive regulation of cell activation","response to oxidative stress","negative regulation of keratinocyte proliferation","cellular response to zinc ion","regulation of ATP metabolic process","positive regulation of leukocyte degranulation","decidualization","regulation of protein-containing complex assembly","ether metabolic process","positive regulation of SMAD protein signal transduction","regulation of telomere capping","cellular response to gamma radiation","negative regulation of telomere maintenance via telomere lengthening","response to radiation","protein autoprocessing","gland development","response to X-ray","mitotic G1 DNA damage checkpoint signaling","mitotic G1/S transition checkpoint signaling","positive regulation of isotype switching","mitochondrial genome maintenance","response to iron ion","regulation of mast cell degranulation","positive regulation of signal transduction by p53 class mediator","muscle system process","mitochondrial fusion","leukocyte mediated immunity","DNA catabolic process","regulation of nucleotide biosynthetic process","regulation of mast cell activation involved in immune response","regulation of purine nucleotide biosynthetic process","establishment of organelle localization","negative regulation of B cell activation","positive regulation of protein localization","negative regulation of response to external stimulus","mononuclear cell differentiation","positive regulation of telomere maintenance via telomerase","cellular response to interleukin-4","positive regulation of cell adhesion","maternal placenta development","determination of adult lifespan","positive regulation of cardiac muscle hypertrophy","monocyte differentiation","regulation of intracellular estrogen receptor signaling pathway","regulation of necroptotic process","glandular epithelial cell development","positive regulation of muscle hypertrophy","response to interleukin-4","mitotic G2 DNA damage checkpoint signaling","response to copper ion","response to mineralocorticoid","positive regulation of telomere maintenance via telomere lengthening","positive regulation of double-strand break repair via homologous recombination","female meiotic nuclear division","regulation of DNA damage response, signal transduction by p53 class mediator","telomere capping","regulation of isotype switching","carbohydrate catabolic process","glycogen catabolic process","glucan catabolic process","polysaccharide catabolic process","glycogen metabolic process","glucan metabolic process","energy reserve metabolic process","polysaccharide metabolic process","pyruvate metabolic process","lactate metabolic process","energy derivation by oxidation of organic compounds","glycolytic process","skeletal myofibril assembly","elastic fiber assembly"],["5/14","3/14","3/14","3/14","3/14","3/14","3/14","3/14","4/14","2/14","3/14","4/14","2/14","2/14","2/14","2/14","2/14","2/14","2/14","2/14","3/14","2/14","3/14","2/14","4/7","4/7","2/7","3/7","2/7","3/7","2/7","3/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","2/7","2/7","1/7","2/7","2/7","2/7","2/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","2/7","2/7","2/7","1/7","1/7","1/7","1/7","2/7","2/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","2/7","2/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","2/7","1/7","2/7","1/7","1/7","1/7","1/7","2/7","1/7","2/7","2/7","2/7","1/7","1/7","2/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","1/7","5/9","3/9","3/9","3/9","3/9","3/9","3/9","3/9","3/9","2/9","3/9","2/9","1/9","1/9"],["163/18870","18/18870","19/18870","21/18870","76/18870","78/18870","89/18870","102/18870","315/18870","18/18870","118/18870","343/18870","22/18870","35/18870","46/18870","48/18870","49/18870","51/18870","60/18870","61/18870","278/18870","66/18870","298/18870","81/18870","235/18870","430/18870","35/18870","298/18870","51/18870","340/18870","61/18870","363/18870","65/18870","71/18870","72/18870","72/18870","78/18870","81/18870","93/18870","94/18870","104/18870","104/18870","106/18870","106/18870","130/18870","136/18870","138/18870","143/18870","146/18870","156/18870","175/18870","179/18870","181/18870","187/18870","196/18870","198/18870","199/18870","232/18870","238/18870","241/18870","242/18870","247/18870","249/18870","255/18870","10/18870","10/18870","10/18870","10/18870","10/18870","10/18870","10/18870","10/18870","265/18870","267/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","11/18870","270/18870","278/18870","278/18870","12/18870","12/18870","12/18870","12/18870","12/18870","12/18870","12/18870","13/18870","295/18870","298/18870","300/18870","301/18870","14/18870","305/18870","307/18870","307/18870","308/18870","310/18870","314/18870","315/18870","15/18870","15/18870","15/18870","15/18870","15/18870","317/18870","319/18870","322/18870","16/18870","16/18870","16/18870","16/18870","334/18870","334/18870","17/18870","17/18870","17/18870","343/18870","18/18870","18/18870","18/18870","18/18870","350/18870","19/18870","19/18870","19/18870","19/18870","359/18870","20/18870","20/18870","20/18870","20/18870","20/18870","21/18870","21/18870","21/18870","21/18870","21/18870","21/18870","380/18870","22/18870","22/18870","22/18870","22/18870","389/18870","23/18870","23/18870","396/18870","400/18870","24/18870","24/18870","24/18870","25/18870","25/18870","419/18870","26/18870","26/18870","26/18870","27/18870","27/18870","428/18870","28/18870","443/18870","29/18870","29/18870","29/18870","29/18870","30/18870","30/18870","30/18870","30/18870","452/18870","31/18870","466/18870","32/18870","32/18870","32/18870","32/18870","469/18870","33/18870","476/18870","481/18870","481/18870","34/18870","34/18870","482/18870","35/18870","35/18870","35/18870","35/18870","35/18870","35/18870","36/18870","36/18870","36/18870","37/18870","37/18870","37/18870","37/18870","37/18870","38/18870","38/18870","39/18870","39/18870","163/18870","18/18870","19/18870","21/18870","76/18870","78/18870","89/18870","102/18870","118/18870","18/18870","337/18870","89/18870","10/18870","10/18870"],[8.499999999999999e-08,2.64e-07,3.13e-07,4.29e-07,2.21e-05,2.39e-05,3.56e-05,5.35e-05,6.68e-05,7.77e-05,8.25e-05,9.3e-05,0.000117,0.0003,0.000519,0.000565,0.000589,0.000638,0.000883,0.0009120000000000001,0.00102,0.00107,0.00125,0.0016,7.97e-07,8.81e-06,6.98e-05,0.00013,0.000149,0.000192,0.000214,0.000233,0.000243,0.00029,0.000298,0.000298,0.00035,0.000377,0.000497,0.000507,0.00062,0.00062,0.000644,0.000644,0.000967,0.00106,0.00109,0.00117,0.00122,0.00139,0.00174,0.00182,0.00186,0.00199,0.00218,0.00222,0.00224,0.00303,0.00319,0.00327,0.0033,0.00343,0.00349,0.00365,0.0037,0.0037,0.0037,0.0037,0.0037,0.0037,0.0037,0.0037,0.00394,0.004,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00407,0.00409,0.00433,0.00433,0.00444,0.00444,0.00444,0.00444,0.00444,0.00444,0.00444,0.00481,0.00486,0.00495,0.00502,0.00505,0.00518,0.00518,0.00525,0.00525,0.00528,0.00535,0.00548,0.00552,0.00555,0.00555,0.00555,0.00555,0.00555,0.00559,0.00566,0.00576,0.00592,0.00592,0.00592,0.00592,0.00619,0.00619,0.00629,0.00629,0.00629,0.00651,0.00666,0.00666,0.00666,0.00666,0.00677,0.00703,0.00703,0.00703,0.00703,0.00712,0.0074,0.0074,0.0074,0.0074,0.0074,0.00777,0.00777,0.00777,0.00777,0.00777,0.00777,0.007939999999999999,0.00813,0.00813,0.00813,0.00813,0.00831,0.008500000000000001,0.008500000000000001,0.0086,0.00877,0.008869999999999999,0.008869999999999999,0.008869999999999999,0.00924,0.00924,0.00959,0.00961,0.00961,0.00961,0.00997,0.00997,0.009990000000000001,0.0103,0.0107,0.0107,0.0107,0.0107,0.0107,0.0111,0.0111,0.0111,0.0111,0.0111,0.0114,0.0118,0.0118,0.0118,0.0118,0.0118,0.0119,0.0122,0.0123,0.0125,0.0125,0.0125,0.0125,0.0126,0.0129,0.0129,0.0129,0.0129,0.0129,0.0129,0.0133,0.0133,0.0133,0.0136,0.0136,0.0136,0.0136,0.0136,0.014,0.014,0.0144,0.0144,5.54e-09,6.1e-08,7.24e-08,9.94e-08,5.18e-06,5.61e-06,8.35e-06,1.26e-05,1.95e-05,3.08e-05,0.000438,0.000775,0.00476,0.00476],[6.26e-05,7.669999999999999e-05,7.669999999999999e-05,7.889999999999999e-05,0.00294,0.00294,0.00374,0.00492,0.00546,0.00552,0.00552,0.0057,0.00663,0.0158,0.0255,0.0255,0.0255,0.0261,0.0336,0.0336,0.0357,0.0357,0.0399,0.0491,0.00052,0.00287,0.0152,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0162,0.0175,0.0176,0.0207,0.0207,0.021,0.021,0.021,0.021,0.03,0.0309,0.0309,0.0317,0.0317,0.0348,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0398,0.0402,0.0402,0.0402,0.0402,0.0402,0.0406,0.0406,0.0406,0.0406,0.0406,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0408,0.0411,0.0411,0.0411,0.0411,0.0411,0.0417,0.042,0.042,0.0422,0.0422,0.0422,0.0422,0.0422,0.0433,0.0433,0.0438,0.0438,0.0438,0.0438,0.0446,0.0446,0.0446,0.0459,0.0459,0.0459,0.0459,0.0459,0.0459,0.0461,0.0461,0.0461,0.0461,0.0461,0.0472,0.0472,0.0472,0.0472,0.0472,0.0472,0.0474,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0476,0.0481,0.0481,0.0481,0.0481,0.0481,0.0481,0.0481,0.0481,0.0489,0.0489,0.0496,0.0496,7.59e-07,3.31e-06,3.31e-06,3.4e-06,0.000128,0.000128,0.000163,0.000215,0.000297,0.000422,0.00545,0.00885,0.0466,0.0466],[3.57e-05,4.38e-05,4.38e-05,4.5e-05,0.00168,0.00168,0.00213,0.00281,0.00312,0.00315,0.00315,0.00325,0.00378,0.008999999999999999,0.0145,0.0146,0.0146,0.0149,0.0192,0.0192,0.0204,0.0204,0.0228,0.028,0.000188,0.00104,0.00548,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00585,0.00634,0.00635,0.00748,0.00748,0.0076,0.0076,0.0076,0.0076,0.0109,0.0112,0.0112,0.0115,0.0115,0.0126,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0144,0.0145,0.0145,0.0145,0.0145,0.0145,0.0147,0.0147,0.0147,0.0147,0.0147,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0148,0.0149,0.0149,0.0149,0.0149,0.0149,0.0151,0.0152,0.0152,0.0153,0.0153,0.0153,0.0153,0.0153,0.0157,0.0157,0.0158,0.0158,0.0158,0.0158,0.0161,0.0161,0.0161,0.0166,0.0166,0.0166,0.0166,0.0166,0.0166,0.0167,0.0167,0.0167,0.0167,0.0167,0.0171,0.0171,0.0171,0.0171,0.0171,0.0171,0.0171,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0172,0.0174,0.0174,0.0174,0.0174,0.0174,0.0174,0.0174,0.0174,0.0177,0.0177,0.0179,0.0179,4.61e-07,2.01e-06,2.01e-06,2.07e-06,7.77e-05,7.77e-05,9.92e-05,0.000131,0.00018,0.000256,0.00331,0.00537,0.0283,0.0283],["GAPDH;LDHA;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","ATM;CA9;EEF2K;GATA6","LDHA;LDHB","GAPDH;LDHA;LDHB","ATM;CA9;EEF2K;GATA6","ATM;INPP4B","ATM;PARP1","EEF2K;PARP1","EEF2K;PARP1","PARP1;PYGL","ATM;PARP1","PARP1;PYGL","ATM;PARP1","ATM;GAPDH;PARP1","ATM;INPP4B","ATM;CA9;GATA6","ATM;PARP1","FASN;GAB2;PARP1;TFRC","FASN;GAB2;PARP1;TFRC","ATM;PARP1","ATM;PARP1;TFRC","ATM;PARP1","ATM;PARP1;TFRC","ATM;PARP1","ATM;PARP1;TFRC","ATM;TFRC","PARP1;TFRC","ATM;TFRC","ATM;TFRC","ATM;TFRC","ATM;PARP1","PARP1;TFRC","ATM;PARP1","ATM;PARP1","ATM;TFRC","GAB2;TFRC","ATM;TFRC","ATM;TFRC","PARP1;TFRC","ATM;PARP1","PARP1;TFRC","ATM;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","ATM;PARP1","PARP1;TFRC","ATM;TFRC","ATM;TFRC","ATM;PARP1","ATM;TFRC","ATM;GAB2","ATM;TFRC","ATM;PARP1","GAB2;TFRC","ATM;PARP1","MYH11","ATM","TFRC","ATM","MYH11","ATM","PARP1","PARP1","PARP1;TFRC","ATM;TFRC","FASN","FASN","PARP1","PARP1","FASN","PARP1","PARP1","ATM","ATM","ATM","FASN","PARP1","GAB2;TFRC","ATM;TFRC","ATM;PARP1","PARP1","ATM","TFRC","PARP1","ATM","FASN","FASN","ATM","ATM;TFRC","ATM;TFRC","PARP1;TFRC","GAB2;TFRC","EPPK1","GAB2;TFRC","ATM;PARP1","ATM;TFRC","ATM;TFRC","PARP1;TFRC","ATM;TFRC","ATM;TFRC","GAB2","PARP1","ATM","GAB2","PARP1","ATM;PARP1","ATM;PARP1","FASN;TFRC","ATM","PARP1","ATM","PARP1","ATM;PARP1","ATM;PARP1","ATM","ATM","ATM","ATM;TFRC","ATM","TFRC","TFRC","PARP1","ATM;TFRC","PARP1","PARP1","PARP1","PARP1","PARP1;TFRC","ATM","PARP1","EPPK1","PARP1","ATM","FASN","GAB2","PARP1","PARP1","PARP1","FASN","GAB2;TFRC","PARP1","PARP1","PARP1","ATM","GAB2;TFRC","PARP1","PARP1","GAB2;TFRC","ATM;PARP1","EPPK1","PARP1","PARP1","GAB2","PARP1","ATM;TFRC","FASN","PARP1","ATM","ATM","PARP1","ATM;PARP1","PARP1","ATM;FASN","ATM","ATM","ATM","TFRC","PARP1","TFRC","GAB2","ATM","MYH11;PARP1","TFRC","GAB2;TFRC","ATM","PARP1","GAB2","PARP1","ATM;GAB2","ATM","PARP1;TFRC","EPPK1;PARP1","ATM;FASN","ATM","FASN","ATM;TFRC","PARP1","ATM","PARP1","FASN","PARP1","PARP1","FASN","PARP1","FASN","ATM","TFRC","PARP1","ATM","PARP1","ATM","ATM","ATM","TFRC","GAPDH;LDHA;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","LDHA;LDHB","PYGB;PYGL;PYGM","GAPDH;LDHA","MYH11","MYH11"],[5,3,3,3,3,3,3,3,4,2,3,4,2,2,2,2,2,2,2,2,3,2,3,2,4,4,2,3,2,3,2,3,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,2,2,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,1,1,1,1,1,1,1,1,2,2,2,2,1,2,2,2,2,2,2,2,1,1,1,1,1,2,2,2,1,1,1,1,2,2,1,1,1,2,1,1,1,1,2,1,1,1,1,2,1,1,1,1,1,1,1,1,1,1,1,2,1,1,1,1,2,1,1,2,2,1,1,1,1,1,2,1,1,1,1,1,2,1,2,1,1,1,1,1,1,1,1,2,1,2,1,1,1,1,2,1,2,2,2,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,5,3,3,3,3,3,3,3,3,2,3,2,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="go-cellular-compartments-terms" class="level4" data-number="3.4.2.2">
<h4 data-number="3.4.2.2" class="anchored" data-anchor-id="go-cellular-compartments-terms"><span class="header-section-number">3.4.2.2</span> GO Cellular Compartments terms</h4>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-2-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-1" role="tab" aria-controls="tabset-2-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-2" role="tab" aria-controls="tabset-2-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-2-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-2-3" role="tab" aria-controls="tabset-2-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-2-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-2-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 11: dotplot of enriched GO Cellular Compartments terms</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-22-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 12: Term maps reporting relationships between GO Cellular Compartments across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-2-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-2-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-23-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 13: gene maps reporting relationships between GO Cellular Compartments enriched across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-745cd2c560f67d857b7c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-745cd2c560f67d857b7c">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Enrich<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23"],["KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["GO:0042470","GO:0048770","GO:0000781","GO:0016323","GO:0009925","GO:0045178","GO:0005859","GO:0016327","GO:1990391","GO:0098687","GO:0032982","GO:0016460","GO:0043596","GO:1990204","GO:0005859","GO:0045121","GO:0098857","GO:0034774","GO:0060205","GO:0031983","GO:0032982","GO:0016460","GO:0031528"],["melanosome","pigment granule","chromosome, telomeric region","basolateral plasma membrane","basal plasma membrane","basal part of cell","muscle myosin complex","apicolateral plasma membrane","DNA repair complex","chromosomal region","myosin filament","myosin II complex","nuclear replication fork","oxidoreductase complex","muscle myosin complex","membrane raft","membrane microdomain","secretory granule lumen","cytoplasmic vesicle lumen","vesicle lumen","myosin filament","myosin II complex","microvillus membrane"],["3/7","3/7","2/7","2/7","2/7","2/7","1/7","1/7","1/7","2/7","1/7","1/7","1/7","2/9","1/9","2/9","2/9","2/9","2/9","2/9","1/9","1/9","1/9"],["112/19886","112/19886","171/19886","249/19886","282/19886","301/19886","15/19886","22/19886","22/19886","399/19886","24/19886","26/19886","34/19886","128/19886","15/19886","286/19886","287/19886","322/19886","325/19886","326/19886","24/19886","26/19886","33/19886"],[5.99e-06,5.99e-06,0.0015,0.00315,0.00401,0.00456,0.00527,0.00772,0.00772,0.007889999999999999,0.00842,0.00912,0.0119,0.00144,0.00677,0.00694,0.00699,0.00873,0.00889,0.00894,0.0108,0.0117,0.0148],[0.000147,0.000147,0.0245,0.0369,0.0369,0.0369,0.0369,0.0372,0.0372,0.0372,0.0372,0.0372,0.0449,0.0383,0.0383,0.0383,0.0383,0.0383,0.0383,0.0383,0.039,0.039,0.0445],[5.99e-05,5.99e-05,0.01,0.0151,0.0151,0.0151,0.0151,0.0152,0.0152,0.0152,0.0152,0.0152,0.0183,0.0188,0.0188,0.0188,0.0188,0.0188,0.0188,0.0188,0.0192,0.0192,0.0219],["FASN;MYH11;TFRC","FASN;MYH11;TFRC","ATM;PARP1","EPPK1;TFRC","EPPK1;TFRC","EPPK1;TFRC","MYH11","EPPK1","ATM","ATM;PARP1","MYH11","MYH11","PARP1","LDHA;LDHB","MYH11","GAB2;LDHB","GAB2;LDHB","PYGB;PYGL","PYGB;PYGL","PYGB;PYGL","MYH11","MYH11","CA9"],[3,3,2,2,2,2,1,1,1,2,1,1,1,2,1,2,2,2,2,2,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="go-molecular-function-terms" class="level4" data-number="3.4.2.3">
<h4 data-number="3.4.2.3" class="anchored" data-anchor-id="go-molecular-function-terms"><span class="header-section-number">3.4.2.3</span> GO Molecular Function terms</h4>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-3-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-1" role="tab" aria-controls="tabset-3-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-2" role="tab" aria-controls="tabset-3-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-3-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-3-3" role="tab" aria-controls="tabset-3-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-3-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-3-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-25-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 14: dotplot of enriched GO Molecular Functions terms</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-26-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 15: Term maps reporting relationships between GO Molecular Functions across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-3-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-3-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-27-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 16: gene maps reporting relationships between GO Molecular Functions enriched across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-b21f5581f841f87a29fe" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-b21f5581f841f87a29fe">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Enrich<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["GO:0051287","GO:0030170","GO:0070279","GO:0016757","GO:0019842","GO:0016758","GO:0016616","GO:0016614","GO:0016303","GO:0016884","GO:0032052","GO:0005536","GO:0019215","GO:0046030","GO:0004089","GO:0016208","GO:0030170","GO:0070279","GO:0019842","GO:0016758","GO:0016757","GO:0051287","GO:0016616","GO:0016614","GO:0032052","GO:0005536","GO:0004089","GO:0016208","GO:0005068","GO:0016769","GO:0043325","GO:0097718","GO:0000146","GO:0005547","GO:0016620"],["NAD binding","pyridoxal phosphate binding","vitamin B6 binding","glycosyltransferase activity","vitamin binding","hexosyltransferase activity","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","oxidoreductase activity, acting on CH-OH group of donors","1-phosphatidylinositol-3-kinase activity","carbon-nitrogen ligase activity, with glutamine as amido-N-donor","bile acid binding","glucose binding","intermediate filament binding","inositol trisphosphate phosphatase activity","carbonate dehydratase activity","AMP binding","pyridoxal phosphate binding","vitamin B6 binding","vitamin binding","hexosyltransferase activity","glycosyltransferase activity","NAD binding","oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor","oxidoreductase activity, acting on CH-OH group of donors","bile acid binding","glucose binding","carbonate dehydratase activity","AMP binding","transmembrane receptor protein tyrosine kinase adaptor activity","transferase activity, transferring nitrogenous groups","phosphatidylinositol-3,4-bisphosphate binding","disordered domain specific binding","microfilament motor activity","phosphatidylinositol-3,4,5-trisphosphate binding","oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor"],["3/14","3/14","3/14","4/14","3/14","3/14","2/14","2/14","1/14","1/14","1/14","1/14","1/14","1/14","1/14","1/14","3/9","3/9","3/9","3/9","3/9","2/9","2/9","2/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9","1/9"],["53/18496","56/18496","57/18496","268/18496","150/18496","196/18496","125/18496","135/18496","10/18496","10/18496","10/18496","11/18496","11/18496","12/18496","15/18496","15/18496","56/18496","57/18496","150/18496","196/18496","268/18496","53/18496","125/18496","135/18496","10/18496","11/18496","15/18496","15/18496","17/18496","26/18496","29/18496","32/18496","37/18496","39/18496","39/18496"],[7.91e-06,9.35e-06,9.86e-06,3.85e-05,0.000178,0.000391,0.00391,0.00454,0.00755,0.00755,0.00755,0.0083,0.0083,0.009050000000000001,0.0113,0.0113,2.18e-06,2.3e-06,4.24e-05,9.390000000000001e-05,0.000237,0.000286,0.00158,0.00184,0.00486,0.00534,0.00728,0.00728,0.008240000000000001,0.0126,0.014,0.0155,0.0179,0.0188,0.0188],[0.00023,0.00023,0.00023,0.000673,0.0025,0.00457,0.0391,0.0398,0.0447,0.0447,0.0447,0.0447,0.0447,0.0452,0.0494,0.0494,5.75e-05,5.75e-05,0.000706,0.00117,0.00237,0.00239,0.0113,0.0115,0.0267,0.0267,0.0303,0.0303,0.0317,0.0449,0.0468,0.0483,0.0495,0.0495,0.0495],[0.000111,0.000111,0.000111,0.000324,0.0012,0.0022,0.0188,0.0191,0.0215,0.0215,0.0215,0.0215,0.0215,0.0218,0.0238,0.0238,2.54e-05,2.54e-05,0.000312,0.000519,0.00105,0.00105,0.00499,0.00509,0.0118,0.0118,0.0134,0.0134,0.014,0.0199,0.0207,0.0214,0.0219,0.0219,0.0219],["GAPDH;LDHB;PARP1","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PARP1;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","ATM","ASNS","PYGL","PYGL","EPPK1","INPP4B","CA9","PYGL","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;LDHB","LDHA;LDHB","LDHA;LDHB","PYGL","PYGL","CA9","PYGL","GAB2","GAPDH","GAB2","GAPDH","MYH11","GAB2","GAPDH"],[3,3,3,4,3,3,2,2,1,1,1,1,1,1,1,1,3,3,3,3,3,2,2,2,1,1,1,1,1,1,1,1,1,1,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="kegg-pathways" class="level4" data-number="3.4.2.4">
<h4 data-number="3.4.2.4" class="anchored" data-anchor-id="kegg-pathways"><span class="header-section-number">3.4.2.4</span> KEGG pathways</h4>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-4-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-1" role="tab" aria-controls="tabset-4-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-2" role="tab" aria-controls="tabset-4-2" aria-selected="false">terms map</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-4-3-tab" data-bs-toggle="tab" data-bs-target="#tabset-4-3" role="tab" aria-controls="tabset-4-3" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-4-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-4-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-29-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 17: dotplot of enriched KEGG pathways</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-30-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 18: Term maps reporting relationships between KEGG pathways enriched across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-4-3" class="tab-pane" role="tabpanel" aria-labelledby="tabset-4-3-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-31-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 19: gene maps reporting relationships between KEGG pathways enriched across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-1daccd2b47e007c805cf" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-1daccd2b47e007c805cf">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Enrich<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRP_vs_KICH","KIRP_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["Organismal Systems","Metabolism","Cellular Processes","Metabolism","Human Diseases","Environmental Information Processing","Organismal Systems","Metabolism","Metabolism","Metabolism","Human Diseases","Metabolism","Environmental Information Processing","Cellular Processes",null,"Environmental Information Processing","Organismal Systems","Metabolism","Metabolism","Human Diseases","Environmental Information Processing","Organismal Systems","Metabolism","Cellular Processes","Metabolism","Metabolism","Human Diseases","Metabolism",null,"Human Diseases"],["Endocrine system","Carbohydrate metabolism","Cell growth and death","Carbohydrate metabolism","Endocrine and metabolic disease","Signal transduction","Endocrine system","Carbohydrate metabolism","Carbohydrate metabolism","Amino acid metabolism","Cancer: overview","Global and overview maps","Signal transduction","Cell growth and death",null,"Signal transduction","Endocrine system","Carbohydrate metabolism","Carbohydrate metabolism","Endocrine and metabolic disease","Signal transduction","Endocrine system","Carbohydrate metabolism","Cell growth and death","Carbohydrate metabolism","Amino acid metabolism","Cancer: overview","Energy metabolism",null,"Infectious disease: bacterial"],["hsa04922","hsa00500","hsa04217","hsa00010","hsa04931","hsa04066","hsa04910","hsa00640","hsa00620","hsa00270","hsa05230","hsa01230","hsa04064","hsa04210","hsa03273","hsa04064","hsa04922","hsa00500","hsa00010","hsa04931","hsa04066","hsa04910","hsa00640","hsa04217","hsa00620","hsa00270","hsa05230","hsa00910","hsa03273","hsa05130"],["Glucagon signaling pathway","Starch and sucrose metabolism","Necroptosis","Glycolysis / Gluconeogenesis","Insulin resistance","HIF-1 signaling pathway","Insulin signaling pathway","Propanoate metabolism","Pyruvate metabolism","Cysteine and methionine metabolism","Central carbon metabolism in cancer","Biosynthesis of amino acids","NF-kappa B signaling pathway","Apoptosis","Virion - Lassa virus and SFTS virus","NF-kappa B signaling pathway","Glucagon signaling pathway","Starch and sucrose metabolism","Glycolysis / Gluconeogenesis","Insulin resistance","HIF-1 signaling pathway","Insulin signaling pathway","Propanoate metabolism","Necroptosis","Pyruvate metabolism","Cysteine and methionine metabolism","Central carbon metabolism in cancer","Nitrogen metabolism","Virion - Lassa virus and SFTS virus","Pathogenic Escherichia coli infection"],["5/12","3/12","4/12","3/12","3/12","3/12","3/12","2/12","2/12","2/12","2/12","2/12","2/12","2/12","2/6","2/6","5/9","3/9","3/9","3/9","3/9","3/9","2/9","3/9","2/9","2/9","2/9","1/9","1/9","2/9"],["107/8538","40/8538","159/8538","67/8538","109/8538","110/8538","138/8538","32/8538","47/8538","52/8538","71/8538","75/8538","105/8538","136/8538","17/8538","105/8538","107/8538","40/8538","67/8538","109/8538","110/8538","138/8538","32/8538","159/8538","47/8538","52/8538","71/8538","17/8538","17/8538","203/8538"],[2.08e-07,2.04e-05,5.1e-05,9.66e-05,0.000409,0.000421,0.000817,0.000877,0.00189,0.00231,0.00426,0.00475,0.00912,0.015,5.57e-05,0.00218,3.41e-08,7.849999999999999e-06,3.75e-05,0.000161,0.000165,0.000323,0.000482,0.00049,0.00104,0.00127,0.00236,0.0178,0.0178,0.0181],[7.900000000000001e-06,0.000387,0.000646,0.000918,0.00266,0.00266,0.00417,0.00417,0.007979999999999999,0.00878,0.0147,0.015,0.0267,0.0406,0.00234,0.0457,1.06e-06,0.000122,0.000388,0.00102,0.00102,0.00167,0.0019,0.0019,0.00359,0.00395,0.00666,0.0402,0.0402,0.0402],[4.81e-06,0.000236,0.000394,0.000559,0.00162,0.00162,0.00254,0.00254,0.00486,0.00535,0.00898,0.00916,0.0163,0.0248,0.00194,0.0378,6.1e-07,7.02e-05,0.000224,0.0005910000000000001,0.0005910000000000001,0.000964,0.0011,0.0011,0.00207,0.00228,0.00385,0.0232,0.0232,0.0232],["LDHA;LDHB;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","PARP1;PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","LDHA;LDHB","LDHA;LDHB","ASNS;GAPDH","ATM;PARP1","ATM;PARP1","MYH11;TFRC","ATM;PARP1","LDHA;LDHB;PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","GAPDH;LDHA;LDHB","PYGB;PYGL;PYGM","LDHA;LDHB","PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","LDHA;LDHB","CA9","MYH11","GAPDH;MYH11"],[5,3,4,3,3,3,3,2,2,2,2,2,2,2,2,2,5,3,3,3,3,3,2,3,2,2,2,1,1,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>category<\/th>\n      <th>subcategory<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[8,9,10,12]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"category","targets":2},{"name":"subcategory","targets":3},{"name":"ID","targets":4},{"name":"Description","targets":5},{"name":"GeneRatio","targets":6},{"name":"BgRatio","targets":7},{"name":"pvalue","targets":8},{"name":"p.adjust","targets":9},{"name":"qvalue","targets":10},{"name":"geneID","targets":11},{"name":"Count","targets":12}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="reactome-pathways" class="level4" data-number="3.4.2.5">
<h4 data-number="3.4.2.5" class="anchored" data-anchor-id="reactome-pathways"><span class="header-section-number">3.4.2.5</span> Reactome pathways</h4>
<div class="tabset-margin-container"></div><div class="panel-tabset">
<ul class="nav nav-tabs" role="tablist"><li class="nav-item" role="presentation"><a class="nav-link active" id="tabset-5-1-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-1" role="tab" aria-controls="tabset-5-1" aria-selected="true">dotplot</a></li><li class="nav-item" role="presentation"><a class="nav-link" id="tabset-5-2-tab" data-bs-toggle="tab" data-bs-target="#tabset-5-2" role="tab" aria-controls="tabset-5-2" aria-selected="false">gene map</a></li></ul>
<div class="tab-content">
<div id="tabset-5-1" class="tab-pane active" role="tabpanel" aria-labelledby="tabset-5-1-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-33-1.png" class="img-fluid figure-img" width="1152"></p>
<figcaption>Figure 20: dotplot of enriched Reactome pathways</figcaption>
</figure>
</div>
</div>
</div>
</div>
<div id="tabset-5-2" class="tab-pane" role="tabpanel" aria-labelledby="tabset-5-2-tab">
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="proteomics-analysis_files/figure-html/unnamed-chunk-34-1.png" class="img-fluid figure-img" width="864"></p>
<figcaption>Figure 22: gene maps reporting relationships between Reactome pathways enriched across the different contrasts</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-c92c8a223c01a922b4fb" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-c92c8a223c01a922b4fb">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Enrich<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10"],["KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KICH","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP","KIRC_vs_KIRP"],["R-HSA-70221","R-HSA-8982491","R-HSA-71387","R-HSA-70268","R-HSA-71406","R-HSA-70221","R-HSA-8982491","R-HSA-71387","R-HSA-70268","R-HSA-71406"],["Glycogen breakdown (glycogenolysis)","Glycogen metabolism","Metabolism of carbohydrates","Pyruvate metabolism","Pyruvate metabolism and Citric Acid (TCA) cycle","Glycogen breakdown (glycogenolysis)","Glycogen metabolism","Metabolism of carbohydrates","Pyruvate metabolism","Pyruvate metabolism and Citric Acid (TCA) cycle"],["3/13","3/13","4/13","2/13","2/13","3/9","3/9","4/9","2/9","2/9"],["16/11009","27/11009","299/11009","31/11009","55/11009","16/11009","27/11009","299/11009","31/11009","55/11009"],[7.14e-07,3.7e-06,0.000314,0.000587,0.00185,2.1e-07,1.09e-06,6.03e-05,0.000273,0.0008630000000000001],[7.499999999999999e-05,0.000194,0.011,0.0154,0.0388,1.24e-05,3.23e-05,0.00119,0.00403,0.0102],[4.21e-05,0.000109,0.00617,0.00865,0.0218,4.43e-06,1.15e-05,0.000423,0.00144,0.00363],["PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB","PYGB;PYGL;PYGM","PYGB;PYGL;PYGM","GAPDH;PYGB;PYGL;PYGM","LDHA;LDHB","LDHA;LDHB"],[3,3,4,2,2,3,3,4,2,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster<\/th>\n      <th>ID<\/th>\n      <th>Description<\/th>\n      <th>GeneRatio<\/th>\n      <th>BgRatio<\/th>\n      <th>pvalue<\/th>\n      <th>p.adjust<\/th>\n      <th>qvalue<\/th>\n      <th>geneID<\/th>\n      <th>Count<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"targets":"_all","render":"$.fn.dataTable.render.ellipsis( 30, false )"},{"className":"dt-right","targets":[6,7,8,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"ID","targets":2},{"name":"Description","targets":3},{"name":"GeneRatio","targets":4},{"name":"BgRatio","targets":5},{"name":"pvalue","targets":6},{"name":"p.adjust","targets":7},{"name":"qvalue","targets":8},{"name":"geneID","targets":9},{"name":"Count","targets":10}],"overflow":"hidden","whiteSpace":"nowrap","scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
</section>
<section id="sec-lesson-learnt-proteomics" class="level2" data-number="3.5">
<h2 data-number="3.5" class="anchored" data-anchor-id="sec-lesson-learnt-proteomics"><span class="header-section-number">3.5</span> Lessons Learnts</h2>
<p>[[[[PROPER DESCRIPTION OF FINDINGS AND TRANSCRIPTOMICS ENRICHMENTS]]]]</p>
<p>So far, we have learnt:</p>
<ul>
<li>A</li>
</ul>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./transcriptomics-analysis.html" class="pagination-link" aria-label="Transcriptomics Analysis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./micro-RNAs-analysis.html" class="pagination-link" aria-label="micro-RNAs Analysis">
        <span class="nav-page-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>