<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.57">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>7&nbsp; Survival Analysis – Multi-omics integration for target discovery in cancer</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./factor-3-worse-outcome.html" rel="next">
<link href="./factor-analysis.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<link href="site_libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="site_libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="site_libs/datatables-binding-0.33/datatables.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="site_libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="site_libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>
<link href="site_libs/dt-ext-fixedcolumns-1.13.6/css/fixedColumns.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedcolumns-1.13.6/js/dataTables.fixedColumns.min.js"></script>
<link href="site_libs/dt-ext-fixedheader-1.13.6/css/fixedHeader.dataTables.min.css" rel="stylesheet">
<script src="site_libs/dt-ext-fixedheader-1.13.6/js/dataTables.fixedHeader.min.js"></script>
<link href="site_libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="site_libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>
<style>
#quarto-sidebar .sidebar-item-text {
  font-weight: bold;
}
#quarto-sidebar .chapter-number {
  font-weight: normal
}
#quarto-sidebar .chapter-title {
  font-weight: normal
}
#.nav-tabs .nav-link.active {
  background-color: rgba(239, 236, 255, 0.5);
}
</style>


</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">Multi-omics integration for target discovery in cancer</span>
    </a>
  </div>
        <div class="quarto-navbar-tools tools-end">
</div>
          <div id="quarto-search" class="" title="Search"></div>
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./survival-analysis.html"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Abstract</span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./gather-datasets-and-QC.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Gather Datasets and QC</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true">
 <span class="menu-text">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" role="navigation" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./dataset-exploration.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">Data Exploration - Kidney Cancer</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./transcriptomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Transcriptomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./proteomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">Proteomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./micro-RNAs-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">micro-RNAs Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./genomics-and-epigenomics-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genomics and Epigenomics Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-analysis.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./survival-analysis.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./factor-3-worse-outcome.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Characterization of Factor 3 (associated with worse outcome)</span></span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./kidney-carcinomas-lessons-learnt.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Characterization of Kidney Carcinomas: Take Home Message</span></span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <span class="sidebar-item-text sidebar-link text-start">
 <span class="menu-text">Discovery of Prognostic Biomarkers in Kidney Carcinomas</span></span>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <span class="sidebar-item-text sidebar-link text-start">
 <span class="menu-text">Therapeutic Target Discovery in Kidney Clear Cell Cancer</span></span>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#on-this-page" id="toc-on-this-page" class="nav-link active" data-scroll-target="#on-this-page"><span class="header-section-number">7.1</span> On this page</a></li>
  <li><a href="#survival-analysis-overall-survival" id="toc-survival-analysis-overall-survival" class="nav-link" data-scroll-target="#survival-analysis-overall-survival"><span class="header-section-number">7.2</span> Survival Analysis: Overall Survival</a>
  <ul>
  <li><a href="#metadata-only" id="toc-metadata-only" class="nav-link" data-scroll-target="#metadata-only"><span class="header-section-number">7.2.1</span> Metadata only</a>
  <ul>
  <li><a href="#kaplan-meier-estimator" id="toc-kaplan-meier-estimator" class="nav-link" data-scroll-target="#kaplan-meier-estimator"><span class="header-section-number">7.2.1.1</span> Kaplan-Meier Estimator</a></li>
  <li><a href="#cox-hazard-model" id="toc-cox-hazard-model" class="nav-link" data-scroll-target="#cox-hazard-model"><span class="header-section-number">7.2.1.2</span> Cox hazard model</a></li>
  </ul></li>
  <li><a href="#multi-omics-predictive-model-for-clinical-outcome" id="toc-multi-omics-predictive-model-for-clinical-outcome" class="nav-link" data-scroll-target="#multi-omics-predictive-model-for-clinical-outcome"><span class="header-section-number">7.2.2</span> Multi-omics predictive model for clinical outcome</a></li>
  </ul></li>
  <li><a href="#survival-analysis-progression-free-interval-pfi" id="toc-survival-analysis-progression-free-interval-pfi" class="nav-link" data-scroll-target="#survival-analysis-progression-free-interval-pfi"><span class="header-section-number">7.3</span> Survival Analysis: Progression-Free Interval (PFI)</a>
  <ul>
  <li><a href="#metadata-only-1" id="toc-metadata-only-1" class="nav-link" data-scroll-target="#metadata-only-1"><span class="header-section-number">7.3.1</span> Metadata only</a>
  <ul>
  <li><a href="#kaplan-meier-estimator-1" id="toc-kaplan-meier-estimator-1" class="nav-link" data-scroll-target="#kaplan-meier-estimator-1"><span class="header-section-number">7.3.1.1</span> Kaplan-Meier Estimator</a></li>
  <li><a href="#cox-regression-model" id="toc-cox-regression-model" class="nav-link" data-scroll-target="#cox-regression-model"><span class="header-section-number">7.3.1.2</span> Cox regression model</a></li>
  </ul></li>
  <li><a href="#multi-omics-predictive-model-for-clinical-outcome-1" id="toc-multi-omics-predictive-model-for-clinical-outcome-1" class="nav-link" data-scroll-target="#multi-omics-predictive-model-for-clinical-outcome-1"><span class="header-section-number">7.3.2</span> Multi-omics predictive model for clinical outcome</a></li>
  </ul></li>
  <li><a href="#sec-lesson-learnt-survival-analysis" id="toc-sec-lesson-learnt-survival-analysis" class="nav-link" data-scroll-target="#sec-lesson-learnt-survival-analysis"><span class="header-section-number">7.4</span> Lessons Learnt</a></li>
  <li><a href="#session-information" id="toc-session-information" class="nav-link" data-scroll-target="#session-information"><span class="header-section-number">7.5</span> Session Information</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default"><nav class="quarto-page-breadcrumbs quarto-title-breadcrumbs d-none d-lg-block" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./dataset-exploration.html">Characterization of Kidney Clear Cell, Papillary Cell and Chromophobe cancers</a></li><li class="breadcrumb-item"><a href="./survival-analysis.html"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></a></li></ol></nav>
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Survival Analysis</span></h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="on-this-page" class="level2" data-number="7.1">
<h2 data-number="7.1" class="anchored" data-anchor-id="on-this-page"><span class="header-section-number">7.1</span> On this page</h2>
<p>Biological insights and take-home messages are at the bottom of the page at section Lesson Learnt: <a href="#sec-lesson-learnt-survival-analysis" class="quarto-xref"><span>Section 7.4</span></a>.</p>
<ul>
<li>Here we investigate the clinical outcome reported for the kidney carcinoma patients, by performing a survival analysis with the <a href="https://doi.org/10.2307%2F2281868">Kaplan-Meier estimator</a> as well as modeling the <a href="https://doi.org/10.1146/annurev-statistics-033021-014043">Cox hazard-model</a> estimation.</li>
<li>We will perform the survival analyses based on both the Overal Survival (OS) data, as well as the progression-free interval (PFI).</li>
<li>We will compare the clinical outcome based on the reported kidney carcinoma type, and we will also group the patients based on their multi-omics profiles, as modeled with the multi-omics factor analysis (see <a href="factor-analysis.html" class="quarto-xref"><span>Chapter 6</span></a>).</li>
<li>Finally, we answer the hypothesis we raised in <em>Data Exploration - Kidney Cancer</em> chapter (see <a href="dataset-exploration.html" class="quarto-xref"><span>Chapter 1</span></a>): is KIRC associated with worse outcome when compared to KICH and KIRC.</li>
</ul>
</section>
<section id="survival-analysis-overall-survival" class="level2" data-number="7.2">
<h2 data-number="7.2" class="anchored" data-anchor-id="survival-analysis-overall-survival"><span class="header-section-number">7.2</span> Survival Analysis: Overall Survival</h2>
<p>TCGA provides the following time periods for the patients with the three kidney carcinomas:</p>
<ul>
<li><em>Overall Survival (OS):</em> the period of time that patients in a study have survived since diagnosis or treatment.</li>
<li><em>Disease-Specific Survival (DSS):</em> the period between the time of diagnosis the time of death caused by cancer. Patients who died from other causes are not included in this measurement.</li>
<li><em>Disease-Free Interval (DFI):</em> the period of time after primary treatment that the patient survives without any signs or symptoms of cancer.</li>
<li><em>Progression-Free Interval (PFI):</em> the period that the patient lives with cancer but it does not get worse.</li>
</ul>
<p>Let’s start analyzing the Overall Survival data. First, we start with the observational metadata only, before stratifying the patients based on the molecular signatures we have generated with the multi-omics factor analysis (see <a href="factor-analysis.html" class="quarto-xref"><span>Chapter 6</span></a>). In both cases, we model the Kaplan-Meier estimator and the Cox hazard regression model.</p>
<section id="metadata-only" class="level3" data-number="7.2.1">
<h3 data-number="7.2.1" class="anchored" data-anchor-id="metadata-only"><span class="header-section-number">7.2.1</span> Metadata only</h3>
<p>Let’s model the observational metadata on the clinical outcome reported by TCGA.</p>
<section id="kaplan-meier-estimator" class="level4" data-number="7.2.1.1">
<h4 data-number="7.2.1.1" class="anchored" data-anchor-id="kaplan-meier-estimator"><span class="header-section-number">7.2.1.1</span> Kaplan-Meier Estimator</h4>
<p>The <a href="https://doi.org/10.1016/j.otohns.2010.05.007">Kaplan-Meier estimator</a> is commonly used to analyze time-to-event data, such as the time until death occurs. For this, the Kaplan-Meier curve graphically represent the survival rate or survival function. Time is plotted on the x-axis and the survival rate is plotted on the y-axis.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 1: Kidney carcinoma types Kaplan-Meier Overall Survival curve.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>We calculated the Kaplan-Meier curve as a function of the Overall Survival time versus the vital status of the patient (<em>Alive</em> or <em>Dead</em>) in function of the kidney carcinoma type. We can already observe some differences in the Kaplan-Meier curves for the three kidney carcinoma types, with KIRC patients having a lower curve (higher mortality), KICH patients having a higher curve (lower mortality) and KIRP patients sitting in between KIRC and KICH curves. This is consistent with what we have observed in the exploration of the datasets looking for significant difference of each observed clinical covariate (see <a href="dataset-exploration.html" class="quarto-xref"><span>Chapter 1</span></a>).</p>
<p>While plotting the curves can help provide a visual representation of the differences between the three time-to-event group of data, we should run a <em>log rank test</em> to measure the statistical difference between the curves. The results are summarized in the table below. We observed a statistical difference between the Overall Survival time of the three kidney carcinomas (<em>n = 885</em>, <em>Chisq = 23.9</em> on 2 degree of freedom, <em>p-value = 6e-06</em>), confirming the hypothesis that KIRC patients have a significantly worse clinical outcome than KIRP patients or KICH patients.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-22636ef90e97a4560cb9" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-22636ef90e97a4560cb9">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 1: Log-ratio of Overall Survival differences bewteen kidney carcinoma types.<\/caption>","data":[["KICH","KIRC","KIRP"],[65,531,289],[9,174,44],[26,140,61],[11.14,8.039999999999999,4.54],[12.87,21.18,6.26]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>N<\/th>\n      <th>Observed<\/th>\n      <th>Expected<\/th>\n      <th>(O-E)^2/E<\/th>\n      <th>(O-E)^2/V<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"N","targets":1},{"name":"Observed","targets":2},{"name":"Expected","targets":3},{"name":"(O-E)^2/E","targets":4},{"name":"(O-E)^2/V","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="cox-hazard-model" class="level4" data-number="7.2.1.2">
<h4 data-number="7.2.1.2" class="anchored" data-anchor-id="cox-hazard-model"><span class="header-section-number">7.2.1.2</span> Cox hazard model</h4>
<p>In addition to the Kaplan-Meier estimate, the <a href="https://epirhandbook.com/en/new_pages/survival_analysis.html">Cox proportional hazards regression</a> is one of the most popular regression techniques for survival analysis. In a Cox proportional hazards regression model, the measure of effect is the hazard rate (HR), which is the risk of failure (here: risk of death), given that the participant has survived up to a specific time. We compare independent groups with in function of their hazards, and we use a hazard ratio, which is analogous to an odds ratio in the setting of multiple logistic regression analysis. The hazard represents the expected number of events per one unit of time.</p>
<ul>
<li>If the hazard ratio for a predictor is close to 1 then that predictor does not affect survival,</li>
<li>if the HR is less than 1, then the predictor is protective (i.e., associated with improved survival),</li>
<li>and if the HR is greater than 1, then the predictor is associated with increased risk (or decreased survival).</li>
</ul>
<p>The forest plot of the Cox Hazard Rations confirm that KIRC and KIRP patients have significantly higher risk of worse outcome than KICH patients.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 2: Kidney carcinomas Cox Proportional Harard model estimates based on patients Overall Survival.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>The Cox model we have generated is among the simpler we could have generated, and does not take into account several clinical covariates that are available for the kidney carcinoma patients that may correlate with each other and contribute the the Hazard Ratios.</p>
<p>Based on the initial exploration of the clinical covariates (see <a href="dataset-exploration.html" class="quarto-xref"><span>Chapter 1</span></a>) and their correlation with the multi-omics signatures we have generated (see <a href="factor-analysis.html" class="quarto-xref"><span>Chapter 6</span></a>), we include the following clinical covariates in our model:</p>
<ul>
<li><em>gender</em></li>
<li>Tumor stage, <em>tumor_stage</em></li>
<li>TMN main tumor pathology, <em>pathologic_t</em></li>
<li>TMN lymph nodes metastatis, <em>pathologic</em></li>
<li>Molecular subtypes based on miRNA, <em>Subtype_miRNA</em></li>
<li>Molecular subtypes based on transcriptomics, <em>Subtype_mRNA</em></li>
<li>Subtype based on immune response, <em>Subtype_Immune_Model_Based</em></li>
<li>Selected cancer subtype, <em>Subtype_Selected</em></li>
<li>Residual disease, <em>follow_up_tumor_status</em></li>
</ul>
<p>Despite their association with the mlti-omics signature we have identified, we could not include in the global Cox Hazard model <em>race</em>, tumor histological grade (<em>histological_grade</em>), molecular subtypes based on proteomics (<em>Subtype_protein</em>), molecular subtypes based on epigenomics (<em>Subtype_DNAmeth</em>) or <em>subtype_integrative</em> because they would completely overlap with one or more of the kidney carcinoma subtypes and would generate infinite coefficients, preventing the modeling.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 3: Full Cox Proportional Harxard model estimates based on patients Overall Survival.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>When including all clinical covariates in the model, we see a different picture on the Hazard Ratios than on the simple model based on kidney carcinoma types only. Under this more comprehensive model, KIRC patients don’t have a higher risk when compared to KICH or KIRP patients. Significant higher risk is associated with more advanced tumor stages (<em>stage iv</em>, HR between 1.112 and 10.5 with a p-value = 0.032) and presence of residual disease (<em>WITH TUMOR</em>, HR between 2.447 and 6.1, with a p-value &lt; 0.001).</p>
<p>With such an inclusive model, we should test if the assumption of the Cox regression model still holds true: the groups should be independent with respect to their hazards. Let’s test the proportional hazard assumption.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-57156b3aa74165da5a0a" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-57156b3aa74165da5a0a">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 2: Cox Proportional Hazard Assumption for the full model based on patients Overall Survival.<\/caption>","data":[["cancer_type","gender","tumor_stage","pathologic_t","pathologic","Subtype_mRNA","Subtype_miRNA","Subtype_Immune_Model_Based","Subtype_Selected","follow_up_tumor_status","GLOBAL"],[0.1487,0.994,12.2967,24.7414,1.4131,14.2402,6.808,13.4509,14.8351,0.129,70.7799],[1,1,4,10,3,3,3,4,5,1,35],[0.6998,0.3188,0.0153,0.0059,0.7025,0.0026,0.07829999999999999,0.009299999999999999,0.0111,0.7194,0.0003]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chisq<\/th>\n      <th>df<\/th>\n      <th>p<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":11,"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chisq","targets":1},{"name":"df","targets":2},{"name":"p","targets":3}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,11,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>The test indicates that <em>tumor_stage</em>, TMN main tumor pathology (<em>pathologic_t</em>), Molecular subtypes based on transcriptomics (<em>Subtype_mRNA</em>), Subtype based on immune response (<em>Subtype_Immune_Model_Based</em>) and Selected cancer subtype (<em>Subtype_Selected</em>) violates the proportional hazards assumption of independence. We can further characterize the association by displaying the scaled Schoenfeld residuals.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 5: Graphical Test of Proportional Hazards based on patients Overall Survival.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>The model results indicate there is a significant positive association between tumor stage and all-cause mortality, and a statistically non-significant positive association with <em>Subtype_mRNA</em> and a statistically non-significant overall negative association with <em>Subtype_Immune_Model_Based</em> and <em>Subtype_Selected</em> and the all-cause mortality.</p>
<p>The full model provides us with fundamental insights to drop clinical covariates that vilates the proportional hazard assumption and allows us to compute the final Cox proportional Hazard Ratios.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 6: Final Cox Proportional Harxard model estimates based on patients Overall Survival.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Let’s test that the proportional hazards assumptions are respected.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-114d0471f99e52588893" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-114d0471f99e52588893">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 3: Cox Proportional Hazard Assumption for the final model based on patients Overall Survival.<\/caption>","data":[["cancer_type","gender","pathologic","Subtype_miRNA","follow_up_tumor_status","GLOBAL"],[0.1177,0.109,3.035,3.4221,0.5229,6.7158],[1,1,3,3,1,9],[0.7315,0.7413,0.3863,0.331,0.4696,0.6667]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chisq<\/th>\n      <th>df<\/th>\n      <th>p<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":11,"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chisq","targets":1},{"name":"df","targets":2},{"name":"p","targets":3}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,11,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>The final model is in contrast with initial, more naive, simple model of Overall Survival in function of the cancer type, and suggests that <strong>there is no significantly higher hazard associated with KIRC when compared to KICH or KIRP</strong>. Presence of metastases on lymphnodes (<em>pathologic</em>) have significantly higher risk: for N1, 1.04 to 3.8 HR with a p-value of 0.037, N2 is not significant (p-value = 0.291) possibly due to its lower counts (N=7). Subtype microRNA 2 is also associated with higher risk (HR between 1.34 and 3.5, p-value = 0.002), and of course, residual disease is strongly associated with higher risk of worse clinical outcome (HR between 3.86 and 7.9, p-value &lt; 0.001).</p>
</section>
</section>
<section id="multi-omics-predictive-model-for-clinical-outcome" class="level3" data-number="7.2.2">
<h3 data-number="7.2.2" class="anchored" data-anchor-id="multi-omics-predictive-model-for-clinical-outcome"><span class="header-section-number">7.2.2</span> Multi-omics predictive model for clinical outcome</h3>
<p>The factors inferred with the multi-omics factor analysis (see <a href="factor-analysis.html" class="quarto-xref"><span>Chapter 6</span></a>) can be related to the <a href="https://raw.githack.com/bioFAM/MOFA2_tutorials/master/R_tutorials/CLL.html">clinical outcomes</a>. If a factor has a influence on the surivival time it will receive a high absolute coefficient in the Cox model. In particular:</p>
<ul>
<li>If the coefficient is positive, samples with large factor values have an increased hazard compared to samples with small factor values.</li>
<li>If the coefficient is negative, samples with small factor values have an increased hazard compared to samples with a large factor values.</li>
</ul>
<p>Now, we are going to test the correlation between the factors we have molecularly characterized (i.e.: Factor 1 and Factor 2) with the Overall Survival of the kidney carcinomas patients. We can test if the biopsies clustered based on the multi-omics signatures can provide a better predictive model of clinical outcome than the one based solely on the provided metadata.</p>
<p>When fitting the Cox hazard model and testing the significance with a <em>log rank test</em> we obtain a significant results (<em>Likelihood ratio test = 171.1 </em> on 10 degrees of freedom, <em>p-value =&lt; 2.2e-16</em>), indicating that there is a significant association between the multi-omics factors and and increased or a decreased risk.</p>
<p>Six of the ten factors are significantly associated with a higher or lower proportional hazard, however, the increase or decrease in Hazard Ratio is very small:</p>
<ul>
<li><strong>Factor 1</strong> is significantly (<em>p-value = 0.0005</em>) associated with a lower hazard. The positive weights of Factor 1 correlated with KICH and KIRP, while the negative weights correlated with KIRC, suggesting that KIRC biopises with the multi-omics signature of Factor 1 have significantly higher risk. The reduced Hazard Ratio in KICH and KIRP, however, is very small (<em>HR: 0.81</em>).</li>
<li><strong>Factor 2</strong> is not significantly associated with a higher or lower hazard (<em>p-value &gt; 0.05</em>).</li>
<li><strong>Factor 3</strong> is significantly associated with higher hazard (<em>p-value &lt; 2.66E-16</em>). Positive weights in Factor 3 were associated with the tumor severity and the covariates <em>pathologic tumor grading</em>, <em>subtype selected</em>, <em>tumor cells</em>, <em>tumor nuclei</em>, <em>tumor stage</em>, <em>tumor status</em> and <em>vital status</em>. The Hazard Ratio is also large (<em>HR: 1.65</em>), suggesting that Factor 3 multi-omics signature can describe patients with higher risk and possible worse clinical outcome.</li>
</ul>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-f2431a59885c4afdf422" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-f2431a59885c4afdf422">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 4: Cox Proportional Hazard model for the multi-omics factors based on patients Overall Survival.<\/caption>","data":[["Factor 1","Factor 2","Factor 3","Factor 4","Factor 5","Factor 6","Factor 7","Factor 8","Factor 9","Factor 10"],["-0.210409","-0.141805","0.502391","-0.042219","0.192537","0.00665","0.179446","0.301923","-0.007285","-0.167755"],["0.810253","0.867791","1.652669","0.958659","1.212321","1.006672","1.196554","1.352457","0.992741","0.845561"],["0.060443","0.074405","0.061359","0.080386","0.061642","0.074367","0.063222","0.053316","0.071742","0.065304"],["-3.481","-1.906","8.188","-0.525","3.123","0.089","2.838","5.663","-0.102","-2.569"],["0.000499","0.056669","2.66E-16","0.599439","0.001787","0.928745","0.004535","1.49E-08","0.919114","0.010204"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>coef<\/th>\n      <th>exp.coef.<\/th>\n      <th>se.coef.<\/th>\n      <th>z<\/th>\n      <th>p<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"coef","targets":1},{"name":"exp.coef.","targets":2},{"name":"se.coef.","targets":3},{"name":"z","targets":4},{"name":"p","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>Visualizing the Cox proportional Hazard Ratio model results on a forest plot help better visualizing the higher Hazard Ratio associated with Factor 3.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-17-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 7: Cox Proportional Harxard model estimates based on multi-omics factos and patients Overall Survival.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Also the Kaplan-Meier survival curves propely visualize the difference in survival times between Factor 1 or Factor 2 multi-omics signatures (discriminating between KIRC vs.&nbsp;KICH and KIRP, and KICH vs.&nbsp;KIRC and KIRP, respectively) and Factor 3 multi-omics signature (correlating with more advanced tumor, residual disease and worse clinical outcome).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-18-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 8: Multi-omics factors Kaplan-Meier Overall Survival curve.</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="survival-analysis-progression-free-interval-pfi" class="level2" data-number="7.3">
<h2 data-number="7.3" class="anchored" data-anchor-id="survival-analysis-progression-free-interval-pfi"><span class="header-section-number">7.3</span> Survival Analysis: Progression-Free Interval (PFI)</h2>
<p>While Overall Survival provides a good overview of the clinical outcome, it does not take into account that in the meanwhile patients may die due to causes unrelated to cancer. Therefore, we have decide to perform a similar survival analysis based on the Progression-Free Interval (PFI) time data available at TCGA for the kidney carcinoma patients. As we have done for the Overall Survival, we start with the observational metadata only, before stratifying the patients based on the molecular signatures we have generated with the multi-omics factor analysis. In both cases, we model the Kaplan-Meier estimator and the Cox hazard regression model.</p>
<section id="metadata-only-1" class="level3" data-number="7.3.1">
<h3 data-number="7.3.1" class="anchored" data-anchor-id="metadata-only-1"><span class="header-section-number">7.3.1</span> Metadata only</h3>
<p>Let’s model the observational metadata on the clinical outcome reported by TCGA.</p>
<section id="kaplan-meier-estimator-1" class="level4" data-number="7.3.1.1">
<h4 data-number="7.3.1.1" class="anchored" data-anchor-id="kaplan-meier-estimator-1"><span class="header-section-number">7.3.1.1</span> Kaplan-Meier Estimator</h4>
<p>The Kaplan-Meier curves obtained in function of the Progression-Free Interval are very similar to the ones obtained with the Overall Survival data, with the only noticeable difference being a slightly lower curve for KICH and KIRP cancers within the first 500 days after diagnosis.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 9: Kidney carcinoma types Kaplan-Meier Progression-Free Interval curve.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>While plotting the curves can help provide a visual representation of the differences between the three time-to-event group of data, we should run a log rank test to measure the statistical difference between the curves. The results are summarized in the table below. We observed a statistical difference between the Progression-Free Interval of the three kidney carcinomas (n = 885, Chisq = 23.9 on 2 degree of freedom, p-value = 6e-06), similarly to what observed with the Overall Survival data.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-2bdb6f3dcdf8cc26b891" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2bdb6f3dcdf8cc26b891">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 5: Log-ratio of the Progression-Free Interval differences bewteen kidney carcinoma types.<\/caption>","data":[["KICH","KIRC","KIRP"],[65,529,288],[9,173,43],[25,135,65],[10.56,10.81,7.33],[12.31,27.11,10.41]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>N<\/th>\n      <th>Observed<\/th>\n      <th>Expected<\/th>\n      <th>(O-E)^2/E<\/th>\n      <th>(O-E)^2/V<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"N","targets":1},{"name":"Observed","targets":2},{"name":"Expected","targets":3},{"name":"(O-E)^2/E","targets":4},{"name":"(O-E)^2/V","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="cox-regression-model" class="level4" data-number="7.3.1.2">
<h4 data-number="7.3.1.2" class="anchored" data-anchor-id="cox-regression-model"><span class="header-section-number">7.3.1.2</span> Cox regression model</h4>
<p>Let’s now model the Cox proportional hazards regression on the Progression-Free Interval data.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-23-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 10: Kidney carcinomas Cox Proportional Harard model estimates based on patients Progression-Free Interval.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>The forest plot of the Cox Hazard Rations on patients Progression-Free Interval indicates that KIRC patients have a significantly higher risk of worse outcome than KICH or KIRP patients. This is in contrast with the simple Cox regression model on Overall Surviaval, where also KIRP patients had a significant higher risk than KICH patients.</p>
<p>Let’s now evaluate the full model by adding the clinical covariates: <em>gender</em>, <em>tumor_stage</em>, <em>pathologic_t</em>, <em>pathologic</em>, <em>Subtype_miRNA</em>, <em>Subtype_mRNA</em>, <em>Subtype_Immune_Model_Based</em>, <em>Subtype_Selected</em>, <em>follow_up_tumor_status</em>.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-25-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 11: Full Cox Proportional Harxard model estimates based on patients Progression-Free Interval.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Similarly to what observed for the Overall Survival data, when including all clinical covariates in the model, we see a different picture on the Hazard Ratios than on the simple model based on kidney carcinoma types only. Under this more comprehensive model, KIRC patients don’t have a significantly higher risk when compared to KICH or KIRP patients. Significant higher risk is associated with more advanced tumor stages (<em>stage iv</em>, HR between 1.109 and 9.80 with a p-value = 0.032), the subtype microRNA 2 (HR between 1.108 and 3.72, p-value = 0.022) and presence of residual disease (<em>WITH TUMOR</em>, HR between 3.773 and 9.71, with a p-value &lt; 0.001). Lower risk is instead significantly associated with the <em>inflammatory immune subtype (immune C3)</em> and the <em>Lymphocyte Depleted (immune C4)</em> subtype, with reduced HR between 0.0075 and 0.63 and 0.053 and 0.75, respectively.</p>
<p>As for theOverall Survival data, we should test if the assumption of the Cox regression model still holds true: the groups should be independent with respect to their hazards. Let’s test the proportional hazard assumption.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-cc9026f45531756468f9" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-cc9026f45531756468f9">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 6: Cox Proportional Hazard Assumption for the full model based on patients Progression-Free Interval.<\/caption>","data":[["cancer_type","gender","tumor_stage","pathologic_t","pathologic","Subtype_mRNA","Subtype_miRNA","Subtype_Immune_Model_Based","Subtype_Selected","follow_up_tumor_status","GLOBAL"],[1.2284,0.3392,14.2257,25.399,0.8114,11.2578,6.4911,5.967,7.5377,0.2592,68.6647],[1,1,4,10,3,3,3,4,5,1,35],[0.2677,0.5603,0.0066,0.0046,0.8467,0.0104,0.09,0.2016,0.1836,0.6107,0.0005999999999999999]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chisq<\/th>\n      <th>df<\/th>\n      <th>p<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chisq","targets":1},{"name":"df","targets":2},{"name":"p","targets":3}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>The test indicates that <em>tumor_stage</em>, TMN main tumor pathology (<em>pathologic_t</em>), Molecular subtypes based on transcriptomics (<em>Subtype_mRNA</em>), Subtype based on immune response (<em>Subtype_Immune_Model_Based</em>) and Selected cancer subtype (<em>Subtype_Selected</em>) violates the proportional hazards assumption of independence. We can further characterize the association by displaying the scaled Schoenfeld residuals.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-27-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 12: Graphical Test of Proportional Hazards based on patients Progression-Free Interval.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>The model results indicate there is a significant positive association between tumor stage and all-cause mortality, and a statistically non-significant positive association with <em>Subtype_mRNA</em> and a statistically non-significant overall negative association with <em>Subtype_Immune_Model_Based</em> and <em>Subtype_Selected</em> and the all-cause mortality.</p>
<p>The full model provides us with fundamental insights to drop clinical covariates that vilates the proportional hazard assumption and allows us to compute the final Cox proportional Hazard Ratios.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-29-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 13: Final Cox Proportional Harxard model estimates based on patients Progression-Free Interval.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Let’s confirm that the proportional hazards assumptions are respected.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-d31d79fd500c0446bbc7" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d31d79fd500c0446bbc7">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 7: Cox Proportional Hazard Assumption for the final model based on patients Progression-Free Interval.<\/caption>","data":[["cancer_type","gender","pathologic","Subtype_miRNA","follow_up_tumor_status","GLOBAL"],[1.2635,0.09859999999999999,0.1331,3.3536,1.6579,6.6227],[1,1,3,3,1,9],[0.261,0.7536,0.9876,0.3402,0.1979,0.6763]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>chisq<\/th>\n      <th>df<\/th>\n      <th>p<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":11,"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"chisq","targets":1},{"name":"df","targets":2},{"name":"p","targets":3}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,11,25,50,100]}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>The final model is in contrast with initial, more naive, simple model of Progression-Free Interval in function of the cancer type, and suggests that <strong>there is no significantly higher hazard associated with KIRC when compared to KICH or KIRP</strong>. Presence of metastases on lymphnodes (<em>pathologic</em>) have significantly higher risk: N1 is not significant (p-value = 0.056), N2 is significant, 1.01 to 14.4 HR with a (p-value = 0.048). Subtype microRNA 2 and 3 are also associated with higher risk, and of course, residual disease is strongly associated with higher risk of worse clinical outcome (HR between 6.04 and 12.8, p-value &lt; 0.001).</p>
</section>
</section>
<section id="multi-omics-predictive-model-for-clinical-outcome-1" class="level3" data-number="7.3.2">
<h3 data-number="7.3.2" class="anchored" data-anchor-id="multi-omics-predictive-model-for-clinical-outcome-1"><span class="header-section-number">7.3.2</span> Multi-omics predictive model for clinical outcome</h3>
<p>Now, we are going to test the correlation between the factors we have molecularly characterized (i.e.: Factor 1 and Factor 2) with the Progression-Free Interval of the kidney carcinomas patients. We can test if the biopsies clustered based on the multi-omics signatures can provide a better predictive model of clinical outcome than the one based solely on the provided metadata.</p>
<p>When fitting the Cox hazard model and testing the significance with a log rank test we obtain a significant results (<em>Likelihood ratio test = 171.1</em> on 10 degrees of freedom, <em>p-value =&lt; 2.2e-16</em>), indicating that there is a significant association between the multi-omics factors and and increased or a decreased risk.</p>
<div class="cell" data-cap-location="margin">
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-3b92e5396ab9eb77b3cb" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3b92e5396ab9eb77b3cb">{"x":{"filter":"none","vertical":false,"extensions":["FixedColumns","FixedHeader"],"caption":"<caption>Table 8: Cox Proportional Hazard model for the multi-omics factors based on patients Progression-Free Interval.<\/caption>","data":[["Factor 1","Factor 2","Factor 3","Factor 4","Factor 5","Factor 6","Factor 7","Factor 8","Factor 9","Factor 10"],["-0.250259","-0.137154","0.51265","-0.025933","0.227808","0.026058","0.25395","0.28466","-0.004506","-0.135766"],["0.778599","0.871836","1.66971","0.9744","1.255844","1.0264","1.289108","1.32931","0.995504","0.873047"],["0.060483","0.07498","0.062282","0.081319","0.063076","0.073998","0.064468","0.053568","0.073052","0.063964"],["-4.138","-1.829","8.231","-0.319","3.612","0.352","3.939","5.314","-0.062","-2.123"],["3.51E-05","0.06737","&lt; 2.00E-16","0.749799","0.000304","0.724731","8.18E-05","1.07E-07","0.950813","0.033794"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>coef<\/th>\n      <th>exp.coef.<\/th>\n      <th>se.coef.<\/th>\n      <th>z<\/th>\n      <th>p<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"paging":true,"fixedHeader":false,"pageLength":10,"columnDefs":[{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"coef","targets":1},{"name":"exp.coef.","targets":2},{"name":"se.coef.","targets":3},{"name":"z","targets":4},{"name":"p","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>Visualizing the Cox proportional Hazard Ratio model results on a forest plot help better visualizing the higher Hazard Ratio associated with Factor 3.</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-33-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 14: Cox Proportional Harxard model estimates based on multi-omics factos and patients Progression-Free Interval.</span></figcaption>
</figure>
</div>
</div>
</div>
<p>Also the Kaplan-Meier survival curves propely visualize the difference in survival times between Factor 1 or Factor 2 multi-omics signatures (discriminating between KIRC vs.&nbsp;KICH and KIRP, and KICH vs.&nbsp;KIRC and KIRP, respectively) and Factor 3 multi-omics signature (correlating with more advanced tumor, residual disease and worse clinical outcome).</p>
<div class="cell">
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="survival-analysis_files/figure-html/unnamed-chunk-34-1.png" class="img-fluid figure-img" width="960"></p>
<figcaption><span style="color: steelblue;">Figure 15: Multi-omics factors Kaplan-Meier Progression-Free Interval curve.</span></figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="sec-lesson-learnt-survival-analysis" class="level2" data-number="7.4">
<h2 data-number="7.4" class="anchored" data-anchor-id="sec-lesson-learnt-survival-analysis"><span class="header-section-number">7.4</span> Lessons Learnt</h2>
<p>The survival analysis finally provided us with enough evidence to answer the hypothesis we have formulated upon a first inspection of the clinical data (see <a href="dataset-exploration.html" class="quarto-xref"><span>Chapter 1</span></a>): is KIRC associated with worse prognosis? Based on the clinical and the multi-omics data, we have learnt:</p>
<ul>
<li>KIRC patients seem to have a poorer prognosis when compared to KICH or KIRP patients based on both Overall Survival and Progression-Free Interval if we model the survival analyses purely on the kidney carcinoma type (i.e.: KICH, KIRC or KIRP).</li>
<li>When including additional clinical covariates, it emerges a different picture: KIRC is not associated with worse clinical outcome, but residual disease and presence of metastasis on lymph nodes (<em>TMN lymph nodes metastatis</em>) are.</li>
<li>The multi-omics signatures we have identified with the factor analysis (see <a href="factor-analysis.html" class="quarto-xref"><span>Chapter 6</span></a>) confirmed that there is little higher risk associated with Factor 1 signature (discriminating KIRC from KICH and KIRP) and Factor 2 (discriminating KICH from KIRC and KIRP). However, Factor 3 is strongly associated with a higher Hazard Ratio (<em>HR: 1.65</em>) and with a poorer prognosis based on the Kaplan-Meier estimate. Factor 3 is associated with the tumor severity and the covariates <em>pathologic tumor grading</em>, <em>subtype selected</em>, <em>tumor cells</em>, <em>tumor nuclei</em>, <em>tumor stage</em>, <em>tumor status</em> and <em>vital status</em>.</li>
<li>The results of the survival analysis are consistent for both the Overall Survival and the Progression-Free Interval times.</li>
</ul>
</section>
<section id="session-information" class="level2" data-number="7.5">
<h2 data-number="7.5" class="anchored" data-anchor-id="session-information"><span class="header-section-number">7.5</span> Session Information</h2>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.3.2 (2023-10-31)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: openSUSE Tumbleweed

Matrix products: default
BLAS/LAPACK: /home/andrea/miniforge3/envs/moai/lib/libmkl_rt.so.2;  LAPACK version 3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=it_IT.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: Europe/Brussels
tzcode source: system (glibc)

attached base packages:
 [1] parallel  grid      stats4    stats     graphics  grDevices utils    
 [8] datasets  methods   base     

other attached packages:
 [1] UpSetR_1.4.0                                      
 [2] tidygraph_1.3.1                                   
 [3] umap_0.2.10.0                                     
 [4] survminer_0.5.0                                   
 [5] ggpubr_0.6.0                                      
 [6] survival_3.8-3                                    
 [7] stringr_1.5.1                                     
 [8] STRINGdb_2.14.3                                   
 [9] scales_1.3.0                                      
[10] RColorBrewer_1.1-3                                
[11] pathview_1.42.0                                   
[12] org.Hs.eg.db_3.18.0                               
[13] MOFA2_1.12.1                                      
[14] IlluminaHumanMethylation450kanno.ilmn12.hg19_0.6.1
[15] minfi_1.48.0                                      
[16] bumphunter_1.44.0                                 
[17] locfit_1.5-9.10                                   
[18] iterators_1.0.14                                  
[19] foreach_1.5.2                                     
[20] Biostrings_2.70.3                                 
[21] XVector_0.42.0                                    
[22] igraph_2.1.4                                      
[23] gridExtra_2.3                                     
[24] ggfortify_0.4.17                                  
[25] ggraph_2.2.1                                      
[26] ggplot2_3.5.1                                     
[27] forcats_1.0.0                                     
[28] edgeR_4.0.16                                      
[29] limma_3.58.1                                      
[30] DT_0.33                                           
[31] dplyr_1.1.4                                       
[32] DOSE_3.28.2                                       
[33] DESeq2_1.42.1                                     
[34] SummarizedExperiment_1.32.0                       
[35] MatrixGenerics_1.14.0                             
[36] matrixStats_1.5.0                                 
[37] GenomicRanges_1.54.1                              
[38] GenomeInfoDb_1.38.8                               
[39] data.table_1.16.4                                 
[40] cowplot_1.1.3                                     
[41] clusterProfiler_4.10.1                            
[42] BiocSingular_1.18.0                               
[43] BiocParallel_1.36.0                               
[44] AnnotationDbi_1.64.1                              
[45] IRanges_2.36.0                                    
[46] S4Vectors_0.40.2                                  
[47] Biobase_2.62.0                                    
[48] BiocGenerics_0.48.1                               

loaded via a namespace (and not attached):
  [1] progress_1.2.3            HDF5Array_1.30.1         
  [3] vctrs_0.6.5               RApiSerialize_0.1.4      
  [5] digest_0.6.37             png_0.1-8                
  [7] ggrepel_0.9.6             corrplot_0.95            
  [9] MASS_7.3-60.0.1           reshape_0.8.9            
 [11] reshape2_1.4.4            qvalue_2.34.0            
 [13] withr_3.0.2               xfun_0.50                
 [15] ggfun_0.1.8               doRNG_1.8.6.1            
 [17] memoise_2.0.1             commonmark_1.9.2         
 [19] gson_0.1.0                tidytree_0.4.6           
 [21] zoo_1.8-12                gtools_3.9.5             
 [23] KEGGgraph_1.62.0          maxstat_0.7-25           
 [25] Formula_1.2-5             prettyunits_1.2.0        
 [27] KEGGREST_1.42.0           httr_1.4.7               
 [29] rstatix_0.7.2             restfulr_0.0.15          
 [31] hash_2.2.6.3              rhdf5filters_1.14.1      
 [33] stringfish_0.16.0         rhdf5_2.46.1             
 [35] generics_0.1.3            dir.expiry_1.10.0        
 [37] curl_6.2.0                zlibbioc_1.48.2          
 [39] ScaledMatrix_1.10.0       polyclip_1.10-7          
 [41] GenomeInfoDbData_1.2.11   quadprog_1.5-8           
 [43] SparseArray_1.2.4         xtable_1.8-4             
 [45] evaluate_1.0.3            S4Arrays_1.2.1           
 [47] BiocFileCache_2.10.2      preprocessCore_1.64.0    
 [49] hms_1.1.3                 irlba_2.3.5.1            
 [51] qs_0.27.2                 colorspace_2.1-1         
 [53] filelock_1.0.3            reticulate_1.40.0        
 [55] magrittr_2.0.3            readr_2.1.5              
 [57] Rgraphviz_2.46.0          viridis_0.6.5            
 [59] ggtree_3.10.1             lattice_0.22-6           
 [61] genefilter_1.84.0         XML_3.99-0.18            
 [63] shadowtext_0.1.4          pillar_1.10.1            
 [65] nlme_3.1-167              caTools_1.18.3           
 [67] compiler_4.3.2            beachmat_2.18.1          
 [69] RSpectra_0.16-2           stringi_1.8.4            
 [71] GenomicAlignments_1.38.2  plyr_1.8.9               
 [73] crayon_1.5.3              abind_1.4-8              
 [75] BiocIO_1.12.0             gridGraphics_0.5-1       
 [77] ggtext_0.1.2              chron_2.3-62             
 [79] graphlayouts_1.2.2        bit_4.5.0.1              
 [81] fastmatch_1.1-6           codetools_0.2-20         
 [83] openssl_2.3.1             crosstalk_1.2.1          
 [85] bslib_0.8.0               multtest_2.58.0          
 [87] splines_4.3.2             markdown_1.13            
 [89] Rcpp_1.0.14               basilisk_1.14.3          
 [91] dbplyr_2.5.0              sparseMatrixStats_1.14.0 
 [93] HDO.db_0.99.1             exactRankTests_0.8-35    
 [95] gridtext_0.1.5            knitr_1.49               
 [97] blob_1.2.4                fs_1.6.5                 
 [99] DelayedMatrixStats_1.24.0 ggsignif_0.6.4           
[101] ggplotify_0.1.2           tibble_3.2.1             
[103] sqldf_0.4-11              Matrix_1.6-5             
[105] statmod_1.5.0             tzdb_0.4.0               
[107] tweenr_2.0.3              pkgconfig_2.0.3          
[109] pheatmap_1.0.12           tools_4.3.2              
[111] cachem_1.1.0              RSQLite_2.3.9            
[113] viridisLite_0.4.2         DBI_1.2.3                
[115] fastmap_1.2.0             rmarkdown_2.29           
[117] Rsamtools_2.18.0          sass_0.4.9               
[119] broom_1.0.7               patchwork_1.3.0          
[121] graph_1.80.0              carData_3.0-5            
[123] scrime_1.3.5              farver_2.1.2             
[125] scatterpie_0.2.4          gsubfn_0.7               
[127] yaml_2.3.10               rtracklayer_1.62.0       
[129] illuminaio_0.44.0         cli_3.6.3                
[131] purrr_1.0.2               siggenes_1.76.0          
[133] GEOquery_2.70.0           lifecycle_1.0.4          
[135] askpass_1.2.1             uwot_0.2.2               
[137] mvtnorm_1.3-3             backports_1.5.0          
[139] annotate_1.80.0           gtable_0.3.6             
[141] rjson_0.2.23              ape_5.8-1                
[143] jsonlite_1.8.9            bitops_1.0-9             
[145] bit64_4.6.0-1             Rtsne_0.17               
[147] yulab.utils_0.2.0         base64_2.0.2             
[149] proto_1.0.0               RcppParallel_5.1.10      
[151] jquerylib_0.1.4           GOSemSim_2.28.1          
[153] survMisc_0.5.6            lazyeval_0.2.2           
[155] htmltools_0.5.8.1         enrichplot_1.22.0        
[157] KMsurv_0.1-5              GO.db_3.18.0             
[159] rappdirs_0.3.3            basilisk.utils_1.14.1    
[161] glue_1.8.0                RCurl_1.98-1.16          
[163] treeio_1.26.0             mclust_6.1.1             
[165] R6_2.5.1                  tidyr_1.3.1              
[167] gplots_3.2.0              km.ci_0.5-6              
[169] labeling_0.4.3            GenomicFeatures_1.54.4   
[171] rngtools_1.5.2            Rhdf5lib_1.24.2          
[173] beanplot_1.3.1            aplot_0.2.4              
[175] DelayedArray_0.28.0       tidyselect_1.2.1         
[177] plotrix_3.8-4             ggforce_0.4.2            
[179] xml2_1.3.6                car_3.1-3                
[181] rsvd_1.0.5                munsell_0.5.1            
[183] KernSmooth_2.23-26        nor1mix_1.3-3            
[185] htmlwidgets_1.6.4         fgsea_1.28.0             
[187] biomaRt_2.58.2            rlang_1.1.5              </code></pre>
</div>
</div>
</div>
</div>
</div>


</section>

</main> <!-- /main -->
<script data-goatcounter="https://19adc99.goatcounter.com/count" async="" src="//gc.zgo.at/count.js"></script>
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./factor-analysis.html" class="pagination-link" aria-label="Factor Analysis">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Factor Analysis</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./factor-3-worse-outcome.html" class="pagination-link" aria-label="Characterization of Factor 3 (associated with worse outcome)">
        <span class="nav-page-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Characterization of Factor 3 (associated with worse outcome)</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
 Copyright © 2025 Andrea Del Cortona.
  </li>  
</ul>
    </div>   
    <div class="nav-footer-center">
      <ul class="footer-items list-unstyled">
    <li class="nav-item">
    <a class="nav-link" href="https://quarto.org/">
<p>This page is built with ❤️ and <em>Quarto</em>.</p>
</a>
  </li>  
</ul>
    </div>
    <div class="nav-footer-right">
      <ul class="footer-items list-unstyled">
    <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/19ADC99">
      <i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
    </a>
  </li>  
    <li class="nav-item compact">
    <a class="nav-link" href="https://www.linkedin.com/in/andrea-del-cortona-86a75474/">
      <i class="bi bi-linkedin" role="img" aria-label="Lindekin">
</i> 
    </a>
  </li>  
</ul>
    </div>
  </div>
</footer>




</body></html>